<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }   
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:1200" >

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Galaxy wrapper id</th>
      <th>Galaxy wrapper version</th>
      <th>Conda version</th>
      <th>Conda id</th>
      <th>Status</th>
      <th>bio.tool id</th>
      <th>bio.tool name</th>
      <th>EDAM operation</th>
      <th>EDAM topic</th>
      <th>Description</th>
      <th>bio.tool description</th>
      <th>Status</th>
      <th>Source</th>
      <th>ToolShed categories</th>
      <th>ToolShed id</th>
      <th>Galaxy wrapper owner</th>
      <th>Galaxy wrapper source</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>braker</td>
      <td>2.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>braker3</td>
      <td>3.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BRAKER3 is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/BRAKER</td>
      <td>Genome annotation</td>
      <td>braker3</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/braker</td>
    </tr>
    <tr>
      <td></td>
      <td>helixer</td>
      <td>0.3.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene calling with Deep Neural Networks</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/weberlab-hhu/Helixer</td>
      <td>Genome annotation</td>
      <td>helixer</td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/helixer</td>
    </tr>
    <tr>
      <td></td>
      <td>logol</td>
      <td>1.7.8</td>
      <td>1.7.8</td>
      <td>logol</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Logol is a pattern matching grammar language and a set of tools to search a pattern in a sequence</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://logol.genouest.org/web/app.php/logol</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>genouest</td>
      <td>https://github.com/genouest/galaxy-tools/tree/master/tools/logol</td>
    </tr>
    <tr>
      <td></td>
      <td>peptimapper</td>
      <td>2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Proteogenomics workflow for the expert annotation of eukaryotic genomes</td>
      <td></td>
      <td>To update</td>
      <td>https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5431-9</td>
      <td>Proteomics</td>
      <td></td>
      <td>genouest</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>GAFA</td>
      <td>0.3.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene Align and Family Aggregator</td>
      <td></td>
      <td>To update</td>
      <td>http://aequatus.tgac.ac.uk</td>
      <td>Visualization</td>
      <td>gafa</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/GAFA/</td>
    </tr>
    <tr>
      <td></td>
      <td>TreeBest</td>
      <td>1.9.2.post0</td>
      <td>1.9.2.post0</td>
      <td>treebest</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TreeBeST best</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://treesoft.sourceforge.net/treebest.shtml</td>
      <td>Phylogenetics</td>
      <td>treebest_best</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/TreeBest</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_parser</td>
      <td>0.1.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert 12- or 24-column BLAST output into 3-column hcluster_sg input</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>blast_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/blast_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>ete</td>
      <td>3.1.2</td>
      <td>3.1.1</td>
      <td>ete3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse phylogenetic trees using the ETE Toolkit</td>
      <td></td>
      <td>To update</td>
      <td>http://etetoolkit.org/</td>
      <td>Phylogenetics</td>
      <td>ete</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/ete</td>
    </tr>
    <tr>
      <td></td>
      <td>gblocks</td>
      <td>0.91b</td>
      <td>0.91b</td>
      <td>gblocks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gblocks</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://molevol.cmima.csic.es/castresana/Gblocks.html</td>
      <td>Sequence Analysis</td>
      <td>gblocks</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/gblocks</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg</td>
      <td>0.5.1.1</td>
      <td>0.5.1</td>
      <td>hcluster_sg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hierarchically clustering on a sparse graph</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/douglasgscofield/hcluster</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg</td>
    </tr>
    <tr>
      <td></td>
      <td>hcluster_sg_parser</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts hcluster_sg 3-column output into lists of ids</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/TGAC/earlham-galaxytools/</td>
      <td>Phylogenetics</td>
      <td>hcluster_sg_parser</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/hcluster_sg_parser</td>
    </tr>
    <tr>
      <td></td>
      <td>lotus2</td>
      <td>2.23</td>
      <td>2.28</td>
      <td>lotus2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LotuS2 OTU processing pipeline</td>
      <td></td>
      <td>To update</td>
      <td>http://lotus2.earlham.ac.uk/</td>
      <td>Metagenomics</td>
      <td>lotus2</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/lotus2</td>
    </tr>
    <tr>
      <td></td>
      <td>miranda</td>
      <td>3.3a+galaxy1</td>
      <td>3.3a</td>
      <td>miranda</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds potential target sites for miRNAs in genomic sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://www.microrna.org/</td>
      <td>RNA</td>
      <td>miranda</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/miranda</td>
    </tr>
    <tr>
      <td></td>
      <td>smart_domains</td>
      <td>0.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMART domains</td>
      <td></td>
      <td>To update</td>
      <td>http://smart.embl.de/</td>
      <td>Sequence Analysis</td>
      <td>smart_domains</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/smart_domains</td>
    </tr>
    <tr>
      <td></td>
      <td>t_coffee</td>
      <td>13.45.0.4846264</td>
      <td>13.46.0.919e8c6b</td>
      <td>t-coffee</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>T-Coffee</td>
      <td></td>
      <td>To update</td>
      <td>http://www.tcoffee.org/</td>
      <td>Sequence Analysis</td>
      <td>t_coffee</td>
      <td>earlhaminst</td>
      <td>https://github.com/TGAC/earlham-galaxytools/tree/master/tools/t_coffee</td>
    </tr>
    <tr>
      <td></td>
      <td>abacas</td>
      <td>1.1</td>
      <td>3.23</td>
      <td>mummer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and Orientate Contigs</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/abacas</td>
      <td>Assembly</td>
      <td>abacas</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/abacas</td>
    </tr>
    <tr>
      <td></td>
      <td>assemblystats</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarise an assembly (e.g. N50 metrics)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>bam2mappingstats</td>
      <td>1.1.0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates mapping stats from a bam file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>bam2mappingstats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>bamclipper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>bamclipper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Soft-clip gene-specific primers from BAM alignment file based on genomic coordinates of primer pairs in BEDPE format.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tommyau/bamclipper</td>
      <td>Sequence Analysis</td>
      <td>bamclipper</td>
      <td>nml</td>
      <td>https://github.com/tommyau/bamclipper</td>
    </tr>
    <tr>
      <td></td>
      <td>biohansel</td>
      <td>2.4.0</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/biohansel</td>
      <td>Sequence Analysis</td>
      <td>biohansel</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/biohansel</td>
    </tr>
    <tr>
      <td></td>
      <td>bundle_collections</td>
      <td>1.3.0</td>
      <td>2.54</td>
      <td>perl-getopt-long</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to bundle up list collection into a single zip to be download</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>bundle_collections</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>collapse_collection</td>
      <td>5.1.0</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collection tool that collapses a list of files into a single datasset in order of appears in collection</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>collapse_collections</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>combineJSON</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>JSON collection tool that takes multiple JSON data arrays and combines them into a single JSON array.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_json</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>combine_assembly_stats</td>
      <td>1.0</td>
      <td>2.54</td>
      <td>perl-getopt-long</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine multiple Assemblystats datasets into a single tabular report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Assembly</td>
      <td>combine_assemblystats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_tabular_collection</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine Tabular Collection into a single file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>combine_tabular_collection</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>cryptogenotyper</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>cryptogenotyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CryptoGenotyper is a standalone tool to *in-silico*  determine species and subtype based on SSU rRNA and gp60 markers.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
      <td>Sequence Analysis</td>
      <td>cryptogenotyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/CryptoGenotyper</td>
    </tr>
    <tr>
      <td></td>
      <td>ectyper</td>
      <td>1.0.0</td>
      <td>1.0.0</td>
      <td>ectyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>EC-Typer - in silico serotyping of Escherichia coli species</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
      <td>Sequence Analysis</td>
      <td>ectyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/ecoli_serotyping</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta2bed</td>
      <td>1.0.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert multiple fasta file into tabular bed file format</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fasta2bed</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_extract</td>
      <td>1.1.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>extract single fasta from multiple fasta file</td>
      <td></td>
      <td>To update</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
      <td>Sequence Analysis</td>
      <td>fasta_extract</td>
      <td>nml</td>
      <td>https://toolshed.g2.bx.psu.edu/view/nml/fasta_extract</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqc_stats</td>
      <td>1.2</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summary multiple FastQC into a single tabular line report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>fastqc_stats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>feht</td>
      <td>0.1.0</td>
      <td>1.1.0</td>
      <td>feht</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Automatically identify makers predictive of groups.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>feht</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_spades_repeats</td>
      <td>1.0.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Remove short and repeat contigs/scaffolds</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
      <td>Assembly</td>
      <td>filter_spades_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/</td>
    </tr>
    <tr>
      <td></td>
      <td>getmlst</td>
      <td>0.1.4.1</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download MLST datasets by species from pubmlst.org</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>getmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hivtrace</td>
      <td>1.0.1</td>
      <td>1.5.0</td>
      <td>hivtrace</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>An application that identifies potential transmission clusters within a supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hivtrace</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/tools/hivtrace</td>
    </tr>
    <tr>
      <td></td>
      <td>kaptive</td>
      <td>0.3.0</td>
      <td>2.0.6</td>
      <td>kaptive</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kaptive reports information about capsular (K) loci found in genome assemblies.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kaptive</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>kat_filter</td>
      <td>2.3</td>
      <td>2.4.2</td>
      <td>kat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering kmers or reads from a database of kmers hashes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>kat_filter</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mauve_contig_mover</td>
      <td>1.0.10</td>
      <td>2.4.0.r4736</td>
      <td>mauve</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order a draft genome relative to a related reference genome</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
      <td>Sequence Analysis</td>
      <td>mauve_contig_mover</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mauve_contig_mover</td>
    </tr>
    <tr>
      <td></td>
      <td>mob_suite</td>
      <td>3.0.3</td>
      <td>3.1.7</td>
      <td>mob_suite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MOB-suite is a set of software tools for clustering, reconstruction and typing of plasmids from draft assemblies</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mob-suite</td>
      <td>Sequence Analysis</td>
      <td>mob_suite</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mob-suite</td>
    </tr>
    <tr>
      <td></td>
      <td>mrbayes</td>
      <td>1.0.2</td>
      <td>3.2.7</td>
      <td>mrbayes</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A program for the Bayesian estimation of phylogeny.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>mrbayes</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe_parser</td>
      <td>0.1.4.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RScript to parse the results of mykrobe predictor.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
      <td>Sequence Analysis</td>
      <td>mykrobe_parser</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/mykrobe-parser</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>1.1.14</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Phylogenetic Assignment of Named Global Outbreak LINeages</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>nml</td>
      <td>https://github.com/hCoV-2019/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>patrist</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract Patristic Distance From a Tree</td>
      <td></td>
      <td>To update</td>
      <td>https://gist.github.com/ArtPoon/7330231e74201ded54b87142a1d6cd02</td>
      <td>Phylogenetics</td>
      <td>patrist</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/patrist</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmid_profiler_suite</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plasmid Profiler suite defining all dependencies for Plasmid Profiler</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_plasmid_profiler</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidspades</td>
      <td>1.1</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Genome assembler for assemblying plasmid</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>plasmidspades</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>promer</td>
      <td>1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aligns two sets of contigs and reports amino acid substitutions between them</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/promer</td>
      <td>Assembly</td>
      <td>promer</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/promer</td>
    </tr>
    <tr>
      <td></td>
      <td>pseudogenome</td>
      <td>1.0.0</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create a pseudogenome from a multiple fasta file either with a JCVI linker or custom length and characters.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/galaxy_tools</td>
      <td>Sequence Analysis</td>
      <td>pseudogenome</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quasitools</td>
      <td>0.7.0</td>
      <td>0.7.0</td>
      <td>quasitools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A collection of tools for analysing Viral Quasispecies</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/quasitools</td>
      <td>Sequence Analysis</td>
      <td>quasitools</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/quasitools</td>
    </tr>
    <tr>
      <td></td>
      <td>refseq_masher</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>refseq_masher</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find what genomes match or are contained within your sequence data using Mash_ and a Mash sketch database.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
      <td>Sequence Analysis</td>
      <td>refseq_masher</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/refseq_masher</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk_nml</td>
      <td>1.0.1</td>
      <td>1.4</td>
      <td>seqtk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to downsample fastq reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk_nml</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>sistr_cmd</td>
      <td>1.1.1</td>
      <td>1.1.1</td>
      <td>sistr_cmd</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SISTR in silico serotyping tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/sistr_cmd</td>
      <td>Sequence Analysis</td>
      <td>sistr_cmd</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>smalt</td>
      <td>0.7.6</td>
      <td>0.7.6</td>
      <td>smalt</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SMALT aligns DNA sequencing reads with a reference genome.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.sanger.ac.uk/science/tools/smalt-0</td>
      <td>Sequence Analysis</td>
      <td>smalt</td>
      <td>nml</td>
      <td>https://sourceforge.net/projects/smalt/</td>
    </tr>
    <tr>
      <td></td>
      <td>spatyper</td>
      <td>0.3.3</td>
      <td>0.3.3</td>
      <td>spatyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Determines SPA type based on repeats in a submitted staphylococcal protein A fasta file.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/HCGB-IGTP/spaTyper</td>
      <td>Sequence Analysis</td>
      <td>spatyper</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/spatyper</td>
    </tr>
    <tr>
      <td></td>
      <td>spolpred</td>
      <td>1.0.1</td>
      <td></td>
      <td>spolpred</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A program for predicting the spoligotype from raw sequence reads</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>spolpred</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.3.7</td>
      <td>0.2.0</td>
      <td>srst2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>srst2</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>staramr</td>
      <td>0.10.0</td>
      <td>0.10.0</td>
      <td>staramr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scan genome contigs against the ResFinder, PlasmidFinder, and PointFinder antimicrobial resistance databases.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/staramr</td>
      <td>Sequence Analysis</td>
      <td>staramr</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/galaxy_tools/tree/master/tools/staramr</td>
    </tr>
    <tr>
      <td></td>
      <td>stringmlst</td>
      <td>1.1.0</td>
      <td>0.6.3</td>
      <td>stringMLST</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid and accurate identification of the sequence type (ST)</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>stringmlst</td>
      <td>nml</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>wade</td>
      <td>0.2.5+galaxy1</td>
      <td>0.2.6</td>
      <td>wade</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>identify regions of interest</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/phac-nml/wade</td>
      <td>Sequence Analysis</td>
      <td>wade</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/wade</td>
    </tr>
    <tr>
      <td></td>
      <td>isoplot</td>
      <td>1.3.0+galaxy1</td>
      <td>1.3.1</td>
      <td>isoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isoplot is a software for the visualisation of MS data from C13 labelling experiments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metabolomics, Visualization</td>
      <td>isoplot</td>
      <td>workflow4metabolomics</td>
      <td>https://github.com/llegregam/Isoplot/tree/main</td>
    </tr>
    <tr>
      <td></td>
      <td>best_regression_subsets</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Best-subsets Regression</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>best_regression_subsets</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/best_regression_subsets</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_coverage_report</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Polymorphism of the Reads</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_coverage_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_coverage_report</td>
    </tr>
    <tr>
      <td></td>
      <td>blat_mapping</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage of the Reads in wiggle format</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Next Gen Mappers, Sequence Analysis</td>
      <td>blat_mapping</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/blat_mapping</td>
    </tr>
    <tr>
      <td></td>
      <td>cd_hit_dup</td>
      <td>0.0.1</td>
      <td>4.8.1</td>
      <td>cd-hit-auxtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>simple tool for removing duplicates from sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cd_hit_dup</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cd_hit_dup</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute Motif Frequencies For All Motifs, motif by motif.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motif_frequencies_for_all_motifs</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motif_frequencies_for_all_motifs</td>
    </tr>
    <tr>
      <td></td>
      <td>compute_motifs_frequency</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute Motif Frequencies in indel flanking regions.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Statistics</td>
      <td>compute_motifs_frequency</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/compute_motifs_frequency</td>
    </tr>
    <tr>
      <td></td>
      <td>count_gff_features</td>
      <td>0.2</td>
      <td>1.1.0</td>
      <td>galaxy-ops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count GFF Features</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>count_gff_features</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/count_gff_features</td>
    </tr>
    <tr>
      <td></td>
      <td>ctd_batch</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CTD analysis of chemicals, diseases, or genes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ctd_batch</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/ctd_batch</td>
    </tr>
    <tr>
      <td></td>
      <td>cummerbund</td>
      <td>2.16.0</td>
      <td></td>
      <td>fonts-conda-ecosystem</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Wrapper for the Bioconductor cummeRbund library</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/cummeRbund.html</td>
      <td>RNA, Visualization</td>
      <td>cummerbund</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/cummerbund</td>
    </tr>
    <tr>
      <td></td>
      <td>delete_overlapping_indels</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Delete Overlapping Indels from a chromosome indels file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>delete_overlapping_indels</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/delete_overlapping_indels</td>
    </tr>
    <tr>
      <td></td>
      <td>divide_pg_snp</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Separate pgSnp alleles into columns</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>divide_pg_snp</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/divide_pg_snp</td>
    </tr>
    <tr>
      <td></td>
      <td>express</td>
      <td>1.1.1</td>
      <td>1.5.1</td>
      <td>eXpress</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Quantify the abundances of a set of target sequences from sampled subsequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>express</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/express</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounter</td>
      <td>2.0.0</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Feature coverage</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>featurecounter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/featurecounter</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_transcripts_via_tracking</td>
      <td>0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter Combined Transcripts</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>filter_transcripts_via_tracking</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/filter_transcripts_via_tracking</td>
    </tr>
    <tr>
      <td></td>
      <td>generate_pc_lda_matrix</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate a Matrix for using PC and LDA</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>generate_pc_lda_matrix</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/generate_pc_lda_matrix</td>
    </tr>
    <tr>
      <td></td>
      <td>getindelrates_3way</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate Indel Rates for 3-way alignments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindelrates_3way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindelrates_3way</td>
    </tr>
    <tr>
      <td></td>
      <td>getindels_2way</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch Indels from pairwise alignments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>getindels_2way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/getindels_2way</td>
    </tr>
    <tr>
      <td></td>
      <td>gmaj</td>
      <td>2.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GMAJ Multiple Alignment Viewer</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>gmaj</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/gmaj</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat</td>
      <td>1.0.3</td>
      <td></td>
      <td>hisat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HISAT is a fast and sensitive spliced alignment program.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/hisat/index.shtml</td>
      <td>Assembly</td>
      <td>hisat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/hisat</td>
    </tr>
    <tr>
      <td></td>
      <td>indels_3way</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch Indels from 3-way alignments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>indels_3way</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/indels_3way</td>
    </tr>
    <tr>
      <td></td>
      <td>logistic_regression_vif</td>
      <td>1.0.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform Logistic Regression with vif</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis, Statistics</td>
      <td>logistic_regression_vif</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/logistic_regression_vif</td>
    </tr>
    <tr>
      <td></td>
      <td>maf_cpg_filter</td>
      <td>1.0.0</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mask CpG/non-CpG sites from MAF file</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>maf_cpg_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/maf_cpg_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_to_ucsc</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Format mapping data as UCSC custom track</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Convert Formats, Next Gen Mappers</td>
      <td>mapping_to_ucsc</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/mapping_to_ucsc</td>
    </tr>
    <tr>
      <td></td>
      <td>microsatellite_birthdeath</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify microsatellite births and deaths</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>microsatellite_birthdeath</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsatellite_birthdeath</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_alignment_level</td>
      <td>1.0.0</td>
      <td></td>
      <td>sputnik</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract Orthologous Microsatellites from pair-wise alignments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_alignment_level</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_alignment_level</td>
    </tr>
    <tr>
      <td></td>
      <td>microsats_mutability</td>
      <td>1.1.0</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate microsatellite mutability by specified attributes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>microsats_mutability</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/microsats_mutability</td>
    </tr>
    <tr>
      <td></td>
      <td>multispecies_orthologous_microsats</td>
      <td>1.0.0</td>
      <td></td>
      <td>bx-sputnik</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract orthologous microsatellites</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>multispecies_orthologous_microsats</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/multispecies_orthologous_microsats</td>
    </tr>
    <tr>
      <td></td>
      <td>quality_filter</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter nucleotides based on quality scores</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>quality_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/quality_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>rcve</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute RCVE</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>rcve</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/rcve</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_high_quality_length</td>
      <td>1.0.0</td>
      <td></td>
      <td>rpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Histogram of high quality score reads</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_high_quality_length</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_high_quality_length</td>
    </tr>
    <tr>
      <td></td>
      <td>short_reads_figure_score</td>
      <td>1.0.2</td>
      <td></td>
      <td>fontconfig</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Build base quality distribution</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Graphics</td>
      <td>short_reads_figure_score</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/short_reads_figure_score</td>
    </tr>
    <tr>
      <td></td>
      <td>substitution_rates</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate substitution rates for non-coding regions</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitution_rates</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitution_rates</td>
    </tr>
    <tr>
      <td></td>
      <td>substitutions</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch substitutions from pairwise alignments</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>substitutions</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/substitutions</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat</td>
      <td>1.5.0</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tophat for Illumina</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat</td>
    </tr>
    <tr>
      <td></td>
      <td>tophat2</td>
      <td>2.1.1</td>
      <td>2.5.2</td>
      <td>bowtie2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tophat - fast splice junction mapper for RNA-Seq reads</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/tophat/index.shtml</td>
      <td>RNA, Next Gen Mappers</td>
      <td>tophat2</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/tophat2</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_custom_track</td>
      <td>1.0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Build custom track for UCSC genome browser</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>ucsc_custom_track</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/main/tools/ucsc_custom_track</td>
    </tr>
    <tr>
      <td></td>
      <td>weightedaverage</td>
      <td>1.0.1</td>
      <td>1.1.0</td>
      <td>galaxy-ops</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assign weighted-average of the values of features overlapping an interval</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>weightedaverage</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/weightedaverage</td>
    </tr>
    <tr>
      <td></td>
      <td>windowsplitter</td>
      <td>1.0.1</td>
      <td>0.10.0</td>
      <td>bx-python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make windows</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>windowsplitter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tools/windowsplitter</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_fundo</td>
      <td>1.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FunDO human genes associated with disease terms</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>hgv_fundo</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_fundo</td>
    </tr>
    <tr>
      <td></td>
      <td>hgv_hilbertvis</td>
      <td>1.0.0</td>
      <td></td>
      <td>R</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HVIS visualization of genomic data with the Hilbert curve</td>
      <td></td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/huber-srv/hilbert/</td>
      <td>Graphics, Visualization</td>
      <td>hgv_hilbertvis</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/hgv/hgv_hilbertvis</td>
    </tr>
    <tr>
      <td></td>
      <td>find_diag_hits</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find diagnostic hits</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>find_diag_hits</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/find_diag_hits</td>
    </tr>
    <tr>
      <td></td>
      <td>gi2taxonomy</td>
      <td>1.1.1</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fetch taxonomic representation</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>gi2taxonomy</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/gi2taxonomy</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2tax</td>
      <td>1.2+galaxy0</td>
      <td></td>
      <td>gawk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert Kraken output to Galaxy taxonomy data.</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>kraken2tax</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/blob/master/tool_collections/taxonomy/kraken2tax/</td>
    </tr>
    <tr>
      <td></td>
      <td>lca_wrapper</td>
      <td>1.0.1</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find lowest diagnostic rank</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>lca_wrapper</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/lca_wrapper</td>
    </tr>
    <tr>
      <td></td>
      <td>poisson2test</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Poisson two-sample test</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Statistics, Metagenomics</td>
      <td>poisson2test</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/poisson2test</td>
    </tr>
    <tr>
      <td></td>
      <td>t2ps</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw phylogeny</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2ps</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2ps</td>
    </tr>
    <tr>
      <td></td>
      <td>t2t_report</td>
      <td>1.0.0</td>
      <td>0.9.0</td>
      <td>taxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarize taxonomy</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/natefoo/taxonomy</td>
      <td>Metagenomics</td>
      <td>t2t_report</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/taxonomy/t2t_report</td>
    </tr>
    <tr>
      <td></td>
      <td>AggregateAlignments</td>
      <td>0.6.0</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aggregate and filter alignment metrics of individual clusters, like the output of graphclust_align_cluster.</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_aggregate_alignments</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AggregateAlignments</td>
    </tr>
    <tr>
      <td></td>
      <td>AlignCluster</td>
      <td>0.1</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align predicted clusters of glob_report_no_align step with locarna and conservation analysis and visualizations.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_align_cluster</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tools/GraphClust/AlignCluster</td>
    </tr>
    <tr>
      <td></td>
      <td>CMFinder</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Determines consensus motives for sequences.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_cmfinder</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CMFinder</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResults</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Post-processing. Redundant clusters are merged and instances that belong to multiple clusters are assigned unambiguously. For every pair of clusters, the relative overlap (i.e. the fraction of instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%. instances that occur in both clusters) is computed and clusters are merged if the overlap exceeds 50%.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResults</td>
    </tr>
    <tr>
      <td></td>
      <td>CollectResultsNoAlign</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Redundant GraphClust clusters are merged and instances that belong to multiple clusters are assigned unambiguously.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_postprocessing_no_align</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/CollectResultsNoAlign</td>
    </tr>
    <tr>
      <td></td>
      <td>GSPAN</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Second step of GraphClust</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_fasta_to_gspan</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/GSPAN</td>
    </tr>
    <tr>
      <td></td>
      <td>LocARNAGraphClust</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MLocARNA computes a multiple sequence-structure alignment of RNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_mlocarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/LocARNAGraphClust</td>
    </tr>
    <tr>
      <td></td>
      <td>NSPDK</td>
      <td>9.2.3.1</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Produces an explicit sparse feature encoding and copmutes global feature index and returns top dense sets.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_nspdk</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/NSPDK</td>
    </tr>
    <tr>
      <td></td>
      <td>Plotting</td>
      <td>0.4</td>
      <td></td>
      <td>seaborn</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting results for GraphClust</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_motif_finder_plot</td>
      <td>rnateam</td>
      <td>https://github.com/eteriSokhoyan/galaxytools/tree/master/tools/GraphClust/Plotting</td>
    </tr>
    <tr>
      <td></td>
      <td>PrepareForMlocarna</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool prepares files for locarna step.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_prepocessing_for_mlocarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust/PrepareForMlocarna</td>
    </tr>
    <tr>
      <td></td>
      <td>Preprocessing</td>
      <td>0.5</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Preprocessing input for GraphClust</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>graphclust_preprocessing</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/GraphClust</td>
    </tr>
    <tr>
      <td></td>
      <td>Structure_GSPAN</td>
      <td>0.4</td>
      <td>0.6.0</td>
      <td>graphclust-wrappers</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert RNA structure to GSPAN graphs</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>structure_to_gspan</td>
      <td>rnateam</td>
      <td>https://github.com/mmiladi/galaxytools/blob/graphclust-gspan/tools/GraphClust/Structure_GSPAN</td>
    </tr>
    <tr>
      <td></td>
      <td>antismash</td>
      <td>6.1.1</td>
      <td>6.1.1</td>
      <td>antismash</td>
      <td>Up-to-date</td>
      <td>antismash</td>
      <td>antiSMASH</td>
      <td>Sequence clustering, Gene prediction, Differential gene expression analysis</td>
      <td>Molecular interactions, pathways and networks, Gene and protein families</td>
      <td>Antismash allows the genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters</td>
      <td>Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.</td>
      <td>Up-to-date</td>
      <td>https://antismash.secondarymetabolites.org</td>
      <td>Sequence Analysis</td>
      <td>antismash</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/antismash</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.1.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>bamhash</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>bamhash</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hash BAM and FASTQ files to verify data integrity</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/DecodeGenetics/BamHash</td>
      <td>Sequence Analysis</td>
      <td>bamhash</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bamhash</td>
    </tr>
    <tr>
      <td></td>
      <td>barcode_collapse</td>
      <td>0.1.0</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Paired End randomer aware duplicate removal algorithm</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YeoLab/gscripts</td>
      <td>RNA, Sequence Analysis</td>
      <td>barcode_collapse</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/barcode_collapse</td>
    </tr>
    <tr>
      <td></td>
      <td>bionano</td>
      <td>3.7.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bionano Solve is a set of tools for analyzing Bionano data</td>
      <td></td>
      <td>To update</td>
      <td>https://bionanogenomics.com/</td>
      <td>Assembly</td>
      <td>bionano</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bionano</td>
    </tr>
    <tr>
      <td></td>
      <td>bismark</td>
      <td>0.22.1</td>
      <td>0.24.1</td>
      <td>bismark</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool to map bisulfite converted sequence reads and determine cytosine methylation states</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioinformatics.babraham.ac.uk/projects/bismark/</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bismark</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/bismark</td>
    </tr>
    <tr>
      <td></td>
      <td>blobtoolkit</td>
      <td>4.0.7</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification and isolation non-target data in draft and publicly available genome assemblies.</td>
      <td></td>
      <td>To update</td>
      <td>https://blobtoolkit.genomehubs.org/</td>
      <td>Sequence Analysis, Assembly</td>
      <td>blobtoolkit</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blobtoolkit</td>
    </tr>
    <tr>
      <td></td>
      <td>blockbuster</td>
      <td>0.1.2</td>
      <td>0.0.1.1</td>
      <td>blockbuster</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Blockbuster detects blocks of overlapping reads using a gaussian-distribution approach.</td>
      <td></td>
      <td>To update</td>
      <td>http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html</td>
      <td>RNA, Sequence Analysis</td>
      <td>blockbuster</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/blockbuster</td>
    </tr>
    <tr>
      <td></td>
      <td>chipseeker</td>
      <td>1.32.0</td>
      <td>1.36.0</td>
      <td>bioconductor-chipseeker</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for ChIP peak annotation and visualization</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</td>
      <td>ChIP-seq, Genome annotation</td>
      <td>chipseeker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer</td>
      <td>1.1.9.0</td>
      <td>1.1.10</td>
      <td>circexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A combined strategy to identify circular RNAs (circRNAs and ciRNAs)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/YangLab/CIRCexplorer</td>
      <td>Sequence Analysis, RNA</td>
      <td>circexplorer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/circexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>combine_metaphlan_humann</td>
      <td>0.3.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combine MetaPhlAn2 and HUMAnN2 outputs to relate genus/species abundances and gene families/pathways abundances</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>combine_metaphlan2_humann2</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/combine_metaphlan2_humann2</td>
    </tr>
    <tr>
      <td></td>
      <td>compare_humann2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compare outputs of HUMAnN2 for several samples and extract similar and specific information</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>compare_humann2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/compare_humann2_output</td>
    </tr>
    <tr>
      <td></td>
      <td>crt</td>
      <td>1.2.0</td>
      <td>1.2</td>
      <td>crisper_recognition_tool</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CRISPR Recognition Tool</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>crispr_recognition_tool</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/crt</td>
    </tr>
    <tr>
      <td></td>
      <td>flye</td>
      <td>2.9.1</td>
      <td>2.9.2</td>
      <td>flye</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly of long and error-prone reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fenderglass/Flye/</td>
      <td>Assembly</td>
      <td>flye</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/flye</td>
    </tr>
    <tr>
      <td></td>
      <td>format_metaphlan2_output</td>
      <td>0.2.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Format MetaPhlAn2 output to extract abundance at different taxonomic levels</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>format_metaphlan2_output</td>
      <td>bebatut</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/format_metaphlan2_output/</td>
    </tr>
    <tr>
      <td></td>
      <td>gfastats</td>
      <td>1.3.6</td>
      <td>1.3.6</td>
      <td>gfastats</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool for generating sequence statistics and simultaneous genome assembly file manipulation.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/vgl-hub/gfastats</td>
      <td>Sequence Analysis</td>
      <td>gfastats</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gfastats</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer_hmm</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model (GHMM)</td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmerhmm/</td>
      <td>Sequence Analysis</td>
      <td>glimmer_hmm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/glimmer_hmm</td>
    </tr>
    <tr>
      <td></td>
      <td>gotohscan</td>
      <td>1.3.0</td>
      <td>1.3</td>
      <td>gotohscan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find subsequences in db</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gotohscan</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/gotohscan</td>
    </tr>
    <tr>
      <td></td>
      <td>graphclust</td>
      <td>0.1</td>
      <td></td>
      <td>GraphClust</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraphClust can be used for structural clustering of RNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/GraphClust/</td>
      <td>RNA</td>
      <td>graphclust</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphclust</td>
    </tr>
    <tr>
      <td></td>
      <td>graphmap</td>
      <td>0.5.2</td>
      <td>0.6.3</td>
      <td>graphmap</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mapper for long, error-prone reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/isovic/graphmap/</td>
      <td>Assembly</td>
      <td>graphmap</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/graphmap</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm</td>
      <td>0.19.7</td>
      <td>0.19.7</td>
      <td>hifiasm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A fast haplotype-resolved de novo assembler</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chhylp123/hifiasm</td>
      <td>Assembly</td>
      <td>hifiasm</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/hifiasm</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Software for motif discovery and next generation sequencing analysis.</td>
      <td></td>
      <td>To update</td>
      <td>http://homer.salk.edu/homer/</td>
      <td>Sequence Analysis</td>
      <td>homer</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/homer</td>
    </tr>
    <tr>
      <td></td>
      <td>illumina_methylation_analyser</td>
      <td>0.1</td>
      <td></td>
      <td>Rscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Methylation analyzer for Illumina 450k DNA emthylation microarrays</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
      <td>Sequence Analysis</td>
      <td>illumina_methylation_analyser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/illumina_methylation_analyser</td>
    </tr>
    <tr>
      <td></td>
      <td>instagraal</td>
      <td>0.1.6</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>instagraal</td>
      <td>instaGRAAL</td>
      <td>Genome assembly, Mapping assembly, Genetic mapping, Scaffolding</td>
      <td>Sequence assembly, Mapping, Metagenomics, Statistics and probability, DNA binding sites</td>
      <td>Large genome reassembly based on Hi-C data</td>
      <td>Chromosome-level quality scaffolding of brown algal genomes using InstaGRAAL.Large genome reassembly based on Hi-C data, continuation of GRAAL.Large genome reassembly based on Hi-C data (continuation and partial rewrite of GRAAL) and post-scaffolding polishing libraries.This work is under continuous development/improvement - see GRAAL for information about the basic principles.sudo pip3 install -e git+https://github.com/koszullab/instagraal.git@master#egg=instagraal.Note to OS X users: There is currently no CUDA support on Mojave (10.14) and it is unclear when it is going to be added, if it is to be added at all. This means instaGRAAL (or indeed any CUDA-based application) will not work on Mojave. If you wish to run it on OS X, the only solution for now is to downgrade to High Sierra (10.13)</td>
      <td>To update</td>
      <td>https://github.com/koszullab/instaGRAAL</td>
      <td>Assembly</td>
      <td>instagraal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/instagraal</td>
    </tr>
    <tr>
      <td></td>
      <td>iprscan5</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>iprscan5</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/iprscan5</td>
    </tr>
    <tr>
      <td></td>
      <td>itsx</td>
      <td>1.1.3</td>
      <td>1.1.3</td>
      <td>itsx</td>
      <td>Up-to-date</td>
      <td>ITSx</td>
      <td>ITSx</td>
      <td>Sequence feature detection</td>
      <td>Functional, regulatory and non-coding RNA, Microbiology</td>
      <td>ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences.</td>
      <td>TSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi. As the inclusion of parts of the neighbouring, very conserved, ribosomal genes (SSU, 5S and LSU rRNA sequences) in the sequence identification process can lead to severely misleading results, ITSx identifies and extracts only the ITS regions themselves.</td>
      <td>Up-to-date</td>
      <td>https://microbiology.se/software/itsx/</td>
      <td>Metagenomics</td>
      <td>itsx</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/itsx</td>
    </tr>
    <tr>
      <td></td>
      <td>labels</td>
      <td>1.0.5.0</td>
      <td></td>
      <td>labels</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>remaps and annotates alignments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
      <td>Sequence Analysis</td>
      <td>labels</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/labels</td>
    </tr>
    <tr>
      <td></td>
      <td>lighter</td>
      <td>1.0</td>
      <td>1.1.2</td>
      <td>lighter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Lighter is a kmer-based error correction method for whole genome sequencing data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>lighter</td>
      <td>bgruening</td>
      <td>https://github.com/mourisl/Lighter</td>
    </tr>
    <tr>
      <td></td>
      <td>mafft</td>
      <td>7.508</td>
      <td>7.520</td>
      <td>mafft</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Multiple alignment program for amino acid or nucleotide sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>mafft</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mafft</td>
    </tr>
    <tr>
      <td></td>
      <td>mcl</td>
      <td>14.137</td>
      <td>22.282</td>
      <td>mcl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Markov Cluster Algorithm</td>
      <td></td>
      <td>To update</td>
      <td>http://micans.org/mcl/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>mcl</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mcl</td>
    </tr>
    <tr>
      <td></td>
      <td>methtools</td>
      <td>0.1.1</td>
      <td></td>
      <td>methtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>tools for methylation analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
      <td>Sequence Analysis</td>
      <td>methtools</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methtools</td>
    </tr>
    <tr>
      <td></td>
      <td>methyldackel</td>
      <td>0.5.2</td>
      <td>0.6.1</td>
      <td>methyldackel</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool for processing bisulfite sequencing alignments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dpryan79/MethylDackel</td>
      <td>Sequence Analysis</td>
      <td>pileometh</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/methyldackel</td>
    </tr>
    <tr>
      <td></td>
      <td>metilene</td>
      <td>0.2.6.1</td>
      <td>0.2.8</td>
      <td>metilene</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Differential DNA methylation calling</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Statistics</td>
      <td>metilene</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/metilene</td>
    </tr>
    <tr>
      <td></td>
      <td>miclip</td>
      <td>1.2.0</td>
      <td></td>
      <td>Rscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification of binding sites in CLIP-Seq data.</td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/src/contrib/Archive/MiClip/</td>
      <td>Sequence Analysis</td>
      <td>miclip</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/miclip</td>
    </tr>
    <tr>
      <td></td>
      <td>minced</td>
      <td>0.2.0</td>
      <td>0.4.2</td>
      <td>minced</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MinCED - Mining CRISPRs in Environmental Datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://bioweb2.pasteur.fr/docs/modules/minced/0.1.5/_README</td>
      <td>Sequence Analysis</td>
      <td>minced</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minced</td>
    </tr>
    <tr>
      <td></td>
      <td>minipolish</td>
      <td>0.1.3</td>
      <td>0.1.3</td>
      <td>minipolish</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Polishing miniasm assemblies</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Minipolish</td>
      <td>Sequence Analysis</td>
      <td>minipolish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/minipolish</td>
    </tr>
    <tr>
      <td></td>
      <td>mitohifi</td>
      <td>3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly mitogenomes from Pacbio HiFi read.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marcelauliano/MitoHiFi/tree/mitohifi_v2</td>
      <td>Assembly</td>
      <td>mitohifi</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/mitohifi</td>
    </tr>
    <tr>
      <td></td>
      <td>nextdenovo</td>
      <td>2.5.0</td>
      <td>2.5.2</td>
      <td>nextdenovo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>String graph-based de novo assembler for long reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Nextomics/NextDenovo</td>
      <td>Assembly</td>
      <td>nextdenovo</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nextdenovo</td>
    </tr>
    <tr>
      <td></td>
      <td>nucleosome_prediction</td>
      <td>3.0</td>
      <td>3.0</td>
      <td>nucleosome_prediction</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Prediction of Nucleosomes Positions on the Genome</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://genie.weizmann.ac.il/software/nucleo_exe.html</td>
      <td>Sequence Analysis</td>
      <td>nucleosome_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/nucleosome_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>2.1.0</td>
      <td>3.1.0</td>
      <td>openms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OpenMS in version 2.1.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/openms</td>
    </tr>
    <tr>
      <td></td>
      <td>peakachu</td>
      <td>0.2.0+galaxy1</td>
      <td>0.2.0</td>
      <td>peakachu</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PEAKachu is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>peakachu</td>
      <td>rnateam</td>
      <td>https://github.com/tbischler/PEAKachu</td>
    </tr>
    <tr>
      <td></td>
      <td>pfamscan</td>
      <td>1.6</td>
      <td>1.6</td>
      <td>pfam_scan</td>
      <td>Up-to-date</td>
      <td>pfamscan</td>
      <td>PfamScan</td>
      <td>Protein sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Search a FASTA sequence against a library of Pfam HMM.</td>
      <td>This tool is used to search a FASTA sequence against a library of Pfam HMM.</td>
      <td>Up-to-date</td>
      <td>http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/</td>
      <td>Sequence Analysis</td>
      <td>pfamscan</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/pfamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>piranha</td>
      <td>1.2.1.0</td>
      <td>1.2.1</td>
      <td>piranha</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Piranha is a peak-caller for CLIP- and RIP-Seq data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>piranha</td>
      <td>rnateam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/piranha</td>
    </tr>
    <tr>
      <td></td>
      <td>platypus</td>
      <td>0.0.11</td>
      <td></td>
      <td>platypus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>efficient and accurate variant-detection in high-throughput sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>http://www.well.ox.ac.uk/platypus</td>
      <td>Sequence Analysis</td>
      <td>platypus</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/platypus</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_ml_performance_plots</td>
      <td>0.2</td>
      <td></td>
      <td>pandas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for machine learning problems</td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics</td>
      <td>Visualization</td>
      <td>plotly_ml_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_ml_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>parallel coordinates plot produced with plotly</td>
      <td></td>
      <td>To update</td>
      <td>https://plot.ly/python/parallel-coordinates-plot/</td>
      <td>Visualization</td>
      <td>plotly_parallel_coordinates_plot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_parallel_coordinates_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>plotly_regression_performance_plots</td>
      <td>0.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>performance plots for regression problems</td>
      <td></td>
      <td>To update</td>
      <td>http://scikit-learn.org/stable/supervised_learning.html#supervised-learning</td>
      <td>Visualization</td>
      <td>plotly_regression_performance_plots</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/plotly_regression_performance_plots</td>
    </tr>
    <tr>
      <td></td>
      <td>protease_prediction</td>
      <td>0.9</td>
      <td>2.0</td>
      <td>eden</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool can learn the cleavage specificity of a given class of proteases.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/fabriziocosta/eden</td>
      <td>Sequence Analysis, Proteomics</td>
      <td>protease_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protease_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_properties</td>
      <td>0.2.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculation of various properties from given protein sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>protein_properties</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/protein_properties</td>
    </tr>
    <tr>
      <td></td>
      <td>improviser</td>
      <td>1.1.0.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Visualisation of PepXML files</td>
      <td></td>
      <td>To update</td>
      <td>http://www.improviser.uni-freiburg.de/</td>
      <td>Proteomics</td>
      <td>proteomics_improviser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/proteomics/improviser</td>
    </tr>
    <tr>
      <td></td>
      <td>racon</td>
      <td>1.5.0</td>
      <td>1.5.0</td>
      <td>racon</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Consensus module for raw de novo DNA assembly of long uncorrected reads.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/isovic/racon</td>
      <td>Sequence Analysis</td>
      <td>racon</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/racon</td>
    </tr>
    <tr>
      <td></td>
      <td>repeat_masker</td>
      <td>0.1.2</td>
      <td>4.1.5</td>
      <td>RepeatMasker</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/repeat_masker</td>
    </tr>
    <tr>
      <td></td>
      <td>antarna</td>
      <td>1.1</td>
      <td>2.0.1.2</td>
      <td>antarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>antaRNA uses ant colony optimization to solve the inverse folding problem in RNA research .</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>antarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/antarna/</td>
    </tr>
    <tr>
      <td></td>
      <td>aresite2</td>
      <td>0.1.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AREsite2 REST Interface</td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/AREsite</td>
      <td>RNA, Data Source, Sequence Analysis</td>
      <td>aresite2</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/aresite2</td>
    </tr>
    <tr>
      <td></td>
      <td>blockclust</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>blockclust</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlockClust detects transcripts with similar processing patterns.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/workflows/blockclust</td>
      <td>RNA</td>
      <td>blockclust</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/blockclust</td>
    </tr>
    <tr>
      <td></td>
      <td>cmsearch_deoverlap</td>
      <td>0.08+galaxy0</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>removes lower scoring overlaps from cmsearch results.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EBI-Metagenomics/pipeline-v5/blob/master/tools/RNA_prediction/cmsearch-deoverlap/cmsearch-deoverlap.pl</td>
      <td>RNA</td>
      <td>cmsearch_deoverlap</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmsearch_deoverlap</td>
    </tr>
    <tr>
      <td></td>
      <td>cmv</td>
      <td>1.0.8</td>
      <td>1.0.8</td>
      <td>cmv</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cmv is a collection of tools for the visualisation of Hidden Markov Models and RNA-family models.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/cmv</td>
      <td>RNA</td>
      <td>cmv</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cmv</td>
    </tr>
    <tr>
      <td></td>
      <td>cofold</td>
      <td>2.0.4.0</td>
      <td>2.0.4</td>
      <td>cofold</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cofold predicts RNA secondary structures that takes co-transcriptional folding into account.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.e-rna.org/cofold/</td>
      <td>RNA</td>
      <td>cofold</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/cofold</td>
    </tr>
    <tr>
      <td></td>
      <td>compalignp</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>compalignp</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute fractional identity between trusted alignment and test alignment</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>RNA, Sequence Analysis</td>
      <td>compalignp</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/compalignp/</td>
    </tr>
    <tr>
      <td></td>
      <td>coprarna</td>
      <td>2.1.1</td>
      <td>2.1.4</td>
      <td>coprarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Target prediction for prokaryotic trans-acting small RNAs</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
      <td>RNA, Sequence Analysis</td>
      <td>coprarna</td>
      <td>rnateam</td>
      <td>https://github.com/PatrickRWright/CopraRNA</td>
    </tr>
    <tr>
      <td></td>
      <td>dewseq</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DEWSeq is a sliding window based peak caller for eCLIP/iCLIP data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/DEWSeq_analysis_helpers</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>dewseq</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dewseq</td>
    </tr>
    <tr>
      <td></td>
      <td>dorina</td>
      <td>1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>data source for RNA interactions in post-transcriptional regulation</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA, Data Source</td>
      <td>dorina</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dorina/</td>
    </tr>
    <tr>
      <td></td>
      <td>dot2ct</td>
      <td>5.7.a</td>
      <td>6.3</td>
      <td>rnastructure</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Dot-Bracket to Connect Table (CT)</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, RNA</td>
      <td>dot2ct</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/dot2ct</td>
    </tr>
    <tr>
      <td></td>
      <td>dotknot</td>
      <td>1.3.1</td>
      <td></td>
      <td>vienna_rna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DotKnot is a heuristic method for pseudoknot prediction in a given RNA sequence</td>
      <td></td>
      <td>To update</td>
      <td>http://dotknot.csse.uwa.edu.au/</td>
      <td>RNA, Proteomics</td>
      <td>dotknot</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna/dotknot</td>
    </tr>
    <tr>
      <td></td>
      <td>exparna</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>exparna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ExpaRNA is a fast, motif-based comparison and alignment tool for RNA molecules.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://rna.informatik.uni-freiburg.de/ExpaRNA/Input.jsp</td>
      <td>RNA</td>
      <td>exparna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/exparna</td>
    </tr>
    <tr>
      <td></td>
      <td>graphprot</td>
      <td>1.1.7+galaxy1</td>
      <td>1.1.7</td>
      <td>graphprot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraphProt models binding preferences of RNA-binding proteins.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dmaticzka/GraphProt</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>graphprot</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/graphprot</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq-clip</td>
      <td>0.1.0+galaxy0</td>
      <td>2.19.0b0</td>
      <td>htseq-clip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EMBL-Hentze-group/htseq-clip</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>htseq_clip</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip</td>
    </tr>
    <tr>
      <td></td>
      <td>infernal</td>
      <td>1.1.4</td>
      <td>1.1.4</td>
      <td>infernal</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Infernal ("INFERence of RNA ALignment") is for searching DNA sequence databases for RNA structure and sequence similarities.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://infernal.janelia.org/</td>
      <td>RNA</td>
      <td>infernal</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/infernal</td>
    </tr>
    <tr>
      <td></td>
      <td>inforna</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.</td>
      <td></td>
      <td>To update</td>
      <td>http://rna.informatik.uni-freiburg.de/INFORNA/Input.jsp</td>
      <td>RNA</td>
      <td>inforna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/inforna</td>
    </tr>
    <tr>
      <td></td>
      <td>intarna</td>
      <td>3.3.2</td>
      <td>3.3.2</td>
      <td>intarna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient RNA-RNA interaction prediction incorporating accessibility and seeding of interaction sites.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BackofenLab/IntaRNA</td>
      <td>RNA</td>
      <td>intarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/intarna</td>
    </tr>
    <tr>
      <td></td>
      <td>kinwalker</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kinwalker splits the folding process into a series of events where each event can either be a folding event or a transcription event.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/Kinwalker/</td>
      <td>RNA</td>
      <td>kinwalker</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/kinwalker</td>
    </tr>
    <tr>
      <td></td>
      <td>locarna</td>
      <td>1.9.2.3</td>
      <td>2.0.0</td>
      <td>locarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>LocARNA - A suite for multiple alignment and folding of RNAs</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-freiburg.de/Software/LocARNA/</td>
      <td>RNA</td>
      <td>locarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/locarna</td>
    </tr>
    <tr>
      <td></td>
      <td>mea</td>
      <td>0.6.4.1</td>
      <td>0.6.4</td>
      <td>mea</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maximum expected accuracy prediction</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinf.uni-leipzig.de/Software/mea</td>
      <td>RNA</td>
      <td>mea</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mea</td>
    </tr>
    <tr>
      <td></td>
      <td>mqc</td>
      <td>1.9</td>
      <td>1.10</td>
      <td>mqc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ribosome profiling mapping quality control tool</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Biobix/mQC</td>
      <td>Sequence Analysis</td>
      <td>mqc</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/mqc/</td>
    </tr>
    <tr>
      <td></td>
      <td>nastiseq</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>r-nastiseq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to identify cis-NATs using ssRNA-seq</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/</td>
      <td>RNA</td>
      <td>nastiseq</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/nastiseq</td>
    </tr>
    <tr>
      <td></td>
      <td>paralyzer</td>
      <td>1.5</td>
      <td>1.5</td>
      <td>paralyzer</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to generate a high resolution map of interaction sites between RNA-binding proteins and their targets.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/</td>
      <td>RNA</td>
      <td>paralyzer</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/paralyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>pipmir</td>
      <td>0.1.0</td>
      <td>1.1</td>
      <td>pipmir</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A method to identify novel plant miRNA.</td>
      <td></td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/</td>
      <td>RNA</td>
      <td>pipmir</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/pipmir</td>
    </tr>
    <tr>
      <td></td>
      <td>rRNA</td>
      <td>0.1</td>
      <td></td>
      <td>hmmsearch3.0</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification of ribosomal RNA genes in metagenomic fragments.</td>
      <td></td>
      <td>To update</td>
      <td>http://weizhong-lab.ucsd.edu/meta_rna/</td>
      <td>RNA</td>
      <td>rrna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rRNA</td>
    </tr>
    <tr>
      <td></td>
      <td>rcas</td>
      <td>1.5.4</td>
      <td>1.26.0</td>
      <td>bioconductor-rcas</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RCAS (RNA Centric Annotation System) for functional analysis of transcriptome-wide regions detected by high-throughput experiments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BIMSBbioinfo/RCAS</td>
      <td>RNA</td>
      <td>rcas</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rcas/</td>
    </tr>
    <tr>
      <td></td>
      <td>reago</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>reago</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reago is tool to assembly 16S ribosomal RNA recovery from metagenomic data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chengyuan/reago-1.1</td>
      <td>Metagenomics, RNA</td>
      <td>reago</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/reago</td>
    </tr>
    <tr>
      <td></td>
      <td>remurna</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>remurna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>remuRNA - Measurement of Single Nucleotide Polymorphism induced Changes of RNA Conformation</td>
      <td></td>
      <td>To update</td>
      <td>https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#remurna</td>
      <td>RNA</td>
      <td>remurna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/remurna</td>
    </tr>
    <tr>
      <td></td>
      <td>ribotaper</td>
      <td>1.3.1a</td>
      <td>1.3.1</td>
      <td>ribotaper</td>
      <td>To update</td>
      <td>ribotaper</td>
      <td>RiboTaper</td>
      <td>Gene expression profiling</td>
      <td>Functional genomics</td>
      <td>A method for defining traslated ORFs using Ribosome Profiling data.</td>
      <td>New analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.</td>
      <td>To update</td>
      <td>https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/</td>
      <td>RNA</td>
      <td>ribotaper</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/ribotaper/</td>
    </tr>
    <tr>
      <td></td>
      <td>rna_shapes</td>
      <td>3.3.0</td>
      <td></td>
      <td>@EXECUTABLE@</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute secondary structures of RNA</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnashapes</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rna_shapes</td>
    </tr>
    <tr>
      <td></td>
      <td>rnabob</td>
      <td>2.2.1.0</td>
      <td>2.2.1</td>
      <td>rnabob</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast pattern searching for RNA structural motifs</td>
      <td></td>
      <td>To update</td>
      <td>http://eddylab.org/software.html</td>
      <td>RNA</td>
      <td>rnabob</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnabob</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacode</td>
      <td>0.3.2</td>
      <td>0.3</td>
      <td>rnacode</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyze the protein coding potential in MSA</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>RNA</td>
      <td>rnacode</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnacode</td>
    </tr>
    <tr>
      <td></td>
      <td>rnacommender</td>
      <td>0.1.1</td>
      <td>3.5</td>
      <td>sam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAcommender is a tool for genome-wide recommendation of RNA-protein interactions.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gianlucacorrado/RNAcommender</td>
      <td>RNA</td>
      <td>rnacommender</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/rna_commander/tools/rna_tools/rna_commender</td>
    </tr>
    <tr>
      <td></td>
      <td>rnalien</td>
      <td>1.3.6</td>
      <td>1.8.0</td>
      <td>rnalien</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAlien unsupervized RNA family model construction</td>
      <td></td>
      <td>To update</td>
      <td>http://rna.tbi.univie.ac.at/rnalien/</td>
      <td>RNA, Sequence Analysis</td>
      <td>rnalien</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/rnalien</td>
    </tr>
    <tr>
      <td></td>
      <td>rnasnp</td>
      <td>1.2.0</td>
      <td>1.2</td>
      <td>rnasnp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAsnp Efficient detection of local RNA secondary structure changes induced by SNPs</td>
      <td></td>
      <td>To update</td>
      <td>http://rth.dk/resources/rnasnp/</td>
      <td>RNA</td>
      <td>rnasnp</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rnasnp</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaz</td>
      <td>2.1.1</td>
      <td>2.1.1</td>
      <td>rnaz</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RNAz is a program for predicting structurally conserved and thermodynamically stable RNA secondary structures in multiple sequence alignments.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.tbi.univie.ac.at/~wash/RNAz/</td>
      <td>RNA</td>
      <td>rnaz</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_team/rnaz</td>
    </tr>
    <tr>
      <td></td>
      <td>selectsequencesfrommsa</td>
      <td>1.0.5</td>
      <td>1.0.5</td>
      <td>selectsequencesfrommsa</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SelectSequences - selects representative entries from a multiple sequence alignment in clustal format</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/eggzilla/SelectSequences</td>
      <td>RNA, Sequence Analysis</td>
      <td>selectsequencesfrommsa</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/selectsequencesfrommsa</td>
    </tr>
    <tr>
      <td></td>
      <td>sortmerna</td>
      <td>4.3.6</td>
      <td>4.3.6</td>
      <td>sortmerna</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://bioinfo.lifl.fr/RNA/sortmerna/</td>
      <td>RNA</td>
      <td>sortmerna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sortmerna</td>
    </tr>
    <tr>
      <td></td>
      <td>sshmm</td>
      <td>1.0.7</td>
      <td>1.0.7</td>
      <td>sshmm</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ssHMM is an RNA sequence-structure motif finder for RNA-binding protein data, such as CLIP-Seq data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.molgen.mpg.de/heller/ssHMM</td>
      <td>Sequence Analysis, RNA</td>
      <td>sshmm</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/sshmm/</td>
    </tr>
    <tr>
      <td></td>
      <td>targetfinder</td>
      <td>1.7</td>
      <td>1.7</td>
      <td>targetfinder</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plant small RNA target prediction tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/carringtonlab/TargetFinder.git</td>
      <td>RNA</td>
      <td>targetfinder</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/targetfinder/</td>
    </tr>
    <tr>
      <td></td>
      <td>trna_prediction</td>
      <td>0.6</td>
      <td>1.2.41</td>
      <td>aragorn</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Aragorn predicts tRNA and tmRNA in nucleotide sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://mbioserv2.mbioekol.lu.se/ARAGORN/</td>
      <td>RNA</td>
      <td>trna_prediction</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/trna_prediction</td>
    </tr>
    <tr>
      <td></td>
      <td>vienna_rna</td>
      <td>2.2.10</td>
      <td>2.6.4</td>
      <td>viennarna</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ViennaRNA - Prediction and comparison of RNA secondary structures</td>
      <td></td>
      <td>To update</td>
      <td>http://www.tbi.univie.ac.at/RNA/</td>
      <td>RNA</td>
      <td>viennarna</td>
      <td>rnateam</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/vienna_rna</td>
    </tr>
    <tr>
      <td></td>
      <td>sailfish</td>
      <td>0.10.1.1</td>
      <td></td>
      <td>bzip2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sailfish is a tool for transcript quantification from RNA-seq data</td>
      <td></td>
      <td>To update</td>
      <td>http://www.cs.cmu.edu/~ckingsf/software/sailfish/</td>
      <td>Sequence Analysis, RNA</td>
      <td>sailfish</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/sailfish</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon</td>
      <td>1.10.1</td>
      <td>1.10.2</td>
      <td>salmon</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq and single-cell data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/COMBINE-lab/salmon</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/salmon</td>
    </tr>
    <tr>
      <td></td>
      <td>tgsgapcloser</td>
      <td>1.0.3</td>
      <td>1.2.1</td>
      <td>tgsgapcloser</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TGS-GapCloser uses error-prone long reads or preassembled contigs to fill N-gap in the genome assembly.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BGI-Qingdao/TGS-GapCloser</td>
      <td>Assembly</td>
      <td>tgsgapcloser</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/tgsgapcloser</td>
    </tr>
    <tr>
      <td></td>
      <td>trim_galore</td>
      <td>0.6.7</td>
      <td>0.6.10</td>
      <td>trim-galore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim Galore adaptive quality and adapter trimmer</td>
      <td></td>
      <td>To update</td>
      <td>http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</td>
      <td>Sequence Analysis, Fastq Manipulation</td>
      <td>trim_galore</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/trim_galore</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprot_rest_interface</td>
      <td>0.4</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UniProt ID mapping and sequence retrieval</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jdrudolph/uniprot</td>
      <td>Proteomics, Sequence Analysis</td>
      <td>uniprot_rest_interface</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_rest_interface</td>
    </tr>
    <tr>
      <td></td>
      <td>vt</td>
      <td>0.2</td>
      <td>2015.11.10</td>
      <td>vt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A tool set for short variant discovery in genetic sequence data.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>vt</td>
      <td>bgruening</td>
      <td>https://github.com/atks/vt</td>
    </tr>
    <tr>
      <td></td>
      <td>wtdbg</td>
      <td>2.5</td>
      <td>2.5</td>
      <td>wtdbg</td>
      <td>Up-to-date</td>
      <td>wtdbg2</td>
      <td>wtdbg2</td>
      <td>Genome assembly, De-novo assembly, Mapping assembly, k-mer counting</td>
      <td>Sequence assembly, Computer science, Informatics</td>
      <td>WTDBG is a fuzzy Bruijn graph (FBG) approach to long noisy reads assembly.</td>
      <td>Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.</td>
      <td>Up-to-date</td>
      <td>https://github.com/ruanjue/wtdbg2</td>
      <td>Assembly</td>
      <td>wtdbg</td>
      <td>bgruening</td>
      <td>https://github.com/bgruening/galaxytools/tree/master/tools/wtdbg</td>
    </tr>
    <tr>
      <td></td>
      <td>align_back_trans</td>
      <td>0.0.10</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Thread nucleotides onto a protein alignment (back-translation)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>align_back_trans</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/align_back_trans</td>
    </tr>
    <tr>
      <td></td>
      <td>chromosome_diagram</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Chromosome Diagrams using Biopython</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>chromosome_diagram</td>
      <td>peterjc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>clc_assembly_cell</td>
      <td>0.0.7</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrapper for the CLC Assembly Cell suite from CLCBio</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>clc_assembly_cell</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clc_assembly_cell</td>
    </tr>
    <tr>
      <td></td>
      <td>clinod</td>
      <td>0.1.0</td>
      <td>1.3</td>
      <td>clinod</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td></td>
      <td>To update</td>
      <td>http://www.compbio.dundee.ac.uk/www-nod/</td>
      <td>Sequence Analysis</td>
      <td>clinod</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/clinod</td>
    </tr>
    <tr>
      <td></td>
      <td>count_roi_variants</td>
      <td>0.0.6</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Count sequence variants in region of interest in BAM file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
      <td>Assembly, SAM</td>
      <td>count_roi_variants</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/count_roi_variants</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_stats</td>
      <td>0.1.0</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAM coverage statistics using samtools idxstats and depth</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
      <td>Assembly, SAM</td>
      <td>coverage_stats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/coverage_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>effectiveT3</td>
      <td>0.0.20</td>
      <td>1.0.1</td>
      <td>effectiveT3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find bacterial type III effectors in protein sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://effectors.org</td>
      <td>Sequence Analysis</td>
      <td>effectivet3</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/effectiveT3</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_filter_by_id</td>
      <td>0.0.6</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTA sequences by ID (DEPRECATED)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fasta_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fasta_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_filter_by_id</td>
      <td>0.0.6</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTQ sequences by ID (DEPRECATED)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
      <td>Fastq Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>fastq_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_pair_names</td>
      <td>0.0.5</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract FASTQ paired read names</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
      <td>Sequence Analysis</td>
      <td>fastq_pair_names</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_pair_names</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_paired_unpaired</td>
      <td>0.1.4</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Divide FASTQ file into paired and unpaired reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
      <td>Sequence Analysis, Text Manipulation</td>
      <td>fastq_paired_unpaired</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/fastq_paired_unpaired</td>
    </tr>
    <tr>
      <td></td>
      <td>get_orfs_or_cdss</td>
      <td>0.2.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Search nucleotide sequences for open reading frames (ORFs), or coding sequences (CDSs)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
      <td>Sequence Analysis</td>
      <td>get_orfs_or_cdss</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer</td>
      <td>0.0.8</td>
      <td>9.18</td>
      <td>ghostscript</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw dotplots using mummer, mucmer, or promer with mummerplot</td>
      <td></td>
      <td>To update</td>
      <td>http://mummer.sourceforge.net/</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>mummer</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/mummer</td>
    </tr>
    <tr>
      <td></td>
      <td>nlstradamus</td>
      <td>0.0.11</td>
      <td>1.8</td>
      <td>NLStradamus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find nuclear localization signals (NLSs) in protein sequences</td>
      <td></td>
      <td>To update</td>
      <td>http://www.moseslab.csb.utoronto.ca/NLStradamus</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>nlstradamus</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/nlstradamus</td>
    </tr>
    <tr>
      <td></td>
      <td>predictnls</td>
      <td>0.0.9</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Python reimplementation of predictNLS for Galaxy</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
      <td>Sequence Analysis</td>
      <td>predictnls</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/predictnls</td>
    </tr>
    <tr>
      <td></td>
      <td>protein_analysis</td>
      <td>0.0.13</td>
      <td></td>
      <td>promoter</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TMHMM, SignalP, Promoter, RXLR motifs, WoLF PSORT and PSORTb</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
      <td>Sequence Analysis</td>
      <td>tmhmm_and_signalp</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/protein_analysis</td>
    </tr>
    <tr>
      <td></td>
      <td>sample_seqs</td>
      <td>0.2.5</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sub-sample sequences files (e.g. to reduce coverage)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>sample_seqs</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/sample_seqs</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depad</td>
      <td>0.0.5</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Re-align a SAM/BAM file with a padded reference (using samtools depad)</td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, SAM, Sequence Analysis</td>
      <td>samtools_depad</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depad</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_depth</td>
      <td>0.0.3</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Coverage depth via samtools</td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Sequence Analysis, SAM</td>
      <td>samtools_depth</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_depth</td>
    </tr>
    <tr>
      <td></td>
      <td>samtools_idxstats</td>
      <td>0.0.6</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAM mapping statistics (using samtools idxstats)</td>
      <td></td>
      <td>To update</td>
      <td>http://www.htslib.org/</td>
      <td>Assembly, Next Gen Mappers, SAM</td>
      <td>samtools_idxstats</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/samtools_idxstats</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_composition</td>
      <td>0.0.5</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence composition</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
      <td>Sequence Analysis</td>
      <td>seq_composition</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_composition</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_id</td>
      <td>0.2.7</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequences by ID</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_filter_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_filter_by_mapping</td>
      <td>0.0.7</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter sequencing reads using SAM/BAM mapping files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
      <td>Assembly, Fasta Manipulation, Fastq Manipulation, SAM, Sequence Analysis</td>
      <td>seq_filter_by_mapping</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_length</td>
      <td>0.0.4</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute sequence length (from FASTA, QUAL, FASTQ, SFF, etc)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>seq_length</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_length</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_primer_clip</td>
      <td>0.0.16</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trim off 5' or 3' primers</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
      <td>Assembly, Fasta Manipulation, Text Manipulation</td>
      <td>seq_primer_clip</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_primer_clip</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_rename</td>
      <td>0.0.9</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rename sequences with ID mapping from a tabular file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_rename</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_rename</td>
    </tr>
    <tr>
      <td></td>
      <td>seq_select_by_id</td>
      <td>0.0.14</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Select sequences by ID</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
      <td>Fasta Manipulation, Sequence Analysis, Text Manipulation</td>
      <td>seq_select_by_id</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_select_by_id</td>
    </tr>
    <tr>
      <td></td>
      <td>venn_list</td>
      <td>0.1.1</td>
      <td>1.1.5</td>
      <td>galaxy_sequence_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Draw Venn Diagram (PDF) from lists, FASTA files, etc</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
      <td>Graphics, Sequence Analysis, Visualization</td>
      <td>venn_list</td>
      <td>peterjc</td>
      <td>https://github.com/peterjc/pico_galaxy/tree/master/tools/venn_list</td>
    </tr>
    <tr>
      <td></td>
      <td>TrimNs</td>
      <td>0.1.0</td>
      <td>1.0</td>
      <td>trimns_vgp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TrimNs is used to trim and remove fake cut sites from bionano hybrid scaffold data in the VGP pipeline</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VGP/vgp-assembly/tree/master/pipeline/bionano/trimNs</td>
      <td>Assembly</td>
      <td>trimns</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trimN</td>
    </tr>
    <tr>
      <td></td>
      <td>abricate</td>
      <td>1.0.1</td>
      <td>1.0.1</td>
      <td>abricate</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mass screening of contigs for antiobiotic resistance genes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/abricate</td>
      <td>Sequence Analysis</td>
      <td>abricate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abricate/</td>
    </tr>
    <tr>
      <td></td>
      <td>abritamr</td>
      <td>1.0.14</td>
      <td>1.0.14</td>
      <td>abritamr</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for running AMRfinderPlus and collating results into functional classes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://zenodo.org/record/7370628</td>
      <td>Sequence Analysis</td>
      <td>abritamr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abritamr</td>
    </tr>
    <tr>
      <td></td>
      <td>abyss</td>
      <td>2.3.6</td>
      <td>2.3.7</td>
      <td>abyss</td>
      <td>To update</td>
      <td>abyss</td>
      <td>ABySS</td>
      <td>Genome assembly, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler</td>
      <td>De novo genome sequence assembler.</td>
      <td>To update</td>
      <td>http://www.bcgsc.ca/platform/bioinfo/software/abyss</td>
      <td>Assembly</td>
      <td>abyss</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/abyss</td>
    </tr>
    <tr>
      <td></td>
      <td>adapter_removal</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>adapterremoval</td>
      <td>Up-to-date</td>
      <td>adapterremoval</td>
      <td>AdapterRemoval</td>
      <td>Sequence trimming, Sequence merging, Primer removal</td>
      <td></td>
      <td>Removes residual adapter sequences from single-end (SE) or paired-end (PE) FASTQ reads.</td>
      <td>AdapterRemoval searches for and removes adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. AdapterRemoval can analyze both single end and paired end data, and can be used to merge overlapping paired-ended reads into (longer) consensus sequences. Additionally, AdapterRemoval can construct a consensus adapter sequence for paired-ended reads, if which this information is not available.</td>
      <td>Up-to-date</td>
      <td>https://github.com/MikkelSchubert/adapterremoval</td>
      <td>Fasta Manipulation, Sequence Analysis</td>
      <td>adapter_removal</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/adapter_removal/</td>
    </tr>
    <tr>
      <td></td>
      <td>aegean</td>
      <td>0.16.0</td>
      <td>0.16.0</td>
      <td>aegean</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>AEGeAn toolkit wrappers</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BrendelGroup/AEGeAn</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>aegean</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aegean</td>
    </tr>
    <tr>
      <td></td>
      <td>aldex2</td>
      <td>1.26.0</td>
      <td>1.32.0</td>
      <td>bioconductor-aldex2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs analysis Of differential abundance taking sample variation into account</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ggloor/ALDEx_bioc</td>
      <td>Metagenomics</td>
      <td>aldex2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/aldex2</td>
    </tr>
    <tr>
      <td></td>
      <td>amplican</td>
      <td>1.14.0</td>
      <td>1.22.1</td>
      <td>bioconductor-amplican</td>
      <td>To update</td>
      <td>amplican</td>
      <td>amplican</td>
      <td>Alignment, Standardisation and normalisation</td>
      <td>PCR experiment, Statistics and probability</td>
      <td>AmpliCan is an analysis tool for genome editing.</td>
      <td>It performs alignment of the amplicon reads, normalizes gathered data, calculates multiple statistics (e.g. cut rates, frameshifts) and presents results in form of aggregated reports. Data and statistics can be broken down by experiments, barcodes, user defined groups, guides and amplicons allowing for quick identification of potential problems.</td>
      <td>To update</td>
      <td>https://github.com/valenlab/amplican</td>
      <td>Sequence Analysis</td>
      <td>amplican</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/amplican</td>
    </tr>
    <tr>
      <td></td>
      <td>ampvis2</td>
      <td>2.7.22</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ampvis2</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MadsAlbertsen/ampvis2/</td>
      <td>Metagenomics</td>
      <td>ampvis2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ampvis2</td>
    </tr>
    <tr>
      <td></td>
      <td>amrfinderplus</td>
      <td>3.11.26</td>
      <td>3.11.26</td>
      <td>ncbi-amrfinderplus</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"AMRFinderPlus is designed to find acquired antimicrobial resistance genes and point mutations in protein and/or assembled nucleotide sequences.It can also search "plus", stress, heat, and biocide resistance and virulence factors for some organisms.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ncbi/amr</td>
      <td>Sequence Analysis</td>
      <td>AMRFinderPlus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/amrfinderplus</td>
    </tr>
    <tr>
      <td></td>
      <td>ancombc</td>
      <td>1.4.0</td>
      <td>2.2.0</td>
      <td>bioconductor-ancombc</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs analysis of compositions of microbiomes with bias correction.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/FrederickHuangLin/ANCOMBC</td>
      <td>Metagenomics</td>
      <td>ancombc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ancombc</td>
    </tr>
    <tr>
      <td></td>
      <td>anndata</td>
      <td>0.7.5</td>
      <td>0.6.22</td>
      <td>anndata</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Import, Export, Inspect and Manipulate Anndata and Loom objects</td>
      <td></td>
      <td>To update</td>
      <td>https://anndata.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>anndata</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/anndata/</td>
    </tr>
    <tr>
      <td></td>
      <td>annotatemyids</td>
      <td>3.17.0</td>
      <td>3.17.0</td>
      <td>bioconductor-org.hs.eg.db</td>
      <td>Up-to-date</td>
      <td>annotatemyids</td>
      <td>annotatemyids</td>
      <td>Annotation</td>
      <td></td>
      <td>annotateMyIDs: get annotation for a set of IDs using the Bioconductor annotation packages</td>
      <td>This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.</td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
      <td>Genome annotation</td>
      <td>annotatemyids</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/annotatemyids</td>
    </tr>
    <tr>
      <td></td>
      <td>arriba</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>arriba</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Arriba detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/suhrig/arriba</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>arriba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/arriba</td>
    </tr>
    <tr>
      <td></td>
      <td>art</td>
      <td>2014.11.03.0</td>
      <td>2016.06.05</td>
      <td>art</td>
      <td>To update</td>
      <td>art</td>
      <td>ART</td>
      <td>Conversion</td>
      <td>Bioinformatics</td>
      <td>Simulator for Illumina, 454, and SOLiD sequencing data</td>
      <td>ART is a set of simulation tools to generate synthetic next-generation sequencing reads. ART simulates sequencing reads by mimicking real sequencing process with empirical error models or quality profiles summarized from large recalibrated sequencing data. ART can also simulate reads using user own read error model or quality profiles. ART supports simulation of single-end, paired-end/mate-pair reads of three major commercial next-generation sequencing platforms. Illuminas Solexa, Roches 454 and Applied Biosystems SOLiD</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Data Source</td>
      <td>art</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/art</td>
    </tr>
    <tr>
      <td></td>
      <td>artic</td>
      <td></td>
      <td>1.2.3</td>
      <td>artic</td>
      <td>To update</td>
      <td>artic</td>
      <td>ARTIC</td>
      <td>Sequence alignment</td>
      <td>Genomics</td>
      <td>The artic pipeline is designed to help run the artic bioinformatics protocols;for example the nCoV-2019 novel coronavirus protocol.Features include: read filtering, primer trimming, amplicon coverage normalisation,variant calling and consensus building</td>
      <td>A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore</td>
      <td>To update</td>
      <td>https://github.com/artic-network/fieldbioinformatics</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/artic</td>
    </tr>
    <tr>
      <td></td>
      <td>assembly-stats</td>
      <td>17.02</td>
      <td>17.02</td>
      <td>rjchallis-assembly-stats</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assembly metric visualisations to facilitate rapid assessment and comparison of assembly quality.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rjchallis/assembly-stats</td>
      <td>Assembly</td>
      <td>assembly_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/assembly-stats</td>
    </tr>
    <tr>
      <td></td>
      <td>augustus</td>
      <td>3.4.0</td>
      <td>3.5.0</td>
      <td>augustus</td>
      <td>To update</td>
      <td>augustus</td>
      <td>AUGUSTUS</td>
      <td>Gene prediction, Ab-initio gene prediction, Homology-based gene prediction, Homology-based gene prediction, Operation</td>
      <td>Gene transcripts, Gene and protein families</td>
      <td>AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
      <td>AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.</td>
      <td>To update</td>
      <td>http://bioinf.uni-greifswald.de/augustus/</td>
      <td>Sequence Analysis</td>
      <td>augustus</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/augustus</td>
    </tr>
    <tr>
      <td></td>
      <td>b2btools</td>
      <td>3.0.5+galaxy0</td>
      <td>3.0.6</td>
      <td>b2btools</td>
      <td>To update</td>
      <td>b2btools</td>
      <td>b2bTools</td>
      <td>Protein disorder prediction, Protein secondary structure prediction, Protein feature detection</td>
      <td></td>
      <td>This software suite provides structural predictions for protein sequences made by Bio2Byte group.About Bio2Byte: We investigate how the dynamics, conformational states, and available experimental data of proteins relate to their amino acid sequence.Underlying physical and chemical principles are computationally unraveled through data integration, analysis, and machine learning, so connecting themto biological events and improving our understanding of the way proteins work.</td>
      <td>The bio2byte tools server (b2btools) offers the following single protein sequence based predictions:- Backbone and sidechain dynamics (DynaMine)- Helix, sheet, coil and polyproline-II propensity- Early folding propensity (EFoldMine)- Disorder (DisoMine)- Beta-sheet aggregation (Agmata)In addition, multiple sequence alignments (MSAs) can be uploaded to scan the 'biophysical limits' of a protein family as defined in the MSA</td>
      <td>To update</td>
      <td>https://bio2byte.be</td>
      <td>Computational chemistry, Molecular Dynamics, Proteomics, Sequence Analysis, Synthetic Biology</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bakta</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>bakta</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"Bakta is a tool for the rapid &amp; standardized annotation of bacterial genomes and plasmids from both isolates and MAGs.It provides dbxref-rich and sORF-including annotations in machine-readable JSON &amp; bioinformatics standard file formats for automatic downstream analysis."</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/oschwengers/bakta</td>
      <td>Sequence Analysis</td>
      <td>bakta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/bakta</td>
    </tr>
    <tr>
      <td></td>
      <td>bamutil</td>
      <td></td>
      <td>1.0.15</td>
      <td>bamutil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bamUtil is a repository that contains several programs that perform operations on SAM/BAM files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/statgen/bamUtil</td>
      <td>Sequence Analysis</td>
      <td>bamutil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bamutil</td>
    </tr>
    <tr>
      <td></td>
      <td>bandage</td>
      <td>2022.09</td>
      <td>2022.09</td>
      <td>bandage_ng</td>
      <td>Up-to-date</td>
      <td>bandage</td>
      <td>Bandage</td>
      <td>Sequence assembly visualisation</td>
      <td>Genomics, Sequence assembly</td>
      <td>Bandage - A Bioinformatics Application for Navigating De novo Assembly Graphs Easily</td>
      <td>GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Bandage</td>
      <td>Visualization</td>
      <td>bandage</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bandage</td>
    </tr>
    <tr>
      <td></td>
      <td>baredsc</td>
      <td>1.1.2</td>
      <td>1.1.2</td>
      <td>baredsc</td>
      <td>Up-to-date</td>
      <td>baredsc</td>
      <td>baredSC</td>
      <td>Data retrieval, Expression correlation analysis, Differential gene expression profiling</td>
      <td>RNA-Seq, Cytometry, Transcriptomics, Gene transcripts, Statistics and probability</td>
      <td>baredSC is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>The baredSC (Bayesian Approach to Retreive Expression Distribution of Single Cell) is a tool that uses a Monte-Carlo Markov Chain to estimate a confidence interval on the probability density function (PDF) of expression of one or two genes from single-cell RNA-seq data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/lldelisle/baredSC</td>
      <td>Transcriptomics, Visualization</td>
      <td>baredsc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/baredsc</td>
    </tr>
    <tr>
      <td></td>
      <td>barrnap</td>
      <td>1.2.2</td>
      <td>0.9</td>
      <td>barrnap</td>
      <td>To update</td>
      <td>barrnap</td>
      <td>Barrnap</td>
      <td>Gene prediction</td>
      <td>Genomics, Model organisms, Model organisms</td>
      <td>Contains the Barrnap tool for finding ribosomal RNAs in FASTA sequences.</td>
      <td>Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>barrnap</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bax2bam</td>
      <td>0.0.11</td>
      <td>0.0.11</td>
      <td>bax2bam</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BAX to BAM converter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pacificbiosciences/bax2bam/</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>bax2bam</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pax2bam</td>
    </tr>
    <tr>
      <td></td>
      <td>bayescan</td>
      <td>2.1</td>
      <td>2.0.1</td>
      <td>bayescan</td>
      <td>To update</td>
      <td>bayescan</td>
      <td>BayeScan</td>
      <td>Statistical inference</td>
      <td>Genetics, Evolutionary biology, Statistics and probability, DNA polymorphism</td>
      <td>Detecting natural selection from population-based genetic data</td>
      <td>BAYEsian genome SCAN for outliers, aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations. It is based on the multinomial-Dirichlet model.</td>
      <td>To update</td>
      <td>http://cmpg.unibe.ch/software/BayeScan/index.html</td>
      <td>Sequence Analysis</td>
      <td>bayescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bayescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>bbtools</td>
      <td>39.01</td>
      <td>39.01</td>
      <td>bbmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BBTools is a suite of fast, multithreaded bioinformatics tools designed for analysis of DNA and RNA sequence data.BBTools can handle common sequencing file formats such as fastq, fasta, sam, scarf, fasta+qual, compressed or raw,with autodetection of quality encoding and interleaving. It is written in Java and works on any platform supportingJava, including Linux, MacOS, and Microsoft Windows and Linux; there are no dependencies other than Java (version7 or higher). Program descriptions and options are shown when running the shell scripts with no parameters.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://jgi.doe.gov/data-and-tools/bbtools/</td>
      <td>Sequence Analysis</td>
      <td>bbtools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bbtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bctools</td>
      <td>0.2.2</td>
      <td>0.2.2</td>
      <td>bctools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bctools is a set of tools for handling barcodes and UMIs in NGS data.bctools can be used to merge PCR duplicates according to unique molecular barcodes (UMIs),to extract barcodes from arbitrary positions relative to the read starts,to clean up readthroughs into UMIs with paired-end sequencing andhandles binary barcodes as used with uvCLAP and FLASH.License: Apache License 2.0</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dmaticzka/bctools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bedtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bellerophon</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>bellerophon</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter mapped reads where the mapping spans a junction, retaining the 5-prime read.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/bellerophon</td>
      <td>Sequence Analysis</td>
      <td>bellerophon</td>
      <td>iuc</td>
      <td>https://github.com/davebx/bellerophon</td>
    </tr>
    <tr>
      <td></td>
      <td>binning_refiner</td>
      <td>1.4.3</td>
      <td>1.4.3</td>
      <td>binning_refiner</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reconciles the outputs of different binning programs with the aim to improve the quality of genome bins,especially with respect to contamination levels.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/songweizhi/Binning_refiner</td>
      <td>Metagenomics</td>
      <td>binning_refiner</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/binning_refiner/</td>
    </tr>
    <tr>
      <td></td>
      <td>bioinformatics_cafe</td>
      <td>0.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miscellanea of scripts for bioinformatics</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/dariober/bioinformatics-cafe/</td>
      <td>Sequence Analysis</td>
      <td>bioinformatics_cafe</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/bioinformatics-cafe</td>
    </tr>
    <tr>
      <td></td>
      <td>biom_format</td>
      <td>2.1.14</td>
      <td>2.1.7</td>
      <td>biom-format</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The biom-format package provides a command line interface and Python API for working with BIOM files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/biocore/biom-format</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/biom_format</td>
    </tr>
    <tr>
      <td></td>
      <td>bioperl</td>
      <td>1.1</td>
      <td>1.7.8</td>
      <td>perl-bioperl</td>
      <td>To update</td>
      <td>bioperl</td>
      <td>BioPerl</td>
      <td>Data handling, Service invocation</td>
      <td>Genomics, Software engineering, Data management</td>
      <td>Converts GenBank format files to GFF3</td>
      <td>A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.</td>
      <td>To update</td>
      <td>https://bioperl.org/</td>
      <td>Sequence Analysis</td>
      <td>bp_genbank2gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bioperl</td>
    </tr>
    <tr>
      <td></td>
      <td>biscot</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>biscot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bionano scaffolding correction tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/biscot</td>
      <td>Assembly</td>
      <td>biscot</td>
      <td>iuc</td>
      <td>https://github.com/bgruening/iuc/tree/master/tools/biscot</td>
    </tr>
    <tr>
      <td></td>
      <td>blastxml_to_gapped_gff3</td>
      <td>1.1</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>BlastXML to gapped GFF3</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>blastxml_to_gapped_gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/blastxml_to_gapped_gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>bracken</td>
      <td>2.8</td>
      <td>2.9</td>
      <td>bracken</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bayesian Reestimation of Abundance with KrakEN</td>
      <td></td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/bracken/</td>
      <td>Sequence Analysis, Metagenomics</td>
      <td>bracken</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bracken</td>
    </tr>
    <tr>
      <td></td>
      <td>busco</td>
      <td>5.4.6</td>
      <td>5.5.0</td>
      <td>busco</td>
      <td>To update</td>
      <td>busco</td>
      <td>BUSCO</td>
      <td>Sequence assembly validation, Scaffolding, Genome assembly, Transcriptome assembly</td>
      <td>Sequence assembly, Genomics, Transcriptomics, Sequence analysis</td>
      <td>BUSCO assess genome and annotation completeness</td>
      <td>Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.</td>
      <td>To update</td>
      <td>https://gitlab.com/ezlab/busco/-/releases</td>
      <td>Sequence Analysis</td>
      <td>busco</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bwameth</td>
      <td>0.2.6</td>
      <td>0.2.7</td>
      <td>bwameth</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast and accurate alignment of BS-seq reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/brentp/bwa-meth</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>bwameth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/bwameth</td>
    </tr>
    <tr>
      <td></td>
      <td>calculate_contrast_threshold</td>
      <td>1.0.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculates a contrast threshold from the CDT file generated by ``tag_pileup_frequency``. The calculated values are then used to set a uniform contrast for all the heatmaps generated downstream.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/CEGRcode/ChIP-QC-tools/tree/master/calculate_contrast_threshold</td>
      <td>Visualization, Genomic Interval Operations, SAM</td>
      <td>calculate_contrast_threshold</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/calculate_contrast_threshold</td>
    </tr>
    <tr>
      <td></td>
      <td>cat</td>
      <td>5.2.3</td>
      <td>5.2.3</td>
      <td>cat</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contig Annotation Tool (CAT)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dutilh/CAT</td>
      <td>Metagenomics</td>
      <td>contig_annotation_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cat</td>
    </tr>
    <tr>
      <td></td>
      <td>cdhit</td>
      <td>4.8.1</td>
      <td>4.8.1</td>
      <td>cd-hit</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Cluster or compare biological sequence datasets</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://weizhongli-lab.org/cd-hit/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>cd_hit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cdhit</td>
    </tr>
    <tr>
      <td></td>
      <td>cemitool</td>
      <td>1.18.1</td>
      <td>1.24.0</td>
      <td>bioconductor-cemitool</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene co-expression network analysis tool</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>cemitool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cemitool</td>
    </tr>
    <tr>
      <td></td>
      <td>charts</td>
      <td>1.0.1</td>
      <td></td>
      <td>r-getopt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enables advanced visualization options in Galaxy Charts</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization</td>
      <td>charts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/charts/</td>
    </tr>
    <tr>
      <td></td>
      <td>checkm</td>
      <td>1.2.0</td>
      <td>1.2.2</td>
      <td>checkm-genome</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Ecogenomics/CheckM</td>
      <td>Metagenomics</td>
      <td>checkm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/checkm</td>
    </tr>
    <tr>
      <td></td>
      <td>cherri</td>
      <td>0.7</td>
      <td>0.8</td>
      <td>cherri</td>
      <td>To update</td>
      <td>cherri</td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computational Help Evaluating RNA-RNA interactions</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BackofenLab/Cherri</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cherri</td>
    </tr>
    <tr>
      <td></td>
      <td>chira</td>
      <td>1.4.3</td>
      <td>1.4.3</td>
      <td>chira</td>
      <td>Up-to-date</td>
      <td>chira</td>
      <td>ChiRA</td>
      <td></td>
      <td>RNA, Molecular interactions, pathways and networks, Functional, regulatory and non-coding RNA</td>
      <td>Chimeric Read Annotator for RNA-RNA interactome data</td>
      <td>ChiRA is a tool suite to analyze RNA-RNA interactome experimental data such as CLASH, CLEAR-CLIP, PARIS, SPLASH, etc.</td>
      <td>Up-to-date</td>
      <td>https://github.com/pavanvidem/chira</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>chira</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chira</td>
    </tr>
    <tr>
      <td></td>
      <td>chromeister</td>
      <td>1.5.a</td>
      <td>1.5.a</td>
      <td>chromeister</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ultra-fast pairwise genome comparisons</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/estebanpw/chromeister</td>
      <td>Sequence Analysis</td>
      <td>chromeister</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/chromeister</td>
    </tr>
    <tr>
      <td></td>
      <td>circexplorer2</td>
      <td>2.3.8</td>
      <td>2.3.8</td>
      <td>circexplorer2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Comprehensive and integrative circular RNA analysis toolset.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/YangLab/CIRCexplorer2</td>
      <td>RNA, Assembly</td>
      <td>circexplorer2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/circexplorer2</td>
    </tr>
    <tr>
      <td></td>
      <td>clair3</td>
      <td>0.1.12</td>
      <td>1.0.4</td>
      <td>clair3</td>
      <td>To update</td>
      <td>clair3</td>
      <td>Clair3</td>
      <td>Variant calling</td>
      <td>Molecular genetics</td>
      <td>Symphonizing pileup and full-alignment for high-performance long-read variant calling</td>
      <td>Clair3 is a germline small variant caller for long-reads. Clair3 makes the best of two major method categories: pileup calling handles most variant candidates with speed, and full-alignment tackles complicated candidates to maximize precision and recall. Clair3 runs fast and has superior performance, especially at lower coverage. Clair3 is simple and modular for easy deployment and integration.</td>
      <td>To update</td>
      <td>https://github.com/HKU-BAL/Clair3</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>clair3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clair3</td>
    </tr>
    <tr>
      <td></td>
      <td>clustalw</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>clustalw</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ClustalW multiple sequence alignment program for DNA or proteins</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.clustal.org/clustal2/</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>clustalw</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/clustalw</td>
    </tr>
    <tr>
      <td></td>
      <td>codeml</td>
      <td>4.9</td>
      <td>4.10.6</td>
      <td>paml</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Detects positive selection</td>
      <td></td>
      <td>To update</td>
      <td>http://abacus.gene.ucl.ac.uk/software/paml.html</td>
      <td>Phylogenetics</td>
      <td>codeml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/codeml</td>
    </tr>
    <tr>
      <td></td>
      <td>cojac</td>
      <td>0.9.1</td>
      <td>0.9.1</td>
      <td>cojac</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>co-occurrence of mutations on amplicons</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/cojac</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>cojac</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cojac</td>
    </tr>
    <tr>
      <td></td>
      <td>colibread</td>
      <td>24.7.14+galaxy0</td>
      <td>24.7.14</td>
      <td>commet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Colib'read tools are all dedicated to the analysis of NGS datasets without the need of any reference genome</td>
      <td></td>
      <td>To update</td>
      <td>https://colibread.inria.fr/</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>colibread</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/colibread</td>
    </tr>
    <tr>
      <td></td>
      <td>concoct</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>concoct</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CONCOCT (Clustering cONtigs with COverage and ComposiTion) does unsupervised binning of metagenomic contigs byusing nucleotide composition - kmer frequencies - and coverage data for multiple samples. CONCOCT can accurately(up to species level) bin metagenomic contigs.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BinPro/CONCOCT</td>
      <td>Metagenomics</td>
      <td>concoct</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/concoct</td>
    </tr>
    <tr>
      <td></td>
      <td>coverage_report</td>
      <td>0.0.4</td>
      <td>1.76</td>
      <td>perl-number-format</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate Detailed Coverage Report from BAM file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
      <td>Sequence Analysis</td>
      <td>coverage_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/coverage_report</td>
    </tr>
    <tr>
      <td></td>
      <td>coverm</td>
      <td>0.6.1</td>
      <td>0.6.1</td>
      <td>coverm</td>
      <td>Up-to-date</td>
      <td>coverm</td>
      <td>CoverM</td>
      <td>Local alignment</td>
      <td>Bioinformatics</td>
      <td>CoverM genome and contig wrappers</td>
      <td>Read coverage calculator for metagenomics</td>
      <td>Up-to-date</td>
      <td>https://github.com/wwood/CoverM</td>
      <td>Sequence Analysis</td>
      <td>coverm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tools/coverm</td>
    </tr>
    <tr>
      <td></td>
      <td>crispr_studio</td>
      <td>1+galaxy0</td>
      <td>1</td>
      <td>crispr_studio</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CRISPR Studio is a program developed to facilitate and accelerate CRISPR array visualization.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/moineaulab/CRISPRStudio</td>
      <td>Sequence Analysis</td>
      <td>crispr_studio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crispr_studio/</td>
    </tr>
    <tr>
      <td></td>
      <td>crosscontamination_barcode_filter</td>
      <td>0.3</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Barcode contamination discovery tool</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Visualization</td>
      <td>crosscontamination_barcode_filter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/crosscontamination_barcode_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>cutadapt</td>
      <td>4.4</td>
      <td>4.5</td>
      <td>cutadapt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Flexible tool to remove adapter sequences (and quality trim) high throughput sequencing reads (fasta/fastq).</td>
      <td></td>
      <td>To update</td>
      <td>https://cutadapt.readthedocs.org/en/stable/</td>
      <td>Fasta Manipulation, Fastq Manipulation, Sequence Analysis</td>
      <td>cutadapt</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/cutadapt</td>
    </tr>
    <tr>
      <td></td>
      <td>dada2</td>
      <td></td>
      <td>1.28.0</td>
      <td>bioconductor-dada2</td>
      <td>To update</td>
      <td>dada2</td>
      <td>dada2</td>
      <td>Variant calling, DNA barcoding</td>
      <td>Sequencing, Genetic variation, Microbial ecology, Metagenomics</td>
      <td>DADA2 wrappers</td>
      <td>This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.</td>
      <td>To update</td>
      <td>https://benjjneb.github.io/dada2/index.html</td>
      <td>Metagenomics</td>
      <td>dada2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dada2</td>
    </tr>
    <tr>
      <td></td>
      <td>das_tool</td>
      <td>1.1.6</td>
      <td>1.1.6</td>
      <td>das_tool</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DAS Tool for genome resolved metagenomics</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cmks/DAS_Tool</td>
      <td>Metagenomics</td>
      <td>das_tool</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/das_tool</td>
    </tr>
    <tr>
      <td></td>
      <td>deepsig</td>
      <td>1.2.5</td>
      <td>1.2.5</td>
      <td>deepsig</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predictor of signal peptides in proteins based on deep learning</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BolognaBiocomp/deepsig</td>
      <td>Genome annotation</td>
      <td>deepsig</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deepsig</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2</td>
      <td>2.11.40.8</td>
      <td>1.40.2</td>
      <td>bioconductor-deseq2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Differential gene expression analysis based on the negative binomial distribution</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>deseq2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/deseq2</td>
    </tr>
    <tr>
      <td></td>
      <td>dexseq</td>
      <td>1.44</td>
      <td>1.46.0</td>
      <td>bioconductor-dexseq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Inference of differential exon usage in RNA-Seq</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>dexseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dexseq</td>
    </tr>
    <tr>
      <td></td>
      <td>diamond</td>
      <td>2.0.15</td>
      <td>2.1.8</td>
      <td>diamond</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIAMOND is a new alignment tool for aligning short DNA sequencing reads to a protein reference database such as NCBI-NR.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/bbuchfink/diamond</td>
      <td>Sequence Analysis</td>
      <td>diamond</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diamond</td>
    </tr>
    <tr>
      <td></td>
      <td>disco</td>
      <td></td>
      <td>1.2</td>
      <td>disco</td>
      <td>To update</td>
      <td>disco</td>
      <td>DISCO</td>
      <td>Protein sequence analysis</td>
      <td>Structure determination</td>
      <td>DISCO is a overlap-layout-consensus (OLC) metagenome assembler</td>
      <td>DISCO is software to perform structure determination of protein homo-oligomers with cyclic symmetry.DISCO computes oligomeric protein structures using geometric constraints derived from RDCs and intermolecular distance restraints such as NOEs or disulfide bonds. When a reliable subunit structure can be calculated from intramolecular restraints, DISCO guarantees that all satisfying oligomer structures will be discovered, yet can run in minutes to hours on only a single desktop-class computer.</td>
      <td>To update</td>
      <td>http://disco.omicsbio.org/</td>
      <td>Metagenomics, Assembly</td>
      <td>disco</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/disco/</td>
    </tr>
    <tr>
      <td></td>
      <td>dram</td>
      <td>1.3.5</td>
      <td>1.4.6</td>
      <td>dram</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DRAM for distilling microbial metabolism to automate the curation of microbiome function</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/WrightonLabCSU/DRAM</td>
      <td>Metagenomics</td>
      <td>dram</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dram</td>
    </tr>
    <tr>
      <td></td>
      <td>drep</td>
      <td>3.4.3</td>
      <td>3.4.5</td>
      <td>drep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>dRep compares and dereplicates genome sets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MrOlm/drep</td>
      <td>Metagenomics</td>
      <td>drep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/drep</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.10.0</td>
      <td>1.20.0</td>
      <td>bioconductor-dropletutils</td>
      <td>To update</td>
      <td>dropletutils</td>
      <td>DropletUtils</td>
      <td>Loading, Community profiling</td>
      <td>Gene expression, RNA-seq, Sequencing, Transcriptomics</td>
      <td>DropletUtils - Utilities for handling droplet-based single-cell RNA-seq data</td>
      <td>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/DropletUtils.html</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>dropletutils</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dropletutils/</td>
    </tr>
    <tr>
      <td></td>
      <td>edger</td>
      <td>3.36.0</td>
      <td>3.42.4</td>
      <td>bioconductor-edger</td>
      <td>To update</td>
      <td>edger</td>
      <td>edgeR</td>
      <td>Differential gene expression analysis</td>
      <td>Genetics, RNA-Seq, ChIP-seq</td>
      <td>Perform RNA-Seq differential expression analysis using edgeR pipeline</td>
      <td>Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/edgeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>edger</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/edger</td>
    </tr>
    <tr>
      <td></td>
      <td>egsea</td>
      <td>1.20.0</td>
      <td>1.28.0</td>
      <td>bioconductor-egsea</td>
      <td>To update</td>
      <td>egsea</td>
      <td>EGSEA</td>
      <td>Gene set testing</td>
      <td>Systems biology</td>
      <td>This tool implements the Ensemble of Gene Set Enrichment Analyses (EGSEA) method for gene set testing</td>
      <td>This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/EGSEA.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>egsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/egsea</td>
    </tr>
    <tr>
      <td></td>
      <td>emboss_5</td>
      <td>5.0.0</td>
      <td>6.6.0</td>
      <td>emboss</td>
      <td>To update</td>
      <td>emboss</td>
      <td>EMBOSS</td>
      <td>Sequence analysis, Local alignment, Sequence alignment analysis, Global alignment, Sequence alignment</td>
      <td>Molecular biology, Sequence analysis, Biology</td>
      <td>Galaxy wrappers for EMBOSS version 5.0.0 tools</td>
      <td>Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.</td>
      <td>To update</td>
      <td>http://emboss.open-bio.org/</td>
      <td>Sequence Analysis, Fasta Manipulation</td>
      <td>emboss_5</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/emboss_5</td>
    </tr>
    <tr>
      <td></td>
      <td>exomedepth</td>
      <td>1.1.0</td>
      <td>1.1.16</td>
      <td>r-exomedepth</td>
      <td>To update</td>
      <td>exomedepth</td>
      <td>ExomeDepth</td>
      <td>Sequence analysis, Variant calling, Genotyping, Copy number estimation</td>
      <td>Exome sequencing, Gene transcripts, Mapping, Sequencing, Genetic variation, Rare diseases</td>
      <td>ExomeDepth: Calls copy number variants (CNVs) from targeted sequence data</td>
      <td>Copy number variant (CNV) calling algorithm designed to control technical variability between samples. It calls CNVs from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.</td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=ExomeDepth</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td>exomedepth</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exomedepth</td>
    </tr>
    <tr>
      <td></td>
      <td>exonerate</td>
      <td>2.4.0</td>
      <td>2.4.0</td>
      <td>exonerate</td>
      <td>Up-to-date</td>
      <td>exonerate</td>
      <td>Exonerate</td>
      <td>Pairwise sequence alignment, Protein threading, Genome alignment</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks</td>
      <td>Exonerate is a generic tool for pairwise sequence comparison.</td>
      <td>A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.</td>
      <td>Up-to-date</td>
      <td>https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate</td>
      <td>Sequence Analysis</td>
      <td>exonerate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/exonerate</td>
    </tr>
    <tr>
      <td></td>
      <td>export2graphlan</td>
      <td>0.20</td>
      <td>0.22</td>
      <td>export2graphlan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/CibioCM/export2graphlan/overview</td>
      <td>Metagenomics</td>
      <td>export2graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/export2graphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>fargene</td>
      <td>0.1</td>
      <td>0.1</td>
      <td>fargene</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/fannyhb/fargene</td>
      <td>Sequence Analysis</td>
      <td>fargene</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fargene</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_nucleotide_color_plot</td>
      <td>1.0.1</td>
      <td></td>
      <td>openjdk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Contains a tool that produces a graphical representation of FASTA data with each nucleotide represented by a selected color.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/seqcode/cegr-tools/tree/master/src/org/seqcode/cegrtools/fourcolorplot</td>
      <td>Visualization</td>
      <td>fasta_nucleotide_color_plot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_nucleotide_color_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>fasta_stats</td>
      <td>2.0</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Display summary statistics for a fasta file.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
      <td>Sequence Analysis</td>
      <td>fasta_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasta_stats/</td>
    </tr>
    <tr>
      <td></td>
      <td>fastani</td>
      <td>1.3</td>
      <td>1.34</td>
      <td>fastani</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast alignment-free computation of whole-genome Average Nucleotide Identity</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ParBLiSS/FastANI</td>
      <td>Sequence Analysis</td>
      <td>fastani</td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>fastp</td>
      <td></td>
      <td>0.23.4</td>
      <td>fastp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast all-in-one preprocessing for FASTQ files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/OpenGene/fastp</td>
      <td>Sequence Analysis</td>
      <td>fastp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastp</td>
    </tr>
    <tr>
      <td></td>
      <td>fastqe</td>
      <td>0.2.6+galaxy2</td>
      <td>0.3.1</td>
      <td>fastqe</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FASTQE</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
      <td>Sequence Analysis</td>
      <td>fastqe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fastqe</td>
    </tr>
    <tr>
      <td></td>
      <td>fasttree</td>
      <td>2.1.10</td>
      <td>2.1.11</td>
      <td>fasttree</td>
      <td>To update</td>
      <td>fasttree</td>
      <td>FastTree</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences - GVL</td>
      <td>Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.</td>
      <td>To update</td>
      <td>http://www.microbesonline.org/fasttree/</td>
      <td>Phylogenetics</td>
      <td>fasttree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fasttree</td>
    </tr>
    <tr>
      <td></td>
      <td>featurecounts</td>
      <td>2.0.3</td>
      <td>2.0.6</td>
      <td>subread</td>
      <td>To update</td>
      <td>featurecounts</td>
      <td>FeatureCounts</td>
      <td>Read summarisation</td>
      <td>Sequencing</td>
      <td>featureCounts counts the number of reads aligned to defined masked regions in a reference genome</td>
      <td>featureCounts is a very efficient read quantifier. It can be used to summarize RNA-seq reads and gDNA-seq reads to a variety of genomic features such as genes, exons, promoters, gene bodies and genomic bins. It is included in the Bioconductor Rsubread package and also in the SourceForge Subread package.</td>
      <td>To update</td>
      <td>http://bioinf.wehi.edu.au/featureCounts</td>
      <td>RNA, Transcriptomics, Sequence Analysis</td>
      <td>featurecounts</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/featurecounts</td>
    </tr>
    <tr>
      <td></td>
      <td>feelnc</td>
      <td>0.2.1</td>
      <td>0.2</td>
      <td>feelnc</td>
      <td>To update</td>
      <td>feelnc</td>
      <td>FEELnc</td>
      <td>Annotation, Classification</td>
      <td>RNA-seq, Functional, regulatory and non-coding RNA</td>
      <td>Galaxy wrapper for FEELnc</td>
      <td>A tool to annotate long non-coding RNAs from RNA-seq assembled transcripts.</td>
      <td>To update</td>
      <td>https://github.com/tderrien/FEELnc</td>
      <td>Sequence Analysis</td>
      <td>feelnc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/feelnc</td>
    </tr>
    <tr>
      <td></td>
      <td>fermikit</td>
      <td>r193</td>
      <td>r193</td>
      <td>fermi2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FermiKit is a de novo assembly based variant calling pipeline for deep Illumina resequencing data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/fermikit</td>
      <td>Assembly, Variant Analysis</td>
      <td>fermikit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fermikit</td>
    </tr>
    <tr>
      <td></td>
      <td>fgsea</td>
      <td>1.8.0+galaxy1</td>
      <td>1.26.0</td>
      <td>bioconductor-fgsea</td>
      <td>To update</td>
      <td>fgsea</td>
      <td>fgsea</td>
      <td>Gene-set enrichment analysis</td>
      <td>Genetics</td>
      <td>Perform gene set testing using fgsea</td>
      <td>The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/fgsea.html</td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>fgsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fgsea</td>
    </tr>
    <tr>
      <td></td>
      <td>filtlong</td>
      <td>0.2.1</td>
      <td>0.2.1</td>
      <td>filtlong</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtlong - Filtering long reads by quality</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Filtlong</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>filtlong</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/filtlong</td>
    </tr>
    <tr>
      <td></td>
      <td>flair</td>
      <td>1.5</td>
      <td>2.0.0</td>
      <td>flair</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FLAIR (Full-Length Alternative Isoform analysis of RNA) for the correction, isoform definition, and alternative splicing analysis of noisy reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/BrooksLabUCSC/flair</td>
      <td>Nanopore</td>
      <td>flair</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flair</td>
    </tr>
    <tr>
      <td></td>
      <td>flash</td>
      <td>1.2.11</td>
      <td>1.2.11</td>
      <td>flash</td>
      <td>Up-to-date</td>
      <td>flash</td>
      <td>FLASH</td>
      <td>Read pre-processing, Sequence merging, Sequence assembly</td>
      <td>Sequencing, Sequence assembly</td>
      <td>Fast Length Adjustment of SHort reads</td>
      <td>Identifies paired-end reads which overlap in the middle, converting them to single long reads</td>
      <td>Up-to-date</td>
      <td>https://ccb.jhu.edu/software/FLASH/</td>
      <td>Assembly, Fastq Manipulation</td>
      <td>flash</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/flash</td>
    </tr>
    <tr>
      <td></td>
      <td>fraggenescan</td>
      <td></td>
      <td>1.31</td>
      <td>fraggenescan</td>
      <td>To update</td>
      <td>fraggenescan</td>
      <td>FragGeneScan</td>
      <td>Gene prediction</td>
      <td>Genetics, Sequence analysis</td>
      <td>Tool for finding (fragmented) genes in short read</td>
      <td>Application for finding (fragmented) genes in short reads</td>
      <td>To update</td>
      <td>https://sourceforge.net/projects/fraggenescan/</td>
      <td>Sequence Analysis</td>
      <td>fraggenescan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/fraggenescan/</td>
    </tr>
    <tr>
      <td></td>
      <td>freyja</td>
      <td>1.4.4</td>
      <td>1.4.7</td>
      <td>freyja</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>lineage abundances estimation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/andersen-lab/Freyja</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>freyja</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/freyja</td>
    </tr>
    <tr>
      <td></td>
      <td>funannotate</td>
      <td>1.8.15</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Funannotate is a genome prediction, annotation, and comparison software package.</td>
      <td></td>
      <td>To update</td>
      <td>https://funannotate.readthedocs.io</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/funannotate</td>
    </tr>
    <tr>
      <td></td>
      <td>gecko</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>gecko</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Ungapped genome comparison</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/otorreno/gecko</td>
      <td>Sequence Analysis</td>
      <td>gecko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gecko</td>
    </tr>
    <tr>
      <td></td>
      <td>gemini</td>
      <td>0.20.1</td>
      <td>0.30.2</td>
      <td>gemini</td>
      <td>To update</td>
      <td>gemini</td>
      <td>GEMINI</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>GEMINI: a flexible framework for exploring genome variation</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td>To update</td>
      <td>https://github.com/arq5x/gemini</td>
      <td>Sequence Analysis, Next Gen Mappers</td>
      <td>gemini</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gemini</td>
    </tr>
    <tr>
      <td></td>
      <td>geneiobio</td>
      <td>4.7.1+galaxy1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene.iobio is an interactive tool for variant and trio analysis.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/iobio/gene.iobio</td>
      <td>Sequence Analysis</td>
      <td>geneiobio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/geneiobio</td>
    </tr>
    <tr>
      <td></td>
      <td>genomic_super_signature</td>
      <td>1.2.0</td>
      <td>1.8.0</td>
      <td>bioconductor-genomicsupersignature</td>
      <td>To update</td>
      <td>genomicsupersignature</td>
      <td>GenomicSuperSignature</td>
      <td>Gene-set enrichment analysis, Essential dynamics, Deposition, Principal component visualisation, Dimensionality reduction</td>
      <td>RNA-Seq, Transcriptomics, Microbial ecology, Genotype and phenotype, Microarray experiment</td>
      <td>Interpretation of RNAseq experiments through robust, efficient comparison to public databases</td>
      <td>GenomicSuperSignature is a package for the interpretation of RNA-seq experiments through robust, efficient comparison to public databases.</td>
      <td>To update</td>
      <td>https://github.com/shbrief/GenomicSuperSignature</td>
      <td>Sequence Analysis, RNA, Transcriptomics</td>
      <td>genomic_super_signature</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/genomic_super_signature</td>
    </tr>
    <tr>
      <td></td>
      <td>getorganelle</td>
      <td>1.7.7.0</td>
      <td>1.7.7.0</td>
      <td>getorganelle</td>
      <td>Up-to-date</td>
      <td>getorganelle</td>
      <td>GetOrganelle</td>
      <td>De-novo assembly, Genome assembly, Mapping assembly, Mapping, Sequence trimming</td>
      <td>Cell biology, Sequence assembly, Whole genome sequencing, Plant biology, Model organisms</td>
      <td>GetOrganelle - This toolkit assembles organelle genomes from genomic skimming data.</td>
      <td>A fast and versatile toolkit for accurate de novo assembly of organelle genomes.This toolkit assemblies organelle genome from genomic skimming data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/Kinggerm/GetOrganelle</td>
      <td>Assembly</td>
      <td>getorganelle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/getorganelle</td>
    </tr>
    <tr>
      <td></td>
      <td>gff3_rebase</td>
      <td>1.2</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rebase a GFF against a parent GFF (e.g. an original genome)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
      <td>Sequence Analysis</td>
      <td>gff3_rebase</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gff3_rebase</td>
    </tr>
    <tr>
      <td></td>
      <td>gffread</td>
      <td>0.12.7</td>
      <td>0.12.7</td>
      <td>gffread</td>
      <td>Up-to-date</td>
      <td>gffread</td>
      <td>gffread</td>
      <td>Sequence annotation</td>
      <td>Nucleic acids, Sequence analysis</td>
      <td>gffread filters and/or converts GFF3/GTF2 records</td>
      <td>program for filtering, converting and manipulating GFF files</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread/</td>
      <td>Sequence Analysis</td>
      <td>gffread</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gffread</td>
    </tr>
    <tr>
      <td></td>
      <td>ggplot2</td>
      <td>3.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>ggplot2</td>
      <td>ggplot2</td>
      <td>Visualisation</td>
      <td>Data visualisation</td>
      <td>ggplot2 is a system for declaratively creating graphics, based on The Grammar of Graphics.You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use,and it takes care of the details.</td>
      <td>Plotting system for R, based on the grammar of graphics.</td>
      <td>To update</td>
      <td>https://github.com/tidyverse/ggplot2</td>
      <td>Visualization</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ggplot2</td>
    </tr>
    <tr>
      <td></td>
      <td>glimmer</td>
      <td></td>
      <td>3.02</td>
      <td>glimmer</td>
      <td>To update</td>
      <td>gemini</td>
      <td>GEMINI</td>
      <td>Sequence analysis, Genetic variation analysis</td>
      <td>Sequence analysis</td>
      <td>Glimmer makes gene predictions.</td>
      <td>GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.</td>
      <td>To update</td>
      <td>https://ccb.jhu.edu/software/glimmer/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/glimmer</td>
    </tr>
    <tr>
      <td></td>
      <td>goenrichment</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>goenrichment</td>
      <td>Up-to-date</td>
      <td>goenrichment</td>
      <td>GOEnrichment</td>
      <td>Gene-set enrichment analysis</td>
      <td>Transcriptomics</td>
      <td>Performs GO Enrichment analysis.</td>
      <td>GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.GOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.The tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.It was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.</td>
      <td>Up-to-date</td>
      <td>https://github.com/DanFaria/GOEnrichment</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goenrichment</td>
    </tr>
    <tr>
      <td></td>
      <td>goseq</td>
      <td>1.50.0</td>
      <td>1.52.0</td>
      <td>bioconductor-goseq</td>
      <td>To update</td>
      <td>goseq</td>
      <td>GOseq</td>
      <td>Gene functional annotation</td>
      <td>RNA-seq</td>
      <td>goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data</td>
      <td>Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/goseq.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>goseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/goseq</td>
    </tr>
    <tr>
      <td></td>
      <td>graphlan</td>
      <td></td>
      <td>1.1.3</td>
      <td>graphlan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/nsegata/graphlan/overview</td>
      <td>Metagenomics, Graphics, Phylogenetics</td>
      <td>graphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann2/</td>
    </tr>
    <tr>
      <td></td>
      <td>gtdbtk</td>
      <td>2.2.2</td>
      <td>2.3.2</td>
      <td>gtdbtk</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GTDB-Tk is a software tool kit for assigning objective taxonomic classifications to bacterial and archaeal genomesbased on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds orthousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can alsobe applied to isolate and single-cell genomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Ecogenomics/GTDBTk</td>
      <td>Metagenomics</td>
      <td>gtdbtk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gtdbtk</td>
    </tr>
    <tr>
      <td></td>
      <td>gubbins</td>
      <td>3.2.1</td>
      <td>3.3.1</td>
      <td>gubbins</td>
      <td>To update</td>
      <td>gubbins</td>
      <td>Gubbins</td>
      <td>Genotyping, Phylogenetic inference, Ancestral reconstruction</td>
      <td>Phylogeny, Genotype and phenotype, Whole genome sequencing</td>
      <td>Gubbins - bacterial recombination detection</td>
      <td>Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.</td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>gubbins</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/gubbins</td>
    </tr>
    <tr>
      <td></td>
      <td>gwastools</td>
      <td>0.1.0</td>
      <td>1.46.0</td>
      <td>bioconductor-gwastools</td>
      <td>To update</td>
      <td>gwastools</td>
      <td>GWASTools</td>
      <td>Deposition, Analysis, Annotation</td>
      <td>GWAS study</td>
      <td></td>
      <td>Classes for storing very large GWAS data sets and annotation, and functions for GWAS data cleaning and analysis.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/GWASTools.html</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>hamronization</td>
      <td>1.0.3</td>
      <td>1.1.4</td>
      <td>hamronization</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert AMR gene detection tool output to hAMRonization specification format.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pha4ge/hAMRonization</td>
      <td>Sequence Analysis</td>
      <td>hamronization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hamronization</td>
    </tr>
    <tr>
      <td></td>
      <td>hansel</td>
      <td>2.6.1</td>
      <td>2.6.1</td>
      <td>bio_hansel</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Heidelberg and Enteritidis SNP Elucidation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/phac-nml/bio_hansel</td>
      <td>Sequence Analysis</td>
      <td>bio_hansel</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hansel</td>
    </tr>
    <tr>
      <td></td>
      <td>hapcut2</td>
      <td>1.3.3</td>
      <td>1.3.3</td>
      <td>hapcut2</td>
      <td>Up-to-date</td>
      <td>hapcut2</td>
      <td>HapCUT2</td>
      <td>Haplotype mapping, Variant classification</td>
      <td></td>
      <td>Robust and accurate haplotype assembly for diverse sequencing technologies</td>
      <td>HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to "just work" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.The output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md</td>
      <td>Up-to-date</td>
      <td>https://github.com/vibansal/HapCUT2</td>
      <td>Assembly</td>
      <td>hapcut2</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>hapog</td>
      <td>1.3.6</td>
      <td>1.3.6</td>
      <td>hapog</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hapo-G - Haplotype-Aware Polishing of Genomes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/institut-de-genomique/HAPO-G</td>
      <td>Assembly</td>
      <td>hapog</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hapog</td>
    </tr>
    <tr>
      <td></td>
      <td>heatmap2</td>
      <td>3.1.3</td>
      <td>2.17.0</td>
      <td>r-gplots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>heatmap.2 function from the R gplots package</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>ggplot2_heatmap2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heatmap2</td>
    </tr>
    <tr>
      <td></td>
      <td>heinz</td>
      <td>1.0</td>
      <td>1.60.0</td>
      <td>bioconductor-bionet</td>
      <td>To update</td>
      <td>bionet</td>
      <td>BioNet</td>
      <td>Protein interaction analysis</td>
      <td>Molecular interactions, pathways and networks, Protein interactions</td>
      <td>An algorithm for identification of the optimal scoring subnetwork.</td>
      <td>This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from these p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.</td>
      <td>To update</td>
      <td>https://github.com/ls-cwi/heinz</td>
      <td>Transcriptomics, Visualization, Statistics</td>
      <td>heinz</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/heinz</td>
    </tr>
    <tr>
      <td></td>
      <td>hicexplorer</td>
      <td>3.7.2</td>
      <td>3.7.2</td>
      <td>hicexplorer</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HiCExplorer: Set of programs to process, analyze and visualize Hi-C data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/deeptools/HiCExplorer</td>
      <td>Sequence Analysis, Visualization</td>
      <td>hicexplorer</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>hicstuff</td>
      <td>3.1.5</td>
      <td>3.2.1</td>
      <td>hicstuff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/koszullab/hicstuff</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hicstuff</td>
    </tr>
    <tr>
      <td></td>
      <td>hifiasm_meta</td>
      <td>0.3.1</td>
      <td>hamtv0.3.1</td>
      <td>hifiasm_meta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A hifiasm fork for metagenome assembly using Hifi reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/xfengnefx/hifiasm-meta</td>
      <td>Metagenomics</td>
      <td>hifiasm_meta</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hifiasm_meta</td>
    </tr>
    <tr>
      <td></td>
      <td>hisat2</td>
      <td>2.2.1</td>
      <td>2.2.1</td>
      <td>hisat2</td>
      <td>Up-to-date</td>
      <td>hisat2</td>
      <td>HISAT2</td>
      <td>Sequence alignment</td>
      <td>RNA-seq</td>
      <td>HISAT2 is a fast and sensitive spliced alignment program.</td>
      <td>Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
      <td>Up-to-date</td>
      <td>http://ccb.jhu.edu/software/hisat2/</td>
      <td>Assembly</td>
      <td>hisat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hisat2</td>
    </tr>
    <tr>
      <td></td>
      <td>hmmer3</td>
      <td>3.3.2</td>
      <td>3.3.2</td>
      <td>hmmer</td>
      <td>Up-to-date</td>
      <td>hmmer3</td>
      <td>HMMER3</td>
      <td>Formatting, Multiple sequence alignment, Sequence profile generation, Format validation, Conversion, Sequence generation, Data retrieval, Statistical calculation, Database search, Formatting, Database search, Database search, Probabilistic sequence generation, Statistical calculation, Statistical calculation, Sequence database search, Formatting, Sequence database search, Database search, Sequence database search</td>
      <td>Sequence analysis, Sequence sites, features and motifs, Gene and protein families</td>
      <td>HMMER is used for searching sequence databases for homologs of proteinsequences, and for making protein sequence alignments. It implementsmethods using probabilistic models called profile hidden Markov models(profile HMMs).</td>
      <td>This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.</td>
      <td>Up-to-date</td>
      <td>http://hmmer.org/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3</td>
    </tr>
    <tr>
      <td></td>
      <td>homer</td>
      <td>4.11</td>
      <td>4.11</td>
      <td>homer</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://homer.ucsd.edu/homer/index.html</td>
      <td>Sequence Analysis</td>
      <td>data_manager_homer_preparse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/homer</td>
    </tr>
    <tr>
      <td></td>
      <td>htseq_count</td>
      <td>0.9.1+galaxy1</td>
      <td>2.0.3</td>
      <td>htseq</td>
      <td>To update</td>
      <td>htseq</td>
      <td>HTSeq</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequence analysis</td>
      <td>Count aligned reads (SAM/BAM) that overlap genomic features (GFF)</td>
      <td>Python framework to process and analyse high-throughput sequencing (HTS) data</td>
      <td>To update</td>
      <td>https://readthedocs.org/projects/htseq/</td>
      <td>Genomic Interval Operations, SAM, Sequence Analysis, RNA</td>
      <td>htseq_count</td>
      <td>lparsons</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/htseq_count</td>
    </tr>
    <tr>
      <td></td>
      <td>humann</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>humann</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HUMAnN for functionally profiling metagenomes and metatranscriptomes at species-level resolution</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/humann</td>
      <td>Metagenomics</td>
      <td>humann</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/humann</td>
    </tr>
    <tr>
      <td></td>
      <td>hybpiper</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>hybpiper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analyse targeted sequence capture data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/mossmatters/HybPiper</td>
      <td>Sequence Analysis, Phylogenetics</td>
      <td>hybpiper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hybpiper</td>
    </tr>
    <tr>
      <td></td>
      <td>hyphy</td>
      <td>2.5.47</td>
      <td>2.5.50</td>
      <td>hyphy</td>
      <td>To update</td>
      <td>HyPhy</td>
      <td>HyPhy</td>
      <td>Statistical calculation</td>
      <td>Phylogeny, Small molecules, Molecular interactions, pathways and networks</td>
      <td>Hypothesis Testing using Phylogenies</td>
      <td>Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.</td>
      <td>To update</td>
      <td>http://www.hyphy.org</td>
      <td>Phylogenetics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hyphy/</td>
    </tr>
    <tr>
      <td></td>
      <td>hypo</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>hypo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Super Fast &amp; Accurate Polisher for Long Read Genome Assemblies</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kensung-lab/hypo</td>
      <td>Assembly</td>
      <td>hypo</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/hypo</td>
    </tr>
    <tr>
      <td></td>
      <td>icescreen</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>icescreen</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ICEscreen identifies Integrative Conjugative Elements (ICEs) and Integrative Mobilizable Elements (IMEs) in Firmicutes genomes.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://icescreen.migale.inrae.fr/</td>
      <td>Genome annotation</td>
      <td>icescreen</td>
      <td>iuc</td>
      <td>https://forgemia.inra.fr/ices_imes_analysis/icescreen</td>
    </tr>
    <tr>
      <td></td>
      <td>idba_ud</td>
      <td></td>
      <td>1.1.3</td>
      <td>idba</td>
      <td>To update</td>
      <td>idba</td>
      <td>IDBA</td>
      <td>Sequence assembly</td>
      <td>Sequence assembly</td>
      <td>Wrappers for the idba assembler variants.</td>
      <td>A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.</td>
      <td>To update</td>
      <td>https://i.cs.hku.hk/~alse/hkubrg/projects/index.html</td>
      <td>Assembly</td>
      <td>idba</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idba_ud</td>
    </tr>
    <tr>
      <td></td>
      <td>idr</td>
      <td>2.0.3</td>
      <td>2.0.4.2</td>
      <td>idr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrappers for the IDR package from Nathan Boleu</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nboley/idr</td>
      <td>Sequence Analysis</td>
      <td>idr</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/idr/</td>
    </tr>
    <tr>
      <td></td>
      <td>iedb_api</td>
      <td>2.15.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get epitope binding predictions from IEDB-API</td>
      <td></td>
      <td>To update</td>
      <td>http://tools.immuneepitope.org/main/tools-api/</td>
      <td>Data Source, Sequence Analysis</td>
      <td>iedb_api</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iedb_api</td>
    </tr>
    <tr>
      <td></td>
      <td>instrain</td>
      <td>1.5.3</td>
      <td>1.8.0</td>
      <td>instrain</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>InStrain is a tool for analysis of co-occurring genome populations from metagenomes</td>
      <td></td>
      <td>To update</td>
      <td>https://instrain.readthedocs.io/en/latest/#</td>
      <td>Metagenomics</td>
      <td>instrain</td>
      <td>iuc</td>
      <td>https://github.com/MrOlm/inStrain</td>
    </tr>
    <tr>
      <td></td>
      <td>integron_finder</td>
      <td>2.0.2</td>
      <td>3.3.2</td>
      <td>hmmer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"IntegronFinder identify integrons with high accuracy and sensitivity.It searches for attC sites using covariance models, for integron-integrases using HMM profiles, and for other features (promoters, attI site) using pattern matching"</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gem-pasteur/Integron_Finder</td>
      <td>Sequence Analysis</td>
      <td>integronfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/integron_finder</td>
    </tr>
    <tr>
      <td></td>
      <td>interproscan</td>
      <td>5.59-91.0</td>
      <td>5.59_91.0</td>
      <td>interproscan</td>
      <td>To update</td>
      <td>interproscan_ebi</td>
      <td>InterProScan (EBI)</td>
      <td>Sequence motif recognition, Protein feature detection</td>
      <td>Gene and protein families, Sequence analysis</td>
      <td>Interproscan queries the interpro database and provides annotations.</td>
      <td>Scan sequences against the InterPro protein signature databases.</td>
      <td>To update</td>
      <td>http://www.ebi.ac.uk/Tools/pfa/iprscan5/</td>
      <td>Sequence Analysis</td>
      <td>interproscan</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/interproscan</td>
    </tr>
    <tr>
      <td></td>
      <td>iqtree</td>
      <td>2.1.2</td>
      <td>2.2.5</td>
      <td>iqtree</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Efficient phylogenomic software by maximum likelihood</td>
      <td></td>
      <td>To update</td>
      <td>http://www.iqtree.org/</td>
      <td>Phylogenetics</td>
      <td>iqtree</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/iqtree/</td>
    </tr>
    <tr>
      <td></td>
      <td>isescan</td>
      <td>1.7.2.3</td>
      <td>1.7.2</td>
      <td>isescan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"ISEScan is a pipeline to identify IS (Insertion Sequence) elements in genome and metagenomebased on profile hidden Markov models constructed from manually curated IS elements."</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/xiezhq/ISEScan</td>
      <td>Sequence Analysis</td>
      <td>ISEScan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/isescan</td>
    </tr>
    <tr>
      <td></td>
      <td>isoformswitchanalyzer</td>
      <td>1.20.0</td>
      <td>2.0.1</td>
      <td>bioconductor-isoformswitchanalyzer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Statistical identification of isoform switching from RNA-seq derived quantification of novel and/or annotated full-length isoforms.</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/IsoformSwitchAnalyzeR.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>isoformswitchanalyzer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/isoformswitchanalyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>ivar</td>
      <td>1.4.2</td>
      <td>1.4.2</td>
      <td>ivar</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>iVar is a computational package that contains functions broadly useful for viral amplicon-based sequencing</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/andersen-lab/ivar</td>
      <td>Sequence Analysis</td>
      <td>ivar</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ivar/</td>
    </tr>
    <tr>
      <td></td>
      <td>jbrowse</td>
      <td>1.16.11</td>
      <td>1.16.11</td>
      <td>jbrowse</td>
      <td>Up-to-date</td>
      <td>jbrowse</td>
      <td>JBrowse</td>
      <td>Genome visualisation</td>
      <td>Genomics</td>
      <td>JBrowse Genome Browser integrated as a Galaxy Tool</td>
      <td>Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.</td>
      <td>Up-to-date</td>
      <td>https://jbrowse.org</td>
      <td>Sequence Analysis</td>
      <td>jbrowse</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jbrowse</td>
    </tr>
    <tr>
      <td></td>
      <td>jcvi_gff_stats</td>
      <td>0.8.4</td>
      <td>1.3.8</td>
      <td>jcvi</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute statistics from a genome annotation in GFF3 format (using JCVI Python utilities)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/tanghaibao/jcvi</td>
      <td>Sequence Analysis</td>
      <td>jcvi_gff_stats</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jcvi_gff_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>jellyfish</td>
      <td></td>
      <td>2.3.0</td>
      <td>kmer-jellyfish</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gmarcais/Jellyfish</td>
      <td>Assembly</td>
      <td>jellyfish</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/jellyfish</td>
    </tr>
    <tr>
      <td></td>
      <td>kc-align</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>kcalign</td>
      <td>Up-to-date</td>
      <td>kc-align</td>
      <td>kc-align</td>
      <td>Multiple sequence alignment</td>
      <td>Mapping</td>
      <td>Kc-Align custom tool</td>
      <td>A fast and accurate tool for performing codon-aware multiple sequence alignments</td>
      <td>Up-to-date</td>
      <td>https://github.com/davebx/kc-align</td>
      <td>Sequence Analysis</td>
      <td>kc_align</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kc-align</td>
    </tr>
    <tr>
      <td></td>
      <td>khmer</td>
      <td>3.0.0a3</td>
      <td>3.0.0a3</td>
      <td>khmer</td>
      <td>Up-to-date</td>
      <td>khmer</td>
      <td>khmer</td>
      <td>Standardisation and normalisation, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more</td>
      <td>khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.</td>
      <td>Up-to-date</td>
      <td>https://khmer.readthedocs.org/</td>
      <td>Assembly, Next Gen Mappers</td>
      <td>khmer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/khmer</td>
    </tr>
    <tr>
      <td></td>
      <td>kleborate</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>kleborate</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Screen genome assemblies of Klebsiella pneumoniae and the Klebsiella pneumoniae species complex (KpSC)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/Kleborate/wiki</td>
      <td>Metagenomics</td>
      <td>kleborate</td>
      <td>iuc</td>
      <td>https://github.com/katholt/Kleborate</td>
    </tr>
    <tr>
      <td></td>
      <td>kofamscan</td>
      <td>1.3.0</td>
      <td>1.3.0</td>
      <td>kofamscan</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gene function annotation tool based on KEGG Orthology and hidden Markov model</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/takaram/kofam_scan</td>
      <td>Sequence Analysis</td>
      <td>kofamscan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kofamscan</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_biom</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>kraken-biom</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create BIOM-format tables (http://biom-format.org) from Kraken output (http://ccb.jhu.edu/software/kraken/)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
      <td>Metagenomics</td>
      <td>kraken_biom</td>
      <td>iuc</td>
      <td>https://github.com/smdabdoub/kraken-biom</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken_taxonomy_report</td>
      <td>0.0.3</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken taxonomy report</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/blankenberg/Kraken-Taxonomy-Report</td>
      <td>Metagenomics</td>
      <td>kraken_taxonomy_report</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/kraken_taxonomy_report</td>
    </tr>
    <tr>
      <td></td>
      <td>krakentools</td>
      <td>1.2</td>
      <td>1.2</td>
      <td>krakentools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>KrakenTools is a suite of scripts to be used alongside the Kraken</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/jenniferlu717/KrakenTools</td>
      <td>Metagenomics</td>
      <td>krakentools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krakentools</td>
    </tr>
    <tr>
      <td></td>
      <td>krocus</td>
      <td>1.0.1</td>
      <td>1.0.3</td>
      <td>krocus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Predict MLST directly from uncorrected long reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/krocus</td>
      <td>Sequence Analysis</td>
      <td>krocus</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/krocus</td>
    </tr>
    <tr>
      <td></td>
      <td>last</td>
      <td>1205</td>
      <td>1454</td>
      <td>last</td>
      <td>To update</td>
      <td>last</td>
      <td>LAST</td>
      <td>Sequence alignment</td>
      <td>Genomics, Comparative genomics</td>
      <td>LAST finds similar regions between sequences.</td>
      <td>Short read alignment program incorporating quality scores</td>
      <td>To update</td>
      <td>http://last.cbrc.jp/</td>
      <td>Sequence Analysis</td>
      <td>last</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/last</td>
    </tr>
    <tr>
      <td></td>
      <td>legsta</td>
      <td>0.5.1</td>
      <td>0.5.1</td>
      <td>legsta</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Performs in silico Legionella pneumophila sequence based typing.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/legsta</td>
      <td>Sequence Analysis</td>
      <td>legsta</td>
      <td>iuc</td>
      <td>https://github.com/tseemann/legsta</td>
    </tr>
    <tr>
      <td></td>
      <td>length_and_gc_content</td>
      <td>0.1.2</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets gene length and gc content from a fasta and a GTF file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
      <td>Fasta Manipulation, Statistics, RNA, Micro-array Analysis</td>
      <td>length_and_gc_content</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/length_and_gc_content</td>
    </tr>
    <tr>
      <td></td>
      <td>limma_voom</td>
      <td>3.50.1</td>
      <td>3.56.2</td>
      <td>bioconductor-limma</td>
      <td>To update</td>
      <td>limma</td>
      <td>limma</td>
      <td>RNA-Seq analysis</td>
      <td>Molecular biology, Genetics</td>
      <td>Perform RNA-Seq differential expression analysis using limma voom pipeline</td>
      <td>Data analysis, linear models and differential expression for microarray data.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/limma.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>limma_voom</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/limma_voom</td>
    </tr>
    <tr>
      <td></td>
      <td>lineagespot</td>
      <td>1.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td>lineagespot</td>
      <td>lineagespot</td>
      <td>Variant calling</td>
      <td>Metagenomics, Gene transcripts, Evolutionary biology, Sequencing, Genetic variation</td>
      <td>Identification of SARS-CoV-2 related metagenomic mutations based on a single (or a list of) variant(s) file(s)</td>
      <td>Lineagespot is a framework written in R, and aims to identify and assign different SARS-CoV-2 lineages based on a single variant file (i.e., variant calling format).</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/lineagespot.html</td>
      <td>Metagenomics, Sequence Analysis</td>
      <td>lineagespot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/lineagespot</td>
    </tr>
    <tr>
      <td></td>
      <td>links</td>
      <td>2.0.1</td>
      <td>2.0.1</td>
      <td>links</td>
      <td>Up-to-date</td>
      <td>links</td>
      <td>LINKS</td>
      <td>Scaffolding, Genome assembly, Read mapping, Read pre-processing, Sequence trimming</td>
      <td>Sequence assembly, Mapping, Sequencing</td>
      <td>Scaffold genome assemblies with long reads.</td>
      <td>LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.</td>
      <td>Up-to-date</td>
      <td>https://github.com/bcgsc/LINKS</td>
      <td>Assembly</td>
      <td>links</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/links</td>
    </tr>
    <tr>
      <td></td>
      <td>lorikeet</td>
      <td>20</td>
      <td>20</td>
      <td>lorikeet</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for M. tuberculosis DNA fingerprinting (spoligotyping)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
      <td>Sequence Analysis</td>
      <td>lorikeet_spoligotype</td>
      <td>iuc</td>
      <td>https://github.com/AbeelLab/lorikeet</td>
    </tr>
    <tr>
      <td></td>
      <td>m6anet</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>m6anet</td>
      <td>Up-to-date</td>
      <td>m6Anet</td>
      <td>m6Anet</td>
      <td>Quantification, Imputation, Gene expression profiling</td>
      <td>RNA-Seq, Transcriptomics, RNA, Machine learning</td>
      <td>m6anet to detect m6A RNA modifications from nanopore data</td>
      <td>Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</td>
      <td>Up-to-date</td>
      <td>https://m6anet.readthedocs.io/en/latest</td>
      <td>Sequence Analysis</td>
      <td>m6anet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/m6anet</td>
    </tr>
    <tr>
      <td></td>
      <td>maaslin2</td>
      <td>0.99.12</td>
      <td>0.99.12</td>
      <td>maaslin2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaAsLin2 is comprehensive R package for efficiently determining multivariable association between microbial meta'omic features and clinical metadata.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://huttenhower.sph.harvard.edu/maaslin</td>
      <td>Metagenomics</td>
      <td>maaslin2</td>
      <td>iuc</td>
      <td>https://github.com/biobakery/Maaslin2</td>
    </tr>
    <tr>
      <td></td>
      <td>macs2</td>
      <td>2.2.9.1</td>
      <td>2.2.9.1</td>
      <td>macs2</td>
      <td>Up-to-date</td>
      <td>macs</td>
      <td>MACS</td>
      <td>Peak calling, Enrichment analysis, Gene regulatory network analysis</td>
      <td>ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites</td>
      <td>MACS - Model-based Analysis of ChIP-Seq</td>
      <td>Model-based Analysis of ChIP-seq data.</td>
      <td>Up-to-date</td>
      <td>https://github.com/taoliu/MACS</td>
      <td>Sequence Analysis, Statistics</td>
      <td>macs2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/macs2</td>
    </tr>
    <tr>
      <td></td>
      <td>maker</td>
      <td>2.31.11</td>
      <td>3.01.03</td>
      <td>maker</td>
      <td>To update</td>
      <td>maker</td>
      <td>MAKER</td>
      <td>Genome annotation</td>
      <td>Genomics, DNA, Sequence analysis</td>
      <td>MAKER is a portable and easily configurable genome annotation pipeline.Its purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.</td>
      <td>To update</td>
      <td>http://www.yandell-lab.org/software/maker.html</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maker</td>
    </tr>
    <tr>
      <td></td>
      <td>mapseq</td>
      <td>2.1.1</td>
      <td></td>
      <td>perl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>fast and accurate sequence read classification tool designed to assign taxonomy and OTU classifications to ribosomal RNA sequences.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jfmrod/MAPseq</td>
      <td>Metagenomics</td>
      <td>mapseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mapseq</td>
    </tr>
    <tr>
      <td></td>
      <td>mash</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>mash</td>
      <td>Up-to-date</td>
      <td>mash</td>
      <td>Mash</td>
      <td>Sequence distance matrix generation</td>
      <td>Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation</td>
      <td>Fast genome and metagenome distance estimation using MinHash</td>
      <td>Fast genome and metagenome distance estimation using MinHash.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/Mash</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mash</td>
    </tr>
    <tr>
      <td></td>
      <td>masigpro</td>
      <td>1.49.3</td>
      <td>8.25</td>
      <td>coreutils</td>
      <td>To update</td>
      <td>masigpro</td>
      <td>maSigPro</td>
      <td>Regression analysis</td>
      <td>Gene expression, Molecular genetics, Microarray experiment, RNA-Seq</td>
      <td>Identify significantly differential expression profiles in time-course microarray experiments</td>
      <td>Regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>masigpro</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/masigpro</td>
    </tr>
    <tr>
      <td></td>
      <td>maxbin2</td>
      <td></td>
      <td>2.2.7</td>
      <td>maxbin2</td>
      <td>To update</td>
      <td>maxbin</td>
      <td>MaxBin</td>
      <td>Sequence assembly</td>
      <td>Metagenomics, Sequence assembly, Microbiology</td>
      <td>clusters metagenomic contigs into bins</td>
      <td>Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.</td>
      <td>To update</td>
      <td>https://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html</td>
      <td>Metagenomics</td>
      <td>maxbin2</td>
      <td>mbernt</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/maxbin2/</td>
    </tr>
    <tr>
      <td></td>
      <td>medaka</td>
      <td>1.7.2</td>
      <td>1.11.1</td>
      <td>medaka</td>
      <td>To update</td>
      <td>medaka</td>
      <td>Medaka</td>
      <td>Base-calling, Variant calling, Sequence assembly</td>
      <td>Sequence assembly, Machine learning</td>
      <td>Sequence correction provided by ONT Research</td>
      <td>medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.</td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/medaka</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/medaka</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit</td>
      <td>1.2.9</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>Up-to-date</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>Up-to-date</td>
      <td>https://github.com/voutcn/megahit</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit</td>
    </tr>
    <tr>
      <td></td>
      <td>megahit_contig2fastg</td>
      <td>1.1.3</td>
      <td>1.2.9</td>
      <td>megahit</td>
      <td>To update</td>
      <td>megahit</td>
      <td>MEGAHIT</td>
      <td>Genome assembly</td>
      <td>Metagenomics, Sequencing, Ecology, Sequence assembly</td>
      <td>A subprogram within the Megahit toolkit for converting contigs to assembly graphs (fastg)</td>
      <td>Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.</td>
      <td>To update</td>
      <td>https://github.com/voutcn/megahit/blob/master/tools/toolkit.cpp</td>
      <td>Sequence Analysis, Assembly, Metagenomics</td>
      <td>megahit_contig2fastg</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megahit_contig2fastg</td>
    </tr>
    <tr>
      <td></td>
      <td>megan</td>
      <td>6.21.7</td>
      <td>6.24.20</td>
      <td>megan</td>
      <td>To update</td>
      <td>megan</td>
      <td>MEGAN</td>
      <td>Sequence analysis, Taxonomic classification</td>
      <td>Sequence analysis</td>
      <td>MEGAN Community Edition - Interactive exploration and analysis of large-scale microbiome sequencing data.  MEGAN is a tool for studying the taxonomic content of a set of DNA reads, typically collected in a metagenomics project.In a preprocessing step, a sequence alignment of all reads against a suitable database of reference DNA or proteinsequences must be performed to produce an input file for the program. MEGAN is suitable for DNA reads (metagenomedata), RNA reads (metatranscriptome data), peptide sequences (metaproteomics data) and, using a suitable synonymsfile that maps SILVA ids to taxon ids, for 16S rRNA data (amplicon sequencing).</td>
      <td>Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.</td>
      <td>To update</td>
      <td>https://github.com/husonlab/megan-ce</td>
      <td>Sequence Analysis</td>
      <td>megan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/megan</td>
    </tr>
    <tr>
      <td></td>
      <td>meningotype</td>
      <td>0.8.5</td>
      <td>0.8.5</td>
      <td>meningotype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Assign sequence type to N. meningitidis genome assemblies</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/MDU-PHL/meningotype</td>
      <td>Sequence Analysis</td>
      <td>meningotype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meningotype</td>
    </tr>
    <tr>
      <td></td>
      <td>merqury</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td>merqury</td>
      <td>Merqury</td>
      <td>Genome assembly, k-mer counting, Scaffolding, Phasing, De-novo assembly</td>
      <td>Sequence assembly, Whole genome sequencing, Plant biology</td>
      <td>Merqury is a tool for evaluating genomes assemblies based of k-mer operations.</td>
      <td>Reference-free quality, completeness, and phasing assessment for genome assemblies.Evaluate genome assemblies with k-mers and more.Often, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.Merqury provides a set of tools for this purpose.</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/merqury</td>
      <td>Assembly</td>
      <td>merqury</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/merqury</td>
    </tr>
    <tr>
      <td></td>
      <td>meryl</td>
      <td>1.3</td>
      <td>1.3</td>
      <td>merqury</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Meryl a k-mer counter.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/meryl</td>
      <td>Assembly</td>
      <td>meryl</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/meryl</td>
    </tr>
    <tr>
      <td></td>
      <td>metabat2</td>
      <td>2.15</td>
      <td>2.15</td>
      <td>metabat2</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaBAT2 (Metagenome Binning based on Abundance and Tetranucleotide frequency) is an automated metagenome binningsoftware that integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/berkeleylab/metabat/src/master/</td>
      <td>Metagenomics</td>
      <td>metabat2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metabat2/</td>
    </tr>
    <tr>
      <td></td>
      <td>metaeuk</td>
      <td>5.34c21f2</td>
      <td>6.a5d39d9</td>
      <td>metaeuk</td>
      <td>To update</td>
      <td>MetaEuk</td>
      <td>MetaEuk</td>
      <td>Homology-based gene prediction</td>
      <td>Metagenomics, Gene and protein families</td>
      <td>MetaEuk is a modular toolkit designed for large-scale gene discovery andannotation in eukaryotic metagenomic contigs. Metaeuk combines the fast andsensitive homology search capabilities of MMseqs2 with a dynamic programmingprocedure to recover optimal exons sets. It reduces redundancies in multiplediscoveries of the same gene and resolves conflicting gene predictions onthe same strand.</td>
      <td>MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics</td>
      <td>To update</td>
      <td>https://github.com/soedinglab/metaeuk</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaeuk</td>
    </tr>
    <tr>
      <td></td>
      <td>metagenomeseq</td>
      <td>1.16.0-0.0.1</td>
      <td>1.42.0</td>
      <td>bioconductor-metagenomeseq</td>
      <td>To update</td>
      <td>metagenomeseq</td>
      <td>metagenomeSeq</td>
      <td>Sequence visualisation, Statistical calculation</td>
      <td>Metagenomics, Sequencing</td>
      <td>metagenomeSeq Normalization</td>
      <td>Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.</td>
      <td>To update</td>
      <td></td>
      <td>Metagenomics</td>
      <td>metagenomeseq_normalization</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metagenomeseq</td>
    </tr>
    <tr>
      <td></td>
      <td>metaphlan</td>
      <td>4.0.6</td>
      <td>4.0.6</td>
      <td>metaphlan</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaPhlAn for Metagenomic Phylogenetic Analysis</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/biobakery/MetaPhlAn</td>
      <td>Metagenomics</td>
      <td>metaphlan</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/metaphlan/</td>
    </tr>
    <tr>
      <td></td>
      <td>migmap</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>migmap</td>
      <td>Up-to-date</td>
      <td>MiGMAP</td>
      <td>MiGMAP</td>
      <td>Sequence analysis, Read mapping</td>
      <td>Immunoproteins, genes and antigens, Sequence analysis</td>
      <td>mapper for full-length T- and B-cell repertoire sequencing</td>
      <td>Mapper for full-length T- and B-cell repertoire sequencing.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mikessh/migmap</td>
      <td>RNA, Sequence Analysis</td>
      <td>migmap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/migmap</td>
    </tr>
    <tr>
      <td></td>
      <td>minia</td>
      <td>3.2.6</td>
      <td>3.2.6</td>
      <td>minia</td>
      <td>Up-to-date</td>
      <td>minia</td>
      <td>Minia</td>
      <td>Genome assembly</td>
      <td>Sequence assembly</td>
      <td>Short-read assembler based on a de Bruijn graph</td>
      <td>Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.</td>
      <td>Up-to-date</td>
      <td>https://gatb.inria.fr/software/minia/</td>
      <td>Assembly</td>
      <td>minia</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/minia</td>
    </tr>
    <tr>
      <td></td>
      <td>miniasm</td>
      <td>0.3_r179</td>
      <td>0.3</td>
      <td>miniasm</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Miniasm - Ultrafast de novo assembly for long noisy reads (though having no consensus step)</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lh3/miniasm</td>
      <td>Assembly</td>
      <td>miniasm</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniasm</td>
    </tr>
    <tr>
      <td></td>
      <td>miniprot</td>
      <td>0.12</td>
      <td>0.12</td>
      <td>miniprot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Align a protein sequence against a genome with affine gap penalty, splicing and frameshift.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/lh3/miniprot</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/miniprot</td>
    </tr>
    <tr>
      <td></td>
      <td>mirnature</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>mirnature</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computational detection of canonical microRNAs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
      <td>RNA, Sequence Analysis</td>
      <td>mirnature</td>
      <td>iuc</td>
      <td>https://github.com/Bierinformatik/miRNAture</td>
    </tr>
    <tr>
      <td></td>
      <td>mitobim</td>
      <td>1.9.1</td>
      <td>1.9.1</td>
      <td>mitobim</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>assemble mitochondrial genomes</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/chrishah/MITObim</td>
      <td>Assembly</td>
      <td>mitobim</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitobim</td>
    </tr>
    <tr>
      <td></td>
      <td>mitos</td>
      <td>1.1.1</td>
      <td>2.1.3</td>
      <td>mitos</td>
      <td>To update</td>
      <td>mitos</td>
      <td>MITOS</td>
      <td>Genome annotation</td>
      <td>Zoology, Whole genome sequencing</td>
      <td>de-novo annotation of metazoan mitochondrial genomes</td>
      <td>De novo metazoan mitochondrial genome annotation.</td>
      <td>To update</td>
      <td>http://mitos.bioinf.uni-leipzig.de/</td>
      <td>Sequence Analysis</td>
      <td>mitos</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mitos</td>
    </tr>
    <tr>
      <td></td>
      <td>mlst</td>
      <td>2.22.0</td>
      <td>2.23.0</td>
      <td>mlst</td>
      <td>To update</td>
      <td>mlst</td>
      <td>MLST</td>
      <td>Taxonomic classification</td>
      <td>Immunoproteins, genes and antigens</td>
      <td>Scan contig files against PubMLST typing schemes</td>
      <td>Multi Locus Sequence Typing from an assembled genome or from a set of reads.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/mlst</td>
      <td>Sequence Analysis</td>
      <td>mlst</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst</td>
    </tr>
    <tr>
      <td></td>
      <td>mothur</td>
      <td>1.0</td>
      <td>1.48.0</td>
      <td>mothur</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mothur wrappers</td>
      <td></td>
      <td>To update</td>
      <td>https://www.mothur.org</td>
      <td>Metagenomics</td>
      <td>mothur</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mothur</td>
    </tr>
    <tr>
      <td></td>
      <td>multigsea</td>
      <td>1.8.0</td>
      <td>1.10.0</td>
      <td>bioconductor-multigsea</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>GSEA-based pathway enrichment analysis for multi-omics data</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/devel/bioc/html/multiGSEA.html</td>
      <td>Transcriptomics, Proteomics, Statistics</td>
      <td>multigsea</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multigsea</td>
    </tr>
    <tr>
      <td></td>
      <td>multiqc</td>
      <td>1.11</td>
      <td>1.17</td>
      <td>multiqc</td>
      <td>To update</td>
      <td>multiqc</td>
      <td>MultiQC</td>
      <td>Validation</td>
      <td>Sequencing, Bioinformatics</td>
      <td>MultiQC aggregates results from bioinformatics analyses across many samples into a single report</td>
      <td>MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</td>
      <td>To update</td>
      <td>http://multiqc.info/</td>
      <td>Fastq Manipulation, Statistics, Visualization</td>
      <td>multiqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/multiqc</td>
    </tr>
    <tr>
      <td></td>
      <td>mummer4</td>
      <td>4.0.0rc1</td>
      <td>4.0.0rc1</td>
      <td>mummer4</td>
      <td>Up-to-date</td>
      <td>mumer4</td>
      <td>MUMmer4</td>
      <td>Multiple sequence alignment</td>
      <td>Sequence analysis, Human genetics</td>
      <td>Mummer4 Tools</td>
      <td>System for rapidly aligning large DNA sequences to one another.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mummer4/mummer</td>
      <td>Sequence Analysis</td>
      <td>mummer4</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mummer4</td>
    </tr>
    <tr>
      <td></td>
      <td>mykrobe</td>
      <td>0.10.0</td>
      <td>0.13.0</td>
      <td>mykrobe</td>
      <td>To update</td>
      <td>Mykrobe</td>
      <td>Mykrobe</td>
      <td>Antimicrobial resistance prediction, Variant calling, Genotyping, Sequence trimming</td>
      <td>Whole genome sequencing, Genotype and phenotype, Probes and primers, Genetic variation, Metagenomics</td>
      <td>Antibiotic resistance predictions</td>
      <td>Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe.Antibiotic resistance prediction in minutes.Table of Contents generated with DocToc.AMR prediction (Mykrobe predictor).Before attempting to install with bioconda, please ensure you have your channels set up as specified in the documentation. If you don't, you may run into issues with an older version of mykrobe being installed</td>
      <td>To update</td>
      <td>https://github.com/Mykrobe-tools/mykrobe</td>
      <td>Sequence Analysis</td>
      <td>mykrobe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/mykrobe</td>
    </tr>
    <tr>
      <td></td>
      <td>naltorfs</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>naltorfs</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>nAlt-ORFs: Nested Alternate Open Reading Frames (nAltORFs)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BlankenbergLab/nAltORFs</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/naltorfs/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanocompore</td>
      <td>1.0.0rc3.post2</td>
      <td>1.0.4</td>
      <td>nanocompore</td>
      <td>To update</td>
      <td>Nanocompore</td>
      <td>Nanocompore</td>
      <td>Post-translation modification site prediction, PolyA signal detection, Genotyping, k-mer counting</td>
      <td>Functional, regulatory and non-coding RNA, RNA-Seq, Gene transcripts, Transcriptomics, Transcription factors and regulatory sites</td>
      <td>Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro.</td>
      <td>RNA modifications detection by comparative Nanopore direct RNA sequencing.RNA modifications detection from Nanopore dRNA-Seq data.Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.Analyses performed for the nanocompore paper.Nanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro</td>
      <td>To update</td>
      <td>https://nanocompore.rna.rocks/</td>
      <td>Sequence Analysis</td>
      <td>nanocompore</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanocompore</td>
    </tr>
    <tr>
      <td></td>
      <td>nanoplot</td>
      <td>1.41.0</td>
      <td>1.41.6</td>
      <td>nanoplot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Plotting tool for long read sequencing data and alignments</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/wdecoster/NanoPlot</td>
      <td>Visualization</td>
      <td>nanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanoplot/</td>
    </tr>
    <tr>
      <td></td>
      <td>nanopolishcomp</td>
      <td>0.6.11</td>
      <td>0.6.12</td>
      <td>nanopolishcomp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NanopolishComp contains 2 modules. Eventalign_collapse collapses the raw file generated by nanopolish eventalign by kmers rather than by event. Freq_meth_calculate methylation frequency at genomic CpG sites from the output of nanopolish call-methylation.</td>
      <td></td>
      <td>To update</td>
      <td>https://a-slide.github.io/NanopolishComp</td>
      <td>Sequence Analysis</td>
      <td>nanopolishcomp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nanopolishcomp</td>
    </tr>
    <tr>
      <td></td>
      <td>necat</td>
      <td>0.0.1_update20200803</td>
      <td>0.0.1_update20200803</td>
      <td>necat</td>
      <td>Up-to-date</td>
      <td>necat</td>
      <td>NECAT</td>
      <td>De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>Error correction and de-novo assembly for ONT Nanopore reads</td>
      <td>NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaochuanle/NECAT</td>
      <td>Assembly</td>
      <td>necat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/necat</td>
    </tr>
    <tr>
      <td></td>
      <td>newick_utils</td>
      <td>1.6+galaxy1</td>
      <td>1.6</td>
      <td>newick_utils</td>
      <td>To update</td>
      <td>newick_utilities</td>
      <td>Newick Utilities</td>
      <td>Phylogenetic tree generation, Phylogenetic tree analysis, Phylogenetic tree reconstruction</td>
      <td>Phylogeny, Genomics, Computer science</td>
      <td>Perform operations on Newick trees</td>
      <td>The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.</td>
      <td>To update</td>
      <td>http://cegg.unige.ch/newick_utils</td>
      <td>Visualization, Metagenomics</td>
      <td>newick_utils</td>
      <td>iuc</td>
      <td>https://github.com/tjunier/newick_utils</td>
    </tr>
    <tr>
      <td></td>
      <td>nextclade</td>
      <td>2.7.0</td>
      <td>2.14.0</td>
      <td>nextalign</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identify differences between your sequences and a reference sequence used by Nextstrain</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nextstrain/nextclade</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nextclade</td>
    </tr>
    <tr>
      <td></td>
      <td>nonpareil</td>
      <td>3.1.1</td>
      <td>3.4.1</td>
      <td>nonpareil</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Estimate average coverage in metagenomic datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://nonpareil.readthedocs.io</td>
      <td>Metagenomics</td>
      <td>nonpareil</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nonpareil</td>
    </tr>
    <tr>
      <td></td>
      <td>novoplasty</td>
      <td>4.3.1</td>
      <td>4.3.3</td>
      <td>novoplasty</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NOVOPlasty is a de novo assembler and heteroplasmy/variance caller for short circular genomes.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ndierckx/NOVOPlasty</td>
      <td>Assembly</td>
      <td>novoplasty</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/novoplasty</td>
    </tr>
    <tr>
      <td></td>
      <td>nugen_nudup</td>
      <td>2.3.3</td>
      <td>2.3.3</td>
      <td>nudup</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Marks/removes PCR introduced duplicate molecules based on the molecular tagging technology used in NuGEN products.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://nugentechnologies.github.io/nudup/</td>
      <td>SAM, Metagenomics, Sequence Analysis, Transcriptomics</td>
      <td>nugen_nudup</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/nugen_nudup</td>
    </tr>
    <tr>
      <td></td>
      <td>obitools</td>
      <td>1.2.13</td>
      <td>1.2.13</td>
      <td>obitools</td>
      <td>Up-to-date</td>
      <td>obitools</td>
      <td>OBITools</td>
      <td>Sequence analysis, Sequence analysis</td>
      <td>Sequence analysis, DNA, Sequencing</td>
      <td>OBITools is a set of programs developed to simplify the manipulation of sequence files</td>
      <td>Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.</td>
      <td>Up-to-date</td>
      <td>http://metabarcoding.org/obitools</td>
      <td>Sequence Analysis</td>
      <td>obitools</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/obitools</td>
    </tr>
    <tr>
      <td></td>
      <td>odgi</td>
      <td>0.3</td>
      <td>0.8.3</td>
      <td>odgi</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Representing large genomic variation graphs with minimal memory overhead requires a careful encoding of the graph entities. odgi follows the dynamic GBWT in developing a byte-packed version of the graph and paths through it.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/odgi</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/odgi/</td>
    </tr>
    <tr>
      <td></td>
      <td>ont_fast5_api</td>
      <td>3.1.3</td>
      <td>4.1.1</td>
      <td>ont-fast5-api</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ont_fast5_api is a simple interface to HDF5 files of the Oxford Nanopore FAST5 file format.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nanoporetech/ont_fast5_api/</td>
      <td>Nanopore</td>
      <td>ont_fast5_api</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ont_fast5_api</td>
    </tr>
    <tr>
      <td></td>
      <td>optitype</td>
      <td>1.3.5</td>
      <td>1.3.5</td>
      <td>optitype</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Precision HLA typing from NGS data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/FRED-2/OptiType</td>
      <td>Sequence Analysis</td>
      <td>optitype</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/optitype1</td>
    </tr>
    <tr>
      <td></td>
      <td>orfipy</td>
      <td>0.0.4</td>
      <td>0.0.4</td>
      <td>orfipy</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy wrapper for ORFIPY</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/urmi-21/orfipy</td>
      <td>Sequence Analysis</td>
      <td>orfipy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orfipy</td>
    </tr>
    <tr>
      <td></td>
      <td>orthofinder</td>
      <td>2.5.4</td>
      <td>2.5.5</td>
      <td>orthofinder</td>
      <td>To update</td>
      <td>OrthoFinder</td>
      <td>OrthoFinder</td>
      <td>Genome comparison, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Genome alignment</td>
      <td>Phylogenetics, Phylogenomics, Bioinformatics, Comparative genomics, Sequence analysis</td>
      <td>Accurate inference of orthologous gene groups made easy</td>
      <td>OrthoFinder is a fast, accurate and comprehensive platform for comparative genomics. It finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplcation events in those gene trees. It also infers a rooted species tree for the species being analysed and maps the gene duplication events from the gene trees to branches in the species tree. OrthoFinder also provides comprehensive statistics for comparative genomic analyses.</td>
      <td>To update</td>
      <td>https://github.com/davidemms/OrthoFinder</td>
      <td>Phylogenetics, Sequence Analysis</td>
      <td>orthofinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/orthofinder</td>
    </tr>
    <tr>
      <td></td>
      <td>pangolin</td>
      <td>4.3</td>
      <td>4.3</td>
      <td>pangolin</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Pangolin assigns SARS-CoV-2 genome sequences their most likely lineages under the Pango nomenclature system.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cov-lineages/pangolin</td>
      <td>Sequence Analysis</td>
      <td>pangolin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pangolin</td>
    </tr>
    <tr>
      <td></td>
      <td>parse_mito_blast</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>parse_mito_blast</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filtering blast out from querying assembly against mitochondrial database.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://raw.githubusercontent.com/VGP/vgp-assembly/master/galaxy_tools/parse_mito_blast/parse_mito_blast.py</td>
      <td>Sequence Analysis</td>
      <td>parse_mito_blast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/parse_mito_blast</td>
    </tr>
    <tr>
      <td></td>
      <td>pathview</td>
      <td>1.34.0</td>
      <td>1.40.0</td>
      <td>bioconductor-pathview</td>
      <td>To update</td>
      <td>pathview</td>
      <td>pathview</td>
      <td>Pathway or network analysis, Pathway or network visualisation</td>
      <td>Molecular interactions, pathways and networks, Systems biology, Data visualisation</td>
      <td>Pathview is a tool set for pathway based data integration and visualization.</td>
      <td>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/pathview.html</td>
      <td>Statistics, RNA, Micro-array Analysis</td>
      <td>pathview</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pathview</td>
    </tr>
    <tr>
      <td></td>
      <td>pharokka</td>
      <td>1.3.2</td>
      <td></td>
      <td>\n                pharokka\n</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>rapid standardised annotation tool for bacteriophage genomes and metagenomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/gbouras13/pharokka</td>
      <td>Genome annotation</td>
      <td>pharokka</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tools/pharokka</td>
    </tr>
    <tr>
      <td></td>
      <td>phyloseq</td>
      <td>1.38.0</td>
      <td>1.44.0</td>
      <td>bioconductor-phyloseq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Handling and analysis of high-throughput microbiome census data</td>
      <td></td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html</td>
      <td>Metagenomics</td>
      <td>phyloseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyloseq</td>
    </tr>
    <tr>
      <td></td>
      <td>phyml</td>
      <td>3.3.20220408</td>
      <td>3.3.20220408</td>
      <td>phyml</td>
      <td>Up-to-date</td>
      <td>phyml</td>
      <td>PhyML</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>Phylogenetics, Bioinformatics, Phylogenetics</td>
      <td>PhyML is a phylogeny software based on the maximum-likelihood principle.</td>
      <td>Phylogenetic estimation software using Maximum Likelihood</td>
      <td>Up-to-date</td>
      <td>http://www.atgc-montpellier.fr/phyml/</td>
      <td>Phylogenetics</td>
      <td>phyml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/phyml</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust</td>
      <td>1.1.1</td>
      <td>1.1.4</td>
      <td>picrust</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PICRUSt wrappers</td>
      <td></td>
      <td>To update</td>
      <td>https://picrust.github.io/picrust/</td>
      <td>Metagenomics</td>
      <td>picrust</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/picrust</td>
    </tr>
    <tr>
      <td></td>
      <td>picrust2</td>
      <td>2.5.1</td>
      <td>2.5.2</td>
      <td>picrust2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PICRUSt2: Phylogenetic Investigation of Communities by Reconstruction of Unobserved States</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/picrust/picrust2/wiki</td>
      <td>Metagenomics</td>
      <td>picrust2</td>
      <td>iuc</td>
      <td>https://github.com/picrust/picrust2</td>
    </tr>
    <tr>
      <td></td>
      <td>plasflow</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>plasflow</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PlasFlow - Prediction of plasmid sequences in metagenomic contigs.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/smaegol/PlasFlow</td>
      <td>Sequence Analysis</td>
      <td>plasflow</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/plasflow</td>
    </tr>
    <tr>
      <td></td>
      <td>plasmidfinder</td>
      <td>2.1.6</td>
      <td>2.1.6</td>
      <td>plasmidfinder</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"PlasmidFinder provides the detection of replicons in the WGSand assigns the plasmids under study to lineages that trace backthe information to the existing knowledge on Inc groups and suggestspossible reference plasmids for each lineage"</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://bitbucket.org/genomicepidemiology/plasmidfinder/src/master/</td>
      <td>Sequence Analysis</td>
      <td>plasmidfinder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/plasmidfinder</td>
    </tr>
    <tr>
      <td></td>
      <td>polypolish</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>polypolish</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"Polypolish is a tool for polishing genome assemblies with short reads.Polypolish uses SAM files where each read has been aligned to all possible locations (not just a single best location).This allows it to repair errors in repeat regions that other alignment-based polishers cannot fix."</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Polypolish</td>
      <td>Sequence Analysis</td>
      <td>polypolish</td>
      <td>iuc</td>
      <td>https://github.com/mesocentre-clermont-auvergne/galaxy-tools/tree/master/tools/polypolish</td>
    </tr>
    <tr>
      <td></td>
      <td>presto</td>
      <td>0.6.2</td>
      <td>0.7.1</td>
      <td>presto</td>
      <td>To update</td>
      <td>presto</td>
      <td>pRESTO</td>
      <td>Nucleic acid sequence analysis</td>
      <td>Sequencing, DNA, Immunology</td>
      <td>pRESTO toolkit for immune repertoire analysis.</td>
      <td>Integrated collection of platform-independent Python modules for processing raw reads from high-throughput (next-generation) sequencing of lymphocyte repertoires.</td>
      <td>To update</td>
      <td>https://presto.readthedocs.io/</td>
      <td>Sequence Analysis</td>
      <td>presto</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/presto</td>
    </tr>
    <tr>
      <td></td>
      <td>pretext</td>
      <td>0.1.9</td>
      <td>0.1.9</td>
      <td>pretextmap</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Process genome contacts maps processing images.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/wtsi-hpag/PretextSnapshot</td>
      <td>Sequence Analysis</td>
      <td>suite_pretext</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pretext</td>
    </tr>
    <tr>
      <td></td>
      <td>prinseq</td>
      <td>@TOOL_VERSION+galaxy2</td>
      <td>0.20.4</td>
      <td>prinseq</td>
      <td>To update</td>
      <td>prinseq</td>
      <td>PRINSEQ</td>
      <td>Read pre-processing, Sequence trimming, Sequence contamination filtering</td>
      <td>Transcriptomics, Metagenomics, Genomics</td>
      <td>PRINSEQ is a tool for easy and rapid quality control and data processing of metagenomic and metatranscriptomic datasets</td>
      <td>PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.</td>
      <td>To update</td>
      <td>http://prinseq.sourceforge.net/manual.html</td>
      <td>Fastq Manipulation, Metagenomics</td>
      <td>prinseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prinseq/</td>
    </tr>
    <tr>
      <td></td>
      <td>progressivemauve</td>
      <td></td>
      <td>snapshot_2015_02_13</td>
      <td>progressivemauve</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mauve/ProgressiveMauve Multiple Sequence Aligner</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>progressivemauve</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/progressivemauve</td>
    </tr>
    <tr>
      <td></td>
      <td>prokka</td>
      <td>1.14.6</td>
      <td>1.14.6</td>
      <td>prokka</td>
      <td>Up-to-date</td>
      <td>prokka</td>
      <td>Prokka</td>
      <td>Gene prediction, Coding region prediction, Genome annotation</td>
      <td>Genomics, Model organisms, Virology</td>
      <td>Rapid annotation of prokaryotic genomes</td>
      <td>Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
      <td>Up-to-date</td>
      <td>http://github.com/tseemann/prokka</td>
      <td>Sequence Analysis</td>
      <td>prokka</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prokka/</td>
    </tr>
    <tr>
      <td></td>
      <td>prot-scriber</td>
      <td>0.1.4</td>
      <td>0.1.4</td>
      <td>prot-scriber</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Protein annotation of short human readable descriptions</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/usadellab/prot-scriber</td>
      <td>Proteomics</td>
      <td>prot_scriber</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/prot-scriber</td>
    </tr>
    <tr>
      <td></td>
      <td>proteinortho</td>
      <td>6.2.3</td>
      <td>6.3.0</td>
      <td>proteinortho</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Proteinortho is a tool to detect orthologous proteins/genes within different species.</td>
      <td></td>
      <td>To update</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
      <td>Proteomics</td>
      <td>proteinortho</td>
      <td>iuc</td>
      <td>https://gitlab.com/paulklemm_PHD/proteinortho</td>
    </tr>
    <tr>
      <td></td>
      <td>pureclip</td>
      <td>1.0.4</td>
      <td>1.3.1</td>
      <td>pureclip</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PureCLIP is an HMM based peak caller specifically designed for eCLIP/iCLIP data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/skrakau/PureCLIP</td>
      <td>Sequence Analysis, RNA, CLIP-seq</td>
      <td>pureclip</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pureclip</td>
    </tr>
    <tr>
      <td></td>
      <td>purge_dups</td>
      <td>1.2.6</td>
      <td>1.2.6</td>
      <td>purge_dups</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Purge haplotigs and overlaps in an assembly based on read depth</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/dfguan/purge_dups</td>
      <td>Assembly</td>
      <td>purge_dups</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/purge_dups</td>
    </tr>
    <tr>
      <td></td>
      <td>pycoqc</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>pycoqc</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QC metrics for ONT Basecalling</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/tleonardi/pycoQC</td>
      <td>Nanopore</td>
      <td>pycoqc</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pycoqc</td>
    </tr>
    <tr>
      <td></td>
      <td>pygenometracks</td>
      <td>3.8</td>
      <td>3.8</td>
      <td>pygenometracks</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>pyGenomeTracks: Standalone program and library to plot beautiful genome browser tracks.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/deeptools/pyGenomeTracks</td>
      <td>Visualization</td>
      <td>pygenometracks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pygenometracks</td>
    </tr>
    <tr>
      <td></td>
      <td>pysradb</td>
      <td>1.4.2</td>
      <td>2.2.0</td>
      <td>pysradb</td>
      <td>To update</td>
      <td>pysradb</td>
      <td>pysradb</td>
      <td>Deposition, Data retrieval</td>
      <td>Sequencing, Gene transcripts, Bioinformatics</td>
      <td>pysradb allows to retrieve metadata, such as run accession numbers, from SRA and ENA based on multiple criteria.</td>
      <td>Python package to query next-generation sequencing metadata and data from NCBI Sequence Read Archive.</td>
      <td>To update</td>
      <td>https://github.com/saketkc/pysradb</td>
      <td>Sequence Analysis</td>
      <td>pysradb_search</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/pysradb</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_add_on</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qiime_core</td>
      <td></td>
      <td>1.9.1</td>
      <td>qiime</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>QIIME to perform microbial community analysis</td>
      <td></td>
      <td>To update</td>
      <td>http://www.qiime.org</td>
      <td>Metagenomics</td>
      <td>qiime</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qiime/</td>
    </tr>
    <tr>
      <td></td>
      <td>qq_tools</td>
      <td>0.1.0</td>
      <td>0.1.4</td>
      <td>r-qqman</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://CRAN.R-project.org/package=qqman</td>
      <td>Visualization, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>qualimap</td>
      <td>2.2.2d</td>
      <td>2.2.2d</td>
      <td>qualimap</td>
      <td>Up-to-date</td>
      <td>qualimap</td>
      <td>QualiMap</td>
      <td>Sequencing quality control</td>
      <td>Data quality management</td>
      <td></td>
      <td>Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.</td>
      <td>Up-to-date</td>
      <td>http://qualimap.bioinfo.cipf.es/</td>
      <td>Sequence Analysis, Transcriptomics, SAM</td>
      <td>qualimap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/qualimap</td>
    </tr>
    <tr>
      <td></td>
      <td>quast</td>
      <td>5.2.0</td>
      <td>5.2.0</td>
      <td>quast</td>
      <td>Up-to-date</td>
      <td>quast</td>
      <td>QUAST</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>Sequence assembly</td>
      <td>Quast (Quality ASsessment Tool) evaluates genome assemblies.</td>
      <td>QUAST stands for QUality ASsessment Tool.  It evaluates a quality of genome assemblies by computing various metrics and providing nice reports.</td>
      <td>Up-to-date</td>
      <td>http://quast.bioinf.spbau.ru/</td>
      <td>Assembly</td>
      <td>quast</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quast</td>
    </tr>
    <tr>
      <td></td>
      <td>quickmerge</td>
      <td>0.3</td>
      <td>0.3</td>
      <td>quickmerge</td>
      <td>Up-to-date</td>
      <td>quickmerge</td>
      <td>quickmerge</td>
      <td>Genome assembly, Scaffolding, De-novo assembly, Genotyping</td>
      <td>Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype</td>
      <td>Merge long-read and hybrid assemblies to increase contiguity</td>
      <td>Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mahulchak/quickmerge</td>
      <td>Assembly</td>
      <td>quickmerge</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/quickmerge</td>
    </tr>
    <tr>
      <td></td>
      <td>ragtag</td>
      <td>2.1.0</td>
      <td>2.1.0</td>
      <td>ragtag</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reference-guided scaffolding of draft genomes tool.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/malonge/RagTag</td>
      <td>Assembly</td>
      <td>ragtag</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ragtag</td>
    </tr>
    <tr>
      <td></td>
      <td>rapidnj</td>
      <td>2.3.2</td>
      <td>2.3.2</td>
      <td>rapidnj</td>
      <td>Up-to-date</td>
      <td>rapidnj</td>
      <td>RapidNJ</td>
      <td>Phylogenetic tree generation</td>
      <td>Phylogeny</td>
      <td>Galaxy wrapper for the RapidNJ tool</td>
      <td>A tool for fast canonical neighbor-joining tree construction.</td>
      <td>Up-to-date</td>
      <td>https://birc.au.dk/software/rapidnj/</td>
      <td>Phylogenetics</td>
      <td>rapidnj</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rapidnj</td>
    </tr>
    <tr>
      <td></td>
      <td>raven</td>
      <td>1.8.0</td>
      <td>1.8.3</td>
      <td>raven-assembler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raven is a de novo genome assembler for long uncorrected reads.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/lbcb-sci/raven</td>
      <td>Assembly</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raven</td>
    </tr>
    <tr>
      <td></td>
      <td>raxml</td>
      <td>8.2.12</td>
      <td>8.2.13</td>
      <td>raxml</td>
      <td>To update</td>
      <td>raxml</td>
      <td>RAxML</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>Phylogenetics, Sequence analysis</td>
      <td>RAxML - A Maximum Likelihood based phylogenetic inference</td>
      <td>A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.</td>
      <td>To update</td>
      <td>http://www.exelixis-lab.org/web/software/raxml/</td>
      <td>Phylogenetics</td>
      <td>raxml</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/raxml</td>
    </tr>
    <tr>
      <td></td>
      <td>read_it_and_keep</td>
      <td>0.2.2</td>
      <td>0.3.0</td>
      <td>read-it-and-keep</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Rapid decontamination of SARS-CoV-2 sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GenomePathogenAnalysisService/read-it-and-keep</td>
      <td>Sequence Analysis</td>
      <td>read_it_and_keep</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/read-it-and-keep</td>
    </tr>
    <tr>
      <td></td>
      <td>recentrifuge</td>
      <td>1.12.1</td>
      <td>1.12.1</td>
      <td>recentrifuge</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>"With Recentrifuge, researchers can analyze results from taxonomic classifiers using interactive charts with emphasis on the confidence level of the classifications.In addition to contamination-subtracted samples.Recentrifuge provides shared and exclusive taxa per sample,thus enabling robust contamination removal and comparative analysis in environmental and clinical metagenomics."</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/khyox/recentrifuge</td>
      <td>Metagenomics</td>
      <td>recentrifuge</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/recentrifuge</td>
    </tr>
    <tr>
      <td></td>
      <td>red</td>
      <td>2018.09.10</td>
      <td>2018.09.10</td>
      <td>red</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Red (REpeat Detector)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/BioinformaticsToolsmith/Red</td>
      <td>Sequence Analysis</td>
      <td>red</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/red</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmasker</td>
      <td>4.1.5</td>
      <td>4.1.5</td>
      <td>repeatmasker</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://www.repeatmasker.org/</td>
      <td>Sequence Analysis</td>
      <td>repeat_masker</td>
      <td>bgruening</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmasker</td>
    </tr>
    <tr>
      <td></td>
      <td>repeatmodeler</td>
      <td>2.0.4</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>RepeatModeler - Model repetitive DNA</td>
      <td></td>
      <td>To update</td>
      <td>https://www.repeatmasker.org/RepeatModeler/</td>
      <td>Genome annotation</td>
      <td>repeatmodeler</td>
      <td>csbl</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/repeatmodeler</td>
    </tr>
    <tr>
      <td></td>
      <td>ribowaltz</td>
      <td>1.2.0</td>
      <td>1.2.0</td>
      <td>ribowaltz</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/LabTranslationalArchitectomics/riboWaltz</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ribowaltz</td>
    </tr>
    <tr>
      <td></td>
      <td>rnaquast</td>
      <td>2.2.3</td>
      <td>2.2.3</td>
      <td>rnaquast</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>rnaQuast (RNA Quality Assessment Tool) evaluates genome assemblies.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/rnaquast</td>
      <td>Assembly, RNA</td>
      <td>rnaquast</td>
      <td>iuc</td>
      <td>https://git.ufz.de/lehmanju/rnaquast</td>
    </tr>
    <tr>
      <td></td>
      <td>roary</td>
      <td>3.13.0</td>
      <td>3.13.0</td>
      <td>roary</td>
      <td>Up-to-date</td>
      <td>roary</td>
      <td>Roary</td>
      <td>Genome assembly</td>
      <td>DNA, Genomics, Mapping</td>
      <td>Roary the pangenome pipeline</td>
      <td>A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.</td>
      <td>Up-to-date</td>
      <td>https://sanger-pathogens.github.io/Roary/</td>
      <td>Sequence Analysis</td>
      <td>roary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/roary</td>
    </tr>
    <tr>
      <td></td>
      <td>rseqc</td>
      <td>5.0.1</td>
      <td>5.0.3</td>
      <td>rseqc</td>
      <td>To update</td>
      <td>rseqc</td>
      <td>RSeQC</td>
      <td>Data handling</td>
      <td>Sequencing</td>
      <td>an RNA-seq quality control package</td>
      <td>Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.</td>
      <td>To update</td>
      <td>https://code.google.com/p/rseqc/</td>
      <td>Convert Formats, Sequence Analysis, RNA, Transcriptomics, Visualization</td>
      <td>rseqc</td>
      <td>nilesh</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/rseqc</td>
    </tr>
    <tr>
      <td></td>
      <td>ruvseq</td>
      <td>1.26.0</td>
      <td>1.34.0</td>
      <td>bioconductor-ruvseq</td>
      <td>To update</td>
      <td>ruvseq</td>
      <td>RUVSeq</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, RNA-seq</td>
      <td>Remove Unwanted Variation from RNA-Seq Data</td>
      <td>This package implements the remove unwanted variation (RUV) methods for the normalization of RNA-Seq read counts between samples.</td>
      <td>To update</td>
      <td>https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>ruvseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/ruvseq</td>
    </tr>
    <tr>
      <td></td>
      <td>salsa2</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>salsa2</td>
      <td>Up-to-date</td>
      <td>SALSA</td>
      <td>SALSA</td>
      <td>Genome assembly, De-novo assembly, Scaffolding</td>
      <td>Sequence assembly, DNA binding sites, Mapping</td>
      <td>A tool to scaffold long read assemblies with Hi-C</td>
      <td>&gt; VERY_LOW CONFIDENCE! | &gt; CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch</td>
      <td>Up-to-date</td>
      <td>https://github.com/marbl/SALSA</td>
      <td>Assembly</td>
      <td>salsa</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/salsa2</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2formatter</td>
      <td>1.0</td>
      <td>1.0</td>
      <td>sarscov2formatter</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2formatter custom script</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/nickeener/sarscov2formatter</td>
      <td>Sequence Analysis</td>
      <td>sarscov2formatter</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2formatter</td>
    </tr>
    <tr>
      <td></td>
      <td>sarscov2summary</td>
      <td>0.1</td>
      <td>0.5</td>
      <td>sarscov2summary</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>sarscov2summary custom script</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nickeener/sarscov2summary</td>
      <td>Sequence Analysis</td>
      <td>sarscov2summary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sarscov2summary</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>@galaxy_version@</td>
      <td>1.7.2</td>
      <td>scanpy</td>
      <td>To update</td>
      <td>scanpy</td>
      <td>SCANPY</td>
      <td>Differential gene expression analysis</td>
      <td>Gene expression, Cell biology, Genetics</td>
      <td>Scanpy – Single-Cell Analysis in Python</td>
      <td>Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.</td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>scanpy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scanpy/</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>1.22.0</td>
      <td>1.28.0</td>
      <td>bioconductor-scater</td>
      <td>To update</td>
      <td>scater</td>
      <td>scater</td>
      <td>Read pre-processing, Sequencing quality control, Sequence visualisation</td>
      <td>RNA-seq, Quality affairs, Molecular genetics</td>
      <td>Scater (Single-Cell Analysis Toolkit for gene Expression data in R) is acollection of tools for doing various analyses of single-cell RNA-seq geneexpression data, with a focus on quality control and visualization.</td>
      <td>Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/scater/</td>
      <td>Transcriptomics, RNA, Visualization</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scater</td>
    </tr>
    <tr>
      <td></td>
      <td>schicexplorer</td>
      <td>4</td>
      <td>7</td>
      <td>schicexplorer</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scHiCExplorer: Set of programs to process, analyze and visualize scHi-C data.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/joachimwolff/schicexplorer</td>
      <td>Sequence Analysis, Transcriptomics, Visualization</td>
      <td>schicexplorer</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/schicexplorer</td>
    </tr>
    <tr>
      <td></td>
      <td>scikit-bio</td>
      <td>0.4.2</td>
      <td>0.4.2</td>
      <td>scikit-bio</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scikit-bio: an open-source, BSD-licensed, python package providing data structures, algorithms, and educational resources for bioinformatics</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://scikit-bio.org/</td>
      <td>Sequence Analysis</td>
      <td>scikit_bio</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scikit_bio</td>
    </tr>
    <tr>
      <td></td>
      <td>scoary</td>
      <td>1.6.16</td>
      <td>1.6.16</td>
      <td>scoary</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scoary calculates the assocations between all genes in the accessory genome and the traits.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/AdmiralenOla/Scoary</td>
      <td>Metagenomics</td>
      <td>scoary</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scoary</td>
    </tr>
    <tr>
      <td></td>
      <td>scpipe</td>
      <td>1.0.0+galaxy2</td>
      <td>2.0.0</td>
      <td>bioconductor-scpipe</td>
      <td>To update</td>
      <td>scpipe</td>
      <td>scPipe</td>
      <td>Genome annotation, Validation, Alignment, Visualisation</td>
      <td>Gene expression, RNA-seq, Sequencing</td>
      <td>A flexible preprocessing pipeline for single-cell RNA-sequencing data</td>
      <td>A preprocessing pipeline for single cell RNA-seq data that starts from the fastq files and produces a gene count matrix with associated quality control information. It can process fastq data generated by CEL-seq, MARS-seq, Drop-seq, Chromium 10x and SMART-seq protocols.</td>
      <td>To update</td>
      <td>http://bioconductor.org/packages/release/bioc/html/scPipe.html</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>scpipe</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/scpipe</td>
    </tr>
    <tr>
      <td></td>
      <td>semibin</td>
      <td>1.5.1</td>
      <td>2.0.1</td>
      <td>semibin</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SemiBin: Semi-supervised Metagenomic Binning Using Siamese Neural Networks</td>
      <td></td>
      <td>To update</td>
      <td>https://semibin.readthedocs.io/en/latest/</td>
      <td>Metagenomics</td>
      <td>semibin</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/semibin</td>
    </tr>
    <tr>
      <td></td>
      <td>seq2hla</td>
      <td>2.3</td>
      <td>2.3</td>
      <td>seq2hla</td>
      <td>Up-to-date</td>
      <td>seq2hla</td>
      <td>Seq2HLA</td>
      <td>Read mapping, Genetic variation analysis</td>
      <td>Transcriptomics, Mapping</td>
      <td>Precision HLA typing and expression from RNAseq data</td>
      <td>seq2HLA is a computational tool to determine Human Leukocyte Antigen (HLA) directly from existing and future short RNA-Seq reads. It takes standard RNA-Seq sequence reads in fastq format as input, uses a bowtie index comprising known HLA alleles and outputs the most likely HLA class I and class II types, a p-value for each call, and the expression of each class.</td>
      <td>Up-to-date</td>
      <td>https://github.com/TRON-Bioinformatics/seq2HLA</td>
      <td>Sequence Analysis</td>
      <td>seq2hla</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seq2hla</td>
    </tr>
    <tr>
      <td></td>
      <td>seqcomplexity</td>
      <td>0.1.2</td>
      <td>0.1.2</td>
      <td>seqcomplexity</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sequence complexity for raw reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/stevenweaver/seqcomplexity</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/stephenshank/tools-iuc/tree/seqcomplexity/tools/seqcomplexity/</td>
    </tr>
    <tr>
      <td></td>
      <td>seqkit</td>
      <td>2.3.1</td>
      <td>2.5.1</td>
      <td>seqkit</td>
      <td>To update</td>
      <td>seqkit</td>
      <td>seqkit</td>
      <td>DNA transcription, Sequence trimming, DNA translation, Sequence conversion</td>
      <td>Database management, Sequence analysis</td>
      <td>A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
      <td>FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.</td>
      <td>To update</td>
      <td>https://bioinf.shenwei.me/seqkit/</td>
      <td>Sequence Analysis</td>
      <td>seqkit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqkit</td>
    </tr>
    <tr>
      <td></td>
      <td>seqtk</td>
      <td>1.3</td>
      <td>1.4</td>
      <td>seqtk</td>
      <td>To update</td>
      <td>seqtk</td>
      <td>seqtk</td>
      <td>Data handling, Sequence file editing</td>
      <td>Data management</td>
      <td>Toolkit for processing sequences in FASTA/Q formats</td>
      <td>A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.</td>
      <td>To update</td>
      <td>https://github.com/lh3/seqtk</td>
      <td>Sequence Analysis</td>
      <td>seqtk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqtk</td>
    </tr>
    <tr>
      <td></td>
      <td>seqwish</td>
      <td>0.7.5</td>
      <td>0.7.9</td>
      <td>seqwish</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Alignment to variation graph inducer</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ekg/seqwish</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seqwish/</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>4.3.0.1</td>
      <td>3.0.2</td>
      <td>r-seurat</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A toolkit for quality control, analysis, and exploration of single cell RNA sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/satijalab/seurat</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>seurat</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/seurat</td>
    </tr>
    <tr>
      <td></td>
      <td>shasta</td>
      <td>0.6.0</td>
      <td>0.11.1</td>
      <td>shasta</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fast de novo assembly of long read sequencing data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/chanzuckerberg/shasta</td>
      <td>Assembly, Nanopore</td>
      <td>shasta</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shasta</td>
    </tr>
    <tr>
      <td></td>
      <td>shorah</td>
      <td>1.1.3</td>
      <td>1.99.2</td>
      <td>shorah</td>
      <td>To update</td>
      <td>shorah</td>
      <td>ShoRAH</td>
      <td>Haplotype mapping, Variant calling</td>
      <td>Metagenomics, Sequencing, Genetics</td>
      <td>Reconstruct haplotypes using ShoRAH in amplicon mode</td>
      <td>Inference of a population from a set of short reads. The package contains programs that support mapping of reads to a reference genome, correcting sequencing errors by locally clustering reads in small windows of the alignment, reconstructing a minimal set of global haplotypes that explain the reads, and estimating the frequencies of the inferred haplotypes.</td>
      <td>To update</td>
      <td>https://github.com/cbg-ethz/shorah/blob/master/README.md</td>
      <td>Sequence Analysis</td>
      <td>shorah_amplicon</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shorah</td>
    </tr>
    <tr>
      <td></td>
      <td>shovill</td>
      <td>1.1.0</td>
      <td>1.1.0</td>
      <td>shovill</td>
      <td>Up-to-date</td>
      <td>shovill</td>
      <td>shovill</td>
      <td>Genome assembly</td>
      <td>Genomics, Microbiology, Sequence assembly</td>
      <td>Faster de novo assembly pipeline based around Spades</td>
      <td>Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.</td>
      <td>Up-to-date</td>
      <td>https://github.com/tseemann/shovill</td>
      <td>Assembly</td>
      <td>shovill</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/shovill</td>
    </tr>
    <tr>
      <td></td>
      <td>sickle</td>
      <td>1.33.2</td>
      <td>1.33</td>
      <td>sickle-trim</td>
      <td>To update</td>
      <td>sickle</td>
      <td>sickle</td>
      <td>Sequence trimming</td>
      <td>Data quality management</td>
      <td>A windowed adaptive trimming tool for FASTQ files using quality</td>
      <td>A  tool that uses sliding windows along with quality and length thresholds to determine when quality is sufficiently low to trim the 3'-end of reads and also determines when the quality is sufficiently high enough to trim the 5'-end of reads.</td>
      <td>To update</td>
      <td>https://github.com/najoshi/sickle</td>
      <td>Fastq Manipulation, Sequence Analysis</td>
      <td>sickle</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sickle</td>
    </tr>
    <tr>
      <td></td>
      <td>sina</td>
      <td>1.7.2</td>
      <td>1.7.2</td>
      <td>sina</td>
      <td>Up-to-date</td>
      <td>sina</td>
      <td>SINA</td>
      <td>Sequence alignment analysis, Multiple sequence alignment, Taxonomic classification, Structure-based sequence alignment</td>
      <td>Sequencing, RNA, Nucleic acid structure analysis, Taxonomy, Sequence analysis, Taxonomy</td>
      <td>SINA reference based multiple sequence alignment</td>
      <td>Aligns and optionally taxonomically classifies your rRNA gene sequences.Reference based multiple sequence alignment</td>
      <td>Up-to-date</td>
      <td>https://sina.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>sina</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sina</td>
    </tr>
    <tr>
      <td></td>
      <td>sinto</td>
      <td>0.9.0</td>
      <td>0.10.0</td>
      <td>sinto</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sinto single-cell analysis tools</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/timoast/sinto</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sinto</td>
    </tr>
    <tr>
      <td></td>
      <td>slamdunk</td>
      <td>0.4.3</td>
      <td>0.4.3</td>
      <td>slamdunk</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Slamdunk maps and quantifies SLAMseq reads</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://t-neumann.github.io/slamdunk</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Next Gen Mappers</td>
      <td>slamdunk</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/slamdunk</td>
    </tr>
    <tr>
      <td></td>
      <td>sleuth</td>
      <td>0.30.1</td>
      <td>0.30.1</td>
      <td>r-sleuth</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Sleuth is a program for differential analysis of RNA-Seq data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pachterlab/sleuth</td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>sleuth</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/sleuth</td>
    </tr>
    <tr>
      <td></td>
      <td>smallgenomeutilities</td>
      <td>0.4.0</td>
      <td>0.4.0</td>
      <td>smallgenomeutilities</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Set of utilities for manipulating small viral genome data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/cbg-ethz/smallgenomeutilities</td>
      <td>Sequence Analysis</td>
      <td>smallgenomeutilities</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/smallgenomeutilities</td>
    </tr>
    <tr>
      <td></td>
      <td>smudgeplot</td>
      <td>0.2.5</td>
      <td>0.2.5</td>
      <td>smudgeplot</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Inference of ploidy and heterozygosity structure using whole genome sequencing</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/KamilSJaron/smudgeplot</td>
      <td>Assembly</td>
      <td>smudgeplot</td>
      <td>galaxy-australia</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>snap</td>
      <td>2013_11_29</td>
      <td>2013_11_29</td>
      <td>snap</td>
      <td>Up-to-date</td>
      <td>snap</td>
      <td>SNAP</td>
      <td>Gene prediction</td>
      <td>DNA, DNA polymorphism, Genetics</td>
      <td>SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes.</td>
      <td>The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.</td>
      <td>Up-to-date</td>
      <td>https://github.com/KorfLab/SNAP</td>
      <td>Sequence Analysis</td>
      <td>snap</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snap</td>
    </tr>
    <tr>
      <td></td>
      <td>sniffles</td>
      <td>1.0.12</td>
      <td>2.2</td>
      <td>sniffles</td>
      <td>To update</td>
      <td>sniffles</td>
      <td>Sniffles</td>
      <td>Sequence analysis, Structural variation detection</td>
      <td>DNA structural variation, Sequencing</td>
      <td>Galaxy wrapper for sniffles</td>
      <td>An algorithm for structural variation detection from third generation sequencing alignment.</td>
      <td>To update</td>
      <td>https://github.com/fritzsedlazeck/Sniffles</td>
      <td>Sequence Analysis</td>
      <td>sniffles</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>snipit</td>
      <td>1.0.7</td>
      <td>1.1.2</td>
      <td>snipit</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Summarise snps relative to a reference sequence</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/aineniamh/snipit</td>
      <td>Variant Analysis, Sequence Analysis</td>
      <td>snipit</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snipit</td>
    </tr>
    <tr>
      <td></td>
      <td>snippy</td>
      <td></td>
      <td>4.6.0</td>
      <td>snippy</td>
      <td>To update</td>
      <td>snippy</td>
      <td>snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>Genomics, Model organisms, DNA polymorphism, Phylogenetics</td>
      <td>Contains the snippy tool for characterising microbial snps</td>
      <td>Rapid haploid variant calling and core SNP phylogeny generation.</td>
      <td>To update</td>
      <td>https://github.com/tseemann/snippy</td>
      <td>Sequence Analysis</td>
      <td>snippy</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/snippy</td>
    </tr>
    <tr>
      <td></td>
      <td>socru</td>
      <td>2.1.7</td>
      <td>2.2.4</td>
      <td>socru</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Order and orientation of complete bacterial genomes</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/quadram-institute-bioscience/socru</td>
      <td>Sequence Analysis</td>
      <td>socru</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/socru</td>
    </tr>
    <tr>
      <td></td>
      <td>sonneityping</td>
      <td>20210201</td>
      <td>20210201</td>
      <td>sonneityping</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Scripts for parsing Mykrobe predict results for Shigella sonnei.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/katholt/sonneityping</td>
      <td>Sequence Analysis</td>
      <td>sonneityping</td>
      <td>iuc</td>
      <td>https://github.com/katholt/sonneityping</td>
    </tr>
    <tr>
      <td></td>
      <td>spades</td>
      <td>3.15.5</td>
      <td>3.15.5</td>
      <td>spades</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SPAdes is an assembly toolkit containing various assembly pipelines. It implements the following 4 stages: assembly graph construction, k-bimer adjustment, construction of paired assembly graph and contig construction.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/ablab/spades</td>
      <td>Assembly, RNA, Metagenomics</td>
      <td>spades</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/spades</td>
    </tr>
    <tr>
      <td></td>
      <td>spaln</td>
      <td>2.4.9</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Spaln (space-efficient spliced alignment) maps and aligns a set of cDNA or protein sequences onto a whole genomic sequence.</td>
      <td></td>
      <td>To update</td>
      <td>http://www.genome.ist.i.kyoto-u.ac.jp/~aln_user/spaln/</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>spaln</td>
      <td>iuc</td>
      <td>https://github.com/ogotoh/spaln</td>
    </tr>
    <tr>
      <td></td>
      <td>spotyping</td>
      <td>2.1</td>
      <td>2.1</td>
      <td>spotyping</td>
      <td>Up-to-date</td>
      <td>spotyping</td>
      <td>SpoTyping</td>
      <td>Variant pattern analysis</td>
      <td>Microbiology, Sequencing, Sequence composition, complexity and repeats, Genetic variation</td>
      <td>SpoTyping allows fast and accurate in silico Mycobacterium spoligotyping from sequence reads</td>
      <td>Fast and accurate in silico Mycobacterium spoligotyping from sequence reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0</td>
      <td>Sequence Analysis</td>
      <td>spotyping</td>
      <td>iuc</td>
      <td>https://github.com/xiaeryu/SpoTyping-v2.0/tree/master/SpoTyping-v2.0-commandLine</td>
    </tr>
    <tr>
      <td></td>
      <td>srst2</td>
      <td>0.2.0</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SRST2 Short Read Sequence Typing for Bacterial Pathogens</td>
      <td></td>
      <td>To update</td>
      <td>http://katholt.github.io/srst2/</td>
      <td>Metagenomics</td>
      <td>srst2</td>
      <td>iuc</td>
      <td>https://github.com/katholt/srst2</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks</td>
      <td></td>
      <td>2.65</td>
      <td>stacks</td>
      <td>To update</td>
      <td>stacks</td>
      <td>Stacks</td>
      <td>Data handling</td>
      <td>Mapping, Population genetics</td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td>Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.</td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td>stacks</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks</td>
    </tr>
    <tr>
      <td></td>
      <td>stacks2</td>
      <td>2.55</td>
      <td>2.65</td>
      <td>stacks</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Stacks is a software pipeline for building loci from short-read sequences, such as RAD-seq</td>
      <td></td>
      <td>To update</td>
      <td>http://catchenlab.life.illinois.edu/stacks/</td>
      <td>Sequence Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/stacks2</td>
    </tr>
    <tr>
      <td></td>
      <td>star_fusion</td>
      <td>0.5.4-3+galaxy1</td>
      <td>1.12.0</td>
      <td>star-fusion</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>STAR Fusion detects fusion genes in RNA-Seq data after running RNA-STAR</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td>star_fusion</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/star_fusion</td>
    </tr>
    <tr>
      <td></td>
      <td>structure</td>
      <td>2.3.4</td>
      <td>2.3.4</td>
      <td>structure</td>
      <td>Up-to-date</td>
      <td>structure</td>
      <td>Structure</td>
      <td>Genetic variation analysis</td>
      <td>Population genetics</td>
      <td>for using multi-locus genotype data to investigate population structure.</td>
      <td>The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.</td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structure</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structure</td>
    </tr>
    <tr>
      <td></td>
      <td>structureharvester</td>
      <td>0.6.94</td>
      <td>0.6.94</td>
      <td>structureharvester</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>for parsing STRUCTURE outputs and for performing the Evanno method</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Phylogenetics, Variant Analysis</td>
      <td>structureharvester</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/structureharvester</td>
    </tr>
    <tr>
      <td></td>
      <td>syndiva</td>
      <td>1.0</td>
      <td>1.2.4</td>
      <td>clustalo</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SynDivA was developed to analyze the diversity of synthetic libraries of a Fibronectin domain.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>syndiva</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/SynDivA</td>
    </tr>
    <tr>
      <td></td>
      <td>tasmanian_mismatch</td>
      <td>1.0.7</td>
      <td>1.0.7</td>
      <td>tasmanian-mismatch</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Analysis of positional mismatches</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>tasmanian_mismatch</td>
      <td>iuc</td>
      <td>https://github.com/nebiolabs/tasmanian-mismatch</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_filter_refseq</td>
      <td>0.3.0</td>
      <td>1.0.7</td>
      <td>rust-ncbitaxonomy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter RefSeq by taxonomy</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/pvanheus/ncbitaxonomy</td>
      <td>Sequence Analysis, Genome annotation</td>
      <td>taxonomy_filter_refseq</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc</td>
    </tr>
    <tr>
      <td></td>
      <td>taxonomy_krona_chart</td>
      <td>2.7.1+galaxy0</td>
      <td>2.8.1</td>
      <td>krona</td>
      <td>To update</td>
      <td>krona</td>
      <td>Krona</td>
      <td>Visualisation</td>
      <td>Metagenomics</td>
      <td>Krona pie chart from taxonomic profile</td>
      <td>Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).</td>
      <td>To update</td>
      <td>http://sourceforge.net/projects/krona/</td>
      <td>Assembly</td>
      <td>taxonomy_krona_chart</td>
      <td>crs4</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/taxonomy_krona_chart</td>
    </tr>
    <tr>
      <td></td>
      <td>tb-profiler</td>
      <td>4.4.1</td>
      <td>5.0.1</td>
      <td>tb-profiler</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Processes M. tuberculosis sequence data to infer strain type and identify known drug resistance markers.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/jodyphelan/TBProfiler</td>
      <td>Sequence Analysis</td>
      <td>tbprofiler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tools/tb-profiler</td>
    </tr>
    <tr>
      <td></td>
      <td>tbl2gff3</td>
      <td>1.2</td>
      <td>0.6.6</td>
      <td>bcbiogff</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Table to GFF3</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
      <td>Convert Formats, Sequence Analysis</td>
      <td>tbl2gff3</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tbl2gff3</td>
    </tr>
    <tr>
      <td></td>
      <td>te_finder</td>
      <td>1.0.1</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Transposable element insertions finder</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/VistaSohrab/TEfinder</td>
      <td>Sequence Analysis</td>
      <td>te_finder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/te_finder/</td>
    </tr>
    <tr>
      <td></td>
      <td>telescope</td>
      <td>1.0.3</td>
      <td>1.0.3</td>
      <td>telescope</td>
      <td>Up-to-date</td>
      <td>Telescope-expression</td>
      <td>Telescope</td>
      <td>Essential dynamics, Sequence trimming, RNA-Seq quantification, Expression analysis, Read mapping</td>
      <td>RNA-Seq, Transcriptomics, Mapping, Gene transcripts, Sequence assembly</td>
      <td>Single locus resolution of Transposable ELEment expression.</td>
      <td>Telescope is a tool for the characterization of the retrotranscriptome by accurate estimation of transposable element expression and the quantification of transposable element expression using RNA-seq.It can be used for Statistical Performance of TE Quantification Methods.All scripts needed to examine the sensitivity and biases of computational approaches for quantifying TE expression: 1) unique counts, 2) best counts, 3) RepEnrich, 4) TEtranscripts, 5) RSEM, 6) SalmonTE, and 7) Telescope.</td>
      <td>Up-to-date</td>
      <td>https://github.com/mlbendall/telescope/</td>
      <td>Genome annotation</td>
      <td>telescope_assign</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/telescope</td>
    </tr>
    <tr>
      <td></td>
      <td>tetoolkit</td>
      <td>2.2.3</td>
      <td>2.2.3</td>
      <td>tetranscripts</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The TEToolkit suite improves the bioinformatic analysis of repetitive sequences, particularly transposable elements, in order to elucidate novel (and previously ignored) biological insights of their functions in development and diseases.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>http://hammelllab.labsites.cshl.edu/software/</td>
      <td>Sequence Analysis</td>
      <td>tetoolkit</td>
      <td>iuc</td>
      <td>https://github.com/mhammell-laboratory/TEtranscripts</td>
    </tr>
    <tr>
      <td></td>
      <td>tetyper</td>
      <td>1.1</td>
      <td>1.1</td>
      <td>tetyper</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Type a specific transposable element (TE) of interest from paired-end sequencing data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/aesheppard/TETyper</td>
      <td>Sequence Analysis</td>
      <td>tetyper</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tetyper</td>
    </tr>
    <tr>
      <td></td>
      <td>tn93</td>
      <td>1.0.6</td>
      <td>1.0.9</td>
      <td>tn93</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute distances between sequences</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/veg/tn93/</td>
      <td>Sequence Analysis</td>
      <td>tn93</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tn93/</td>
    </tr>
    <tr>
      <td></td>
      <td>transdecoder</td>
      <td>5.5.0</td>
      <td>5.7.1</td>
      <td>transdecoder</td>
      <td>To update</td>
      <td>TransDecoder</td>
      <td>TransDecoder</td>
      <td>Coding region prediction, de Novo sequencing, De-novo assembly</td>
      <td>Genomics, Gene transcripts, RNA-Seq, Gene expression, Sequence assembly, Whole genome sequencing</td>
      <td>TransDecoder finds coding regions within transcripts</td>
      <td>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</td>
      <td>To update</td>
      <td>https://transdecoder.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>transdecoder</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transdecoder</td>
    </tr>
    <tr>
      <td></td>
      <td>transit</td>
      <td>3.0.2</td>
      <td>3.2.3</td>
      <td>transit</td>
      <td>To update</td>
      <td>transit</td>
      <td>TRANSIT</td>
      <td>Transposon prediction</td>
      <td>DNA, Sequencing, Mobile genetic elements</td>
      <td>TRANSIT</td>
      <td>A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.</td>
      <td>To update</td>
      <td>https://github.com/mad-lab/transit/</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transit/</td>
    </tr>
    <tr>
      <td></td>
      <td>transtermhp</td>
      <td></td>
      <td>2.09</td>
      <td>transtermhp</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Finds rho-independent transcription terminators in bacterial genomes</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>transtermhp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/transtermhp</td>
    </tr>
    <tr>
      <td></td>
      <td>trinity</td>
      <td>2.15.1</td>
      <td>2.15.1</td>
      <td>trinity</td>
      <td>Up-to-date</td>
      <td>trinity</td>
      <td>Trinity</td>
      <td>Transcriptome assembly</td>
      <td>Transcriptomics, Gene expression, Gene transcripts</td>
      <td>Trinity represents a method for the efficient and robust de novo reconstruction of transcriptomes from RNA-seq datahttps://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/trinityrnaseq/trinityrnaseq</td>
      <td>Transcriptomics, RNA</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinity</td>
    </tr>
    <tr>
      <td></td>
      <td>trinotate</td>
      <td>3.2.2</td>
      <td>4.0.2</td>
      <td>trinotate</td>
      <td>To update</td>
      <td>trinotate</td>
      <td>Trinotate</td>
      <td>Gene functional annotation</td>
      <td>Gene expression, Transcriptomics</td>
      <td>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of de novo transcriptomes.</td>
      <td>Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.</td>
      <td>To update</td>
      <td>https://trinotate.github.io/</td>
      <td>Transcriptomics, RNA</td>
      <td>trinotate</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trinotate</td>
    </tr>
    <tr>
      <td></td>
      <td>trycycler</td>
      <td>0.5.4</td>
      <td>0.5.4</td>
      <td>trycycler</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Trycycler toolkit wrappers</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Trycycler</td>
      <td>Assembly</td>
      <td>trycycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/trycycler</td>
    </tr>
    <tr>
      <td></td>
      <td>tsebra</td>
      <td>1.1.1</td>
      <td>1.1.2</td>
      <td>tsebra</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>This tool has been developed to combine BRAKER predictions.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Gaius-Augustus/TSEBRA</td>
      <td>Genome annotation</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/tsebra</td>
    </tr>
    <tr>
      <td></td>
      <td>ucsc_blat</td>
      <td>377</td>
      <td>445</td>
      <td>ucsc-blat</td>
      <td>To update</td>
      <td>blat</td>
      <td>BLAT</td>
      <td>Sequence alignment</td>
      <td>Sequence analysis</td>
      <td>Standalone blat sequence search command line tool</td>
      <td>Fast, accurate spliced alignment of DNA sequences.</td>
      <td>To update</td>
      <td>http://genome.ucsc.edu/goldenPath/help/blatSpec.html</td>
      <td>Sequence Analysis</td>
      <td>ucsc_blat</td>
      <td>yating-l</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>umi_tools</td>
      <td>1.1.2</td>
      <td>1.1.4</td>
      <td>umi_tools</td>
      <td>To update</td>
      <td>umi-tools</td>
      <td>UMI-tools</td>
      <td>Sequencing quality control</td>
      <td>NGS, Sequence sites, features and motifs, Quality affairs</td>
      <td>UMI-tools extract - Extract UMIs from fastq</td>
      <td>Tools for handling Unique Molecular Identifiers in NGS data sets.</td>
      <td>To update</td>
      <td>https://github.com/CGATOxford/UMI-tools</td>
      <td>Sequence Analysis, Transcriptomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/umi_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>unicycler</td>
      <td>0.5.0</td>
      <td>0.5.0</td>
      <td>unicycler</td>
      <td>Up-to-date</td>
      <td>unicycler</td>
      <td>Unicycler</td>
      <td>Genome assembly, Aggregation</td>
      <td>Microbiology, Genomics, Sequencing, Sequence assembly</td>
      <td>Unicycler is a hybrid assembly pipeline for bacterial genomes.</td>
      <td>A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.</td>
      <td>Up-to-date</td>
      <td>https://github.com/rrwick/Unicycler</td>
      <td>Assembly</td>
      <td>unicycler</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/unicycler</td>
    </tr>
    <tr>
      <td></td>
      <td>usher</td>
      <td>0.2.1</td>
      <td>0.6.2</td>
      <td>usher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>UShER toolkit wrappers</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/yatisht/usher</td>
      <td>Phylogenetics</td>
      <td>usher</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/usher</td>
    </tr>
    <tr>
      <td></td>
      <td>valet</td>
      <td></td>
      <td>1.0</td>
      <td>valet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A pipeline for detecting mis-assemblies in metagenomic assemblies.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/VALET</td>
      <td>Metagenomics</td>
      <td>valet</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/valet</td>
    </tr>
    <tr>
      <td></td>
      <td>vapor</td>
      <td>1.0.2</td>
      <td>1.0.2</td>
      <td>vapor</td>
      <td>Up-to-date</td>
      <td>vapor</td>
      <td>VAPOR</td>
      <td>Data retrieval, De-novo assembly, Read mapping</td>
      <td>Whole genome sequencing, Mapping, Sequence assembly</td>
      <td>Classify Influenza samples from raw short read sequence data</td>
      <td>VAPOR is a tool for classification of Influenza samples from raw short read sequence data for downstream bioinformatics analysis. VAPOR is provided with a fasta file of full-length sequences (&gt; 20,000) for a given segment, a set of reads, and attempts to retrieve a reference that is closest to the sample strain.</td>
      <td>Up-to-date</td>
      <td>https://github.com/connor-lab/vapor</td>
      <td>Sequence Analysis</td>
      <td>vapor</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vapor</td>
    </tr>
    <tr>
      <td></td>
      <td>vegan</td>
      <td>2.4-3</td>
      <td>2.3_4</td>
      <td>r-vegan</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/package=vegan</td>
      <td>Metagenomics</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vegan/</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet</td>
      <td></td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvet</td>
      <td>Velvet</td>
      <td>Formatting, De-novo assembly</td>
      <td>Sequence assembly</td>
      <td>de novo genomic assembler specially designed for short read sequencing technologies</td>
      <td>A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.</td>
      <td>To update</td>
      <td>https://www.ebi.ac.uk/~zerbino/velvet/</td>
      <td>Assembly</td>
      <td>velvet</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet</td>
    </tr>
    <tr>
      <td></td>
      <td>velvet_optimiser</td>
      <td>2.2.6+galaxy2</td>
      <td>1.2.10</td>
      <td>velvet</td>
      <td>To update</td>
      <td>velvetoptimiser</td>
      <td>VelvetOptimiser</td>
      <td>Optimisation and refinement, Sequence assembly</td>
      <td>Genomics, Sequence assembly</td>
      <td>Automatically optimize Velvet assemblies</td>
      <td>This tool is designed to run as a wrapper script for the Velvet assembler (Daniel Zerbino, EBI UK) and to assist with optimising the assembly.</td>
      <td>To update</td>
      <td></td>
      <td>Assembly</td>
      <td>velvetoptimiser</td>
      <td>simon-gladman</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvetoptimiser</td>
    </tr>
    <tr>
      <td></td>
      <td>verkko</td>
      <td>1.3.1</td>
      <td>1.4.1</td>
      <td>verkko</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Telomere-to-telomere assembly pipeline</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/marbl/verkko</td>
      <td>Assembly</td>
      <td>verkko</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/verkko</td>
    </tr>
    <tr>
      <td></td>
      <td>vg</td>
      <td>1.23.0</td>
      <td>1.52.0</td>
      <td>vg</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Variation graph data structures, interchange formats, alignment, genotyping, and variant calling methods</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vgteam/vg</td>
      <td>Sequence Analysis, Variant Analysis</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vg</td>
    </tr>
    <tr>
      <td></td>
      <td>volcanoplot</td>
      <td>0.0.5</td>
      <td>2.2.1</td>
      <td>r-ggplot2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to create a Volcano Plot</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Visualization, Transcriptomics, Statistics</td>
      <td>volcanoplot</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/volcanoplot</td>
    </tr>
    <tr>
      <td></td>
      <td>vsearch</td>
      <td>2.8.3</td>
      <td>2.23.0</td>
      <td>vsearch</td>
      <td>To update</td>
      <td>vsearch</td>
      <td>VSEARCH</td>
      <td>DNA mapping, Chimera detection</td>
      <td>Metagenomics, Sequence analysis</td>
      <td>VSEARCH including searching, clustering, chimera detection, dereplication, sorting, masking and shuffling of sequences.</td>
      <td>High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.</td>
      <td>To update</td>
      <td>https://github.com/torognes/vsearch</td>
      <td>Sequence Analysis</td>
      <td>vsearch</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsearch</td>
    </tr>
    <tr>
      <td></td>
      <td>vsnp</td>
      <td>3.0.6</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The vSNP tools are critical components of several workflows that validate SNPs and produce annotatedSNP tables and corresponding phylogenetic trees.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/USDA-VS/vSNP</td>
      <td>Sequence Analysis</td>
      <td>vsnp</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/vsnp</td>
    </tr>
    <tr>
      <td></td>
      <td>weather_app</td>
      <td>0.1.2</td>
      <td></td>
      <td>curl</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>provides simple weather in text format</td>
      <td></td>
      <td>To update</td>
      <td>http://wttr.in/</td>
      <td>Visualization, Web Services</td>
      <td>simpleweather</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/weather_app</td>
    </tr>
    <tr>
      <td></td>
      <td>yahs</td>
      <td>1.2a.2</td>
      <td>1.2a.2</td>
      <td>yahs</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Yet Another Hi-C scaffolding tool</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/c-zhou/yahs</td>
      <td>Assembly</td>
      <td>yahs</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/yahs</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Operate on and transform BAM datasets in various ways using bamtools</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_filter</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter BAM datasets on various attributes using bamtools filter</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td>bamtools_filter</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_filter</td>
    </tr>
    <tr>
      <td></td>
      <td>bamtools_split</td>
      <td>2.5.2</td>
      <td>2.5.2</td>
      <td>bamtools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/pezmaster31/bamtools</td>
      <td>Sequence Analysis, SAM</td>
      <td></td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/bamtools/bamtools_split</td>
    </tr>
    <tr>
      <td></td>
      <td>biotradis</td>
      <td>1.4.5</td>
      <td>1.4.5</td>
      <td>biotradis</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Bio-Tradis is a tool suite dedicated to essentiality analyses with TraDis data.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://www.sanger.ac.uk/science/tools/bio-tradis</td>
      <td>Genome annotation</td>
      <td>biotradis</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/biotradis</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken</td>
      <td></td>
      <td>1.1.1</td>
      <td>kraken</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken is a system for assigning taxonomic labels to short DNAsequences, usually obtained through metagenomic studies. Previous attempts by otherbioinformatics software to accomplish this task have often used sequence alignmentor machine learning techniques that were quite slow, leading to the developmentof less sensitive but much faster abundance estimation programs. Kraken aims toachieve high sensitivity and high speed by utilizing exact alignments of k-mersand a novel classification algorithm.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken</td>
      <td>devteam</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/</td>
    </tr>
    <tr>
      <td></td>
      <td>kraken2</td>
      <td>2.1.1</td>
      <td>2.1.3</td>
      <td>kraken2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Kraken2 for taxonomic designation.</td>
      <td></td>
      <td>To update</td>
      <td>http://ccb.jhu.edu/software/kraken/</td>
      <td>Metagenomics</td>
      <td>kraken2</td>
      <td>iuc</td>
      <td>https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken2/kraken2/</td>
    </tr>
    <tr>
      <td></td>
      <td>ThermoRawFileParser</td>
      <td>1.3.4</td>
      <td>1.4.3</td>
      <td>thermorawfileparser</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Thermo RAW file converter</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/ThermoRawFileParser</td>
      <td>Proteomics</td>
      <td>thermo_raw_file_converter</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/ThermoRawFileParser</td>
    </tr>
    <tr>
      <td></td>
      <td>bed_to_protein_map</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Converts a BED file to a tabular list of exon locations</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>bed_to_protein_map</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/bed_to_protein_map</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_plus_remote_blastp</td>
      <td>2.6.0</td>
      <td>2.14.1</td>
      <td>blast</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>NCBI BLAST+ with -remote option</td>
      <td></td>
      <td>To update</td>
      <td>https://blast.ncbi.nlm.nih.gov/</td>
      <td>Sequence Analysis</td>
      <td>blast_plus_remote_blastp</td>
      <td>galaxyp</td>
      <td>https://github.com/peterjc/galaxy_blast/tree/master/tools/ncbi_blast_plus</td>
    </tr>
    <tr>
      <td></td>
      <td>bumbershoot</td>
      <td>3.0.21142</td>
      <td>3_0_21142_0e4f4a4</td>
      <td>bumbershoot</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>http://proteowizard.sourceforge.net/</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot</td>
    </tr>
    <tr>
      <td></td>
      <td>calisp</td>
      <td>3.0.13</td>
      <td>3.0.13</td>
      <td>calisp</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calgary approach to isotopes in proteomics</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/kinestetika/Calisp/</td>
      <td>Proteomics</td>
      <td>calisp</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/calisp</td>
    </tr>
    <tr>
      <td></td>
      <td>cardinal</td>
      <td>2.10.0</td>
      <td>3.2.1</td>
      <td>bioconductor-cardinal</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Statistical and computational tools for analyzing mass spectrometry imaging datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://cardinalmsi.org</td>
      <td>Proteomics, Metabolomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/cardinal</td>
    </tr>
    <tr>
      <td></td>
      <td>dbbuilder</td>
      <td>0.3.4</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Protein Database Downloader</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
      <td>Proteomics</td>
      <td>dbbuilder</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dbbuilder</td>
    </tr>
    <tr>
      <td></td>
      <td>decoyfasta</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Galaxy tool wrapper for the transproteomic pipeline decoyFASTA tool.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>decoyfasta</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/decoyfasta</td>
    </tr>
    <tr>
      <td></td>
      <td>dia_umpire</td>
      <td>2.1.3</td>
      <td>2.1.6</td>
      <td>dia_umpire</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIA-Umpire analysis for data independent acquisition (DIA) mass spectrometry-based proteomics</td>
      <td></td>
      <td>To update</td>
      <td>http://diaumpire.sourceforge.net/</td>
      <td>Proteomics</td>
      <td>dia_umpire</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/dia_umpire</td>
    </tr>
    <tr>
      <td></td>
      <td>dialignr</td>
      <td>1.2.0</td>
      <td>2.8.0</td>
      <td>bioconductor-dialignr</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DIAlignR is an R package for retention time alignment of targeted mass spectrometric data, including DIA and SWATH-MS data. This tool works with MS2 chromatograms directly and uses dynamic programming for alignment of raw chromatographic traces. DIAlignR uses a hybrid approach of global (feature-based) and local (raw data-based) alignment to establish correspondence between peaks.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/shubham1637/DIAlignR</td>
      <td>Proteomics</td>
      <td>dialignr</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/dialignr</td>
    </tr>
    <tr>
      <td></td>
      <td>diann</td>
      <td>1.8.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>DiaNN (DIA-based Neural Networks) is a software for DIA/SWATH data processing.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/vdemichev/DiaNN</td>
      <td>Proteomics</td>
      <td>diann</td>
      <td>galaxyp</td>
      <td>https://github.com/vdemichev/DiaNN</td>
    </tr>
    <tr>
      <td></td>
      <td>diapysef</td>
      <td>0.3.5.0</td>
      <td>1.0.10</td>
      <td>diapysef</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>diapysef is a convenience package for working with DIA-PASEF data</td>
      <td></td>
      <td>To update</td>
      <td>https://pypi.org/project/diapysef/</td>
      <td>Proteomics</td>
      <td>diapysef</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproject/tools-iuc/tree/master/tools/diapysef</td>
    </tr>
    <tr>
      <td></td>
      <td>diffacto</td>
      <td>1.0.6</td>
      <td>1.0.6</td>
      <td>diffacto</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Diffacto comparative protein abundance estimation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/statisticalbiotechnology/diffacto</td>
      <td>Proteomics</td>
      <td>diffacto</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/diffacto</td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>downloads eggnog data for eggnog-mapper</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper_data_manager</td>
    </tr>
    <tr>
      <td></td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>download eggnog data for eggnog-mapper</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>data_manager_eggnog_mapper_abspath</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/data_manager_eggnog_mapper_abspath</td>
    </tr>
    <tr>
      <td></td>
      <td>eggnog_mapper</td>
      <td>2.1.8</td>
      <td>2.1.12</td>
      <td>eggnog-mapper</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>eggnog-mapper fast functional annotation of novel sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>eggnog_mapper</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/eggnog_mapper/eggnog_mapper</td>
    </tr>
    <tr>
      <td></td>
      <td>encyclopedia</td>
      <td>1.12.34</td>
      <td>2.12.30</td>
      <td>encyclopedia</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Mass Spec Data-Independent Acquisition (DIA) MS/MS analysis</td>
      <td></td>
      <td>To update</td>
      <td>https://bitbucket.org/searleb/encyclopedia/wiki/Home</td>
      <td>Proteomics</td>
      <td>encyclopedia</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/encyclopedia/tools/encyclopedia</td>
    </tr>
    <tr>
      <td></td>
      <td>fastg2protlib</td>
      <td>1.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate FASTA from FASTG</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/fastg2protlib.git</td>
      <td>Proteomics</td>
      <td>fastg2protlib</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/fastg2protlib</td>
    </tr>
    <tr>
      <td></td>
      <td>feature_alignment</td>
      <td>0.11.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>TRIC integrates information from all available runs via a graph-based alignment strategy</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>feature_alignment</td>
      <td>galaxyp</td>
      <td>https://github.com/msproteomicstools/msproteomicstools/blob/master/TRIC-README.md</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_by_fasta_ids</td>
      <td>2.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter FASTA on the headers and/or the sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>filter_by_fasta_ids</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/filter_by_fasta_ids</td>
    </tr>
    <tr>
      <td></td>
      <td>flashlfq</td>
      <td>1.0.3.1</td>
      <td>1.2.5</td>
      <td>flashlfq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>FlashLFQ mass-spectrometry proteomics label-free quantification</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/smith-chem-wisc/FlashLFQ</td>
      <td>Proteomics</td>
      <td>flashlfq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/flashlfq</td>
    </tr>
    <tr>
      <td></td>
      <td>hardklor</td>
      <td>2.30.1+galaxy1</td>
      <td>2.3.2</td>
      <td>hardklor</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Hardklör</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hardklor</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/hardklor</td>
    </tr>
    <tr>
      <td></td>
      <td>idconvert</td>
      <td></td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert mass spectrometry identification files on linux or MacOSX</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>idconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>lfq_protein_quant</td>
      <td>1.0</td>
      <td>2.26.0</td>
      <td>bioconductor-msnbase</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Enable protein summarisation and quantitation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/compomics/LFQ_galaxy_p</td>
      <td>Proteomics</td>
      <td>lfq_protein_quant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/lfq_protein_quant</td>
    </tr>
    <tr>
      <td></td>
      <td>maldiquant</td>
      <td>1.22.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MALDIquant provides a complete analysis pipeline for MALDI-TOF and other 2D mass spectrometry data.</td>
      <td></td>
      <td>To update</td>
      <td>http://strimmerlab.org/software/maldiquant/</td>
      <td>Proteomics</td>
      <td>MALDIquant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/MALDIquant</td>
    </tr>
    <tr>
      <td></td>
      <td>map_peptides_to_bed</td>
      <td>0.2</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Map peptides to a reference genome for display by a genome browser</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>map_peptides_to_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/map_peptides_to_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>maxquant</td>
      <td>2.0.3.0</td>
      <td>2.0.3.0</td>
      <td>maxquant</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>wrapper for MaxQuant</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
      <td>Proteomics</td>
      <td>maxquant</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/maxquant</td>
    </tr>
    <tr>
      <td></td>
      <td>meta_proteome_analyzer</td>
      <td>2.0.0</td>
      <td>2.0.0</td>
      <td>mpa-portable</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaProteomeAnalyzer</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/meta-proteome-analyzer/</td>
      <td>Proteomics</td>
      <td>meta_proteome_analyzer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/meta_proteome_analyzer</td>
    </tr>
    <tr>
      <td></td>
      <td>metagene_annotator</td>
      <td>1.0.0</td>
      <td>1.0</td>
      <td>metagene_annotator</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MetaGeneAnnotator gene-finding program for prokaryote and phage</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>metagene_annotator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/metagene_annotator</td>
    </tr>
    <tr>
      <td></td>
      <td>metanovo</td>
      <td>1.9.4</td>
      <td>1.9.4</td>
      <td>metanovo</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Produce targeted databases for mass spectrometry analysis.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/uct-cbio/proteomics-pipelines</td>
      <td>Proteomics</td>
      <td>metanovo</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metanovo</td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>metaquantome</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>quantitative analysis of microbiome taxonomy and function</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/galaxyproteomics/metaquantome/</td>
      <td>Proteomics</td>
      <td>metaquantome</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/metaquantome</td>
    </tr>
    <tr>
      <td></td>
      <td>moFF</td>
      <td>2.0.3</td>
      <td>2.0.3</td>
      <td>moff</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>moFF (a modest Feature Finder) extracts MS1 intensities from RAW and mzML spectrum files.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://github.com/compomics/moFF</td>
      <td>Proteomics</td>
      <td>proteomics_moff</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/moFF</td>
    </tr>
    <tr>
      <td></td>
      <td>morpheus</td>
      <td>2.255.0</td>
      <td>287</td>
      <td>morpheus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Morpheus MS Search Application</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>morpheus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/morpheus</td>
    </tr>
    <tr>
      <td></td>
      <td>mqppep</td>
      <td>0.1.19</td>
      <td>1.62.1</td>
      <td>bioconductor-preprocesscore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MaxQuant Phosphoproteomic Enrichment Pipeline - Preprocessing and ANOVA</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/</td>
      <td>Proteomics</td>
      <td>mqppep</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep</td>
    </tr>
    <tr>
      <td></td>
      <td>msconvert</td>
      <td>3.0.20287</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>msconvert Convert and/or filter mass spectrometry files (including vendor formats) using the official Docker container</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msconvert</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msconvert</td>
    </tr>
    <tr>
      <td></td>
      <td>msgfplus</td>
      <td>0.5</td>
      <td>2023.01.1202</td>
      <td>msgf_plus</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSGF+</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msgfplus</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msgfplus</td>
    </tr>
    <tr>
      <td></td>
      <td>msms_extractor</td>
      <td>1.0.0</td>
      <td>3_0_9992</td>
      <td>proteowizard</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Extract MS/MS scans from the mzML file(s) based on PSM report.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>msms_extractor</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>msstats</td>
      <td>4.0.0</td>
      <td>4.8.3</td>
      <td>bioconductor-msstats</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstats tool for analyzing mass spectrometry proteomic datasets</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/MeenaChoi/MSstats</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstats</td>
    </tr>
    <tr>
      <td></td>
      <td>msstatstmt</td>
      <td>2.0.0</td>
      <td>2.8.0</td>
      <td>bioconductor-msstatstmt</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
      <td></td>
      <td>To update</td>
      <td>http://msstats.org/msstatstmt/</td>
      <td>Proteomics</td>
      <td>msstatstmt</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt</td>
    </tr>
    <tr>
      <td></td>
      <td>mt2mq</td>
      <td>1.1.0</td>
      <td></td>
      <td>r-tidyverse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mt2mq</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>mz_to_sqlite</td>
      <td>2.1.1+galaxy0</td>
      <td>2.1.1</td>
      <td>mztosqlite</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a SQLite database for proteomics data</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>mz_to_sqlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mz_to_sqlite</td>
    </tr>
    <tr>
      <td></td>
      <td>openms</td>
      <td>2.8</td>
      <td>3.1.0</td>
      <td>openms</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>OpenMS Suite for LC/MS data management and analyses</td>
      <td></td>
      <td>To update</td>
      <td>https://www.openms.de/</td>
      <td>Proteomics</td>
      <td>openms</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms</td>
    </tr>
    <tr>
      <td></td>
      <td>pathwaymatcher</td>
      <td></td>
      <td>1.9.1</td>
      <td>pathwaymatcher</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Reactome Pathway Matcher</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/LuisFranciscoHS/PathwayMatcher</td>
      <td>Proteomics</td>
      <td>reactome_pathwaymatcher</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pathwaymatcher</td>
    </tr>
    <tr>
      <td></td>
      <td>pep_pointer</td>
      <td>0.1.3+galaxy1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepPointer categorizes peptides by their genomic coordinates.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Genomic Interval Operations, Proteomics</td>
      <td>pep_pointer</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pep_pointer</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery</td>
      <td>1.6.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A peptide-centric MS search engine for novel peptide identification and validation.</td>
      <td></td>
      <td>To update</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery</td>
    </tr>
    <tr>
      <td></td>
      <td>pepquery2</td>
      <td>2.0.2</td>
      <td>2.0.2</td>
      <td>pepquery</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PepQuery2 peptide-centric MS search for peptide identification and validation</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://pepquery.org</td>
      <td>Proteomics</td>
      <td>pepquery2</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepquery2</td>
    </tr>
    <tr>
      <td></td>
      <td>peptide_genomic_coordinate</td>
      <td>1.0.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Gets genomic coordinate of peptides based on the information in mzsqlite and genomic mapping sqlite files</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>peptide_genomic_coordinate</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>peptideshaker</td>
      <td></td>
      <td>4.3.1</td>
      <td>searchgui</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PeptideShaker and SearchGUI</td>
      <td></td>
      <td>To update</td>
      <td>http://compomics.github.io</td>
      <td>Proteomics</td>
      <td>peptideshaker</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/peptideshaker</td>
    </tr>
    <tr>
      <td></td>
      <td>pepxml_to_xls</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Convert PepXML to Tabular</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>pepxml_to_xls</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pepxml_to_xls</td>
    </tr>
    <tr>
      <td></td>
      <td>percolator</td>
      <td>3.5</td>
      <td>3.5</td>
      <td>percolator</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Percolator</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics</td>
      <td>percolator</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/percolator</td>
    </tr>
    <tr>
      <td></td>
      <td>pi_db_tools</td>
      <td>1.3</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>HiRIEF tools</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>hirieftools</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/pi_db_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>pmd_fdr</td>
      <td>1.4.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Calculate Precursor Mass Discrepancy (PMD) for MS/MS</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/slhubler/PMD-FDR-for-Galaxy-P</td>
      <td>Proteomics</td>
      <td>pmd_fdr</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pmd_fdr</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db</td>
      <td>1.22.0</td>
      <td>1.37.1</td>
      <td>bioconductor-rgalaxy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CustomProDB</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
    </tr>
    <tr>
      <td></td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>CustomProDB Annotation</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/customProDB.html</td>
      <td>Proteomics</td>
      <td>custom_pro_db_annotation_data_manager</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/custom_pro_db</td>
    </tr>
    <tr>
      <td></td>
      <td>psm2sam</td>
      <td>1.3.2.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>PSM to SAM</td>
      <td></td>
      <td>To update</td>
      <td>https://bioconductor.org/packages/release/bioc/html/proBAMr.html</td>
      <td>Proteomics</td>
      <td>psm_to_sam</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tools/bumbershoot/psm2sam</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed</td>
      <td>0.1.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Translate BED transcript CDS or cDNA in 3 frames</td>
      <td></td>
      <td>To update</td>
      <td>http://rest.ensembl.org/</td>
      <td>Proteomics</td>
      <td>translate_bed</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteogenomics/translate_bed</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>0.0.1</td>
      <td>0.0.2</td>
      <td>proteomiqon-joinquantpepionswithproteins</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/JoinQuantPepIonsWithProteins.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_joinquantpepionswithproteins</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_joinquantpepionswithproteins</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labeledproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabeledProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labeledproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labeledproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>0.0.1</td>
      <td>0.0.3</td>
      <td>proteomiqon-labelfreeproteinquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool LabelFreeProteinQuantification estimates protein abundances using quantified peptide ions.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/LabelfreeProteinQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_labelfreeproteinquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_labelfreeproteinquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-mzmltomzlite</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool MzMLToMzLite allows to convert mzML files to mzLite files.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_mzmltomzlite</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomIQon_MzMLToMzLite</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidedb</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidedb</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The tool ProteomIQon PeptideDB creates a peptide database in the SQLite format.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideDB.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidedb</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidedb</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-peptidespectrummatching</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Given raw an MS run in the mzLite format, this tool iterates across all MS/MS scans, determines precursor charge states and possible peptide spectrum matches using reimplementations of SEQUEST,Andromeda and XTandem.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PeptideSpectrumMatching.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_peptidespectrummatching</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_peptidespectrummatching</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_proteininference</td>
      <td>0.0.7</td>
      <td>0.0.7</td>
      <td>proteomiqon-proteininference</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MS-based shotgun proteomics estimates protein abundances using a proxy: peptides. The process of 'Protein Inference' is concerned with the mapping of identified peptides to the proteins they putatively originated from.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/ProteinInference.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_proteininference</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_proteininference</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>0.0.8</td>
      <td>0.0.9</td>
      <td>proteomiqon-psmbasedquantification</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMBasedQuantification tool was designed to allow label-free quantification as well as quantification of full metabolic labeled samples.</td>
      <td></td>
      <td>To update</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMBasedQuantification.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmbasedquantification</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmbasedquantification</td>
    </tr>
    <tr>
      <td></td>
      <td>proteomiqon_psmstatistics</td>
      <td>0.0.8</td>
      <td>0.0.8</td>
      <td>proteomiqon-psmstatistics</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>The PSMStatistics tool utilizes semi supervised machine learning techniques to integrate search engine scores as well as the mentioned quality scores into one single consensus score.</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://csbiology.github.io/ProteomIQon/tools/PSMStatistics.html</td>
      <td>Proteomics</td>
      <td>proteomiqon_psmstatistics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/proteomiqon_psmstatistics</td>
    </tr>
    <tr>
      <td></td>
      <td>proteore_venn_diagram</td>
      <td>2021.06.08</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>ProteoRE JVenn Diagram</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>proteore_venn_diagram</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>psm_validation</td>
      <td>1.0.3</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Validate PSM from Ion Fragmentation</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/psm_fragments.git</td>
      <td>Proteomics</td>
      <td>psm_validation</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/psm_validation</td>
    </tr>
    <tr>
      <td></td>
      <td>pyprophet</td>
      <td>2.1.4</td>
      <td>2.2.5</td>
      <td>pyprophet</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Semi-supervised learning and scoring of OpenSWATH results.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PyProphet/pyprophet</td>
      <td>Proteomics</td>
      <td></td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyprophet</td>
    </tr>
    <tr>
      <td></td>
      <td>pyteomics</td>
      <td>4.4.1</td>
      <td>4.6.2</td>
      <td>pyteomics</td>
      <td>To update</td>
      <td>pyteomics</td>
      <td>Pyteomics</td>
      <td>Protein identification</td>
      <td>Proteomics, Proteomics experiment</td>
      <td>Tools using the pyteomics library</td>
      <td>Framework for proteomics data analysis, supporting mzML, MGF, pepXML and more.</td>
      <td>To update</td>
      <td>https://pyteomics.readthedocs.io/en/latest/</td>
      <td>Proteomics, Metabolomics</td>
      <td>pyteomics</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/pyteomics</td>
    </tr>
    <tr>
      <td></td>
      <td>quantp</td>
      <td>1.1.2</td>
      <td>1.11.6</td>
      <td>r-data.table</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Correlation between protein and transcript abundance</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>quantp</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>quantwiz_iq</td>
      <td>2.0</td>
      <td>2.0</td>
      <td>quantwiz-iq</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Isobaric Quantitation using QuantWiz-IQ</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://sourceforge.net/projects/quantwiz/</td>
      <td>Proteomics</td>
      <td>quantwiz_iq</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/quantwiz_iq</td>
    </tr>
    <tr>
      <td></td>
      <td>rawtools</td>
      <td></td>
      <td>2.0.4</td>
      <td>rawtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Raw Tools</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/kevinkovalchik/RawTools</td>
      <td>Proteomics</td>
      <td>rawtools</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/rawtools</td>
    </tr>
    <tr>
      <td></td>
      <td>sixgill</td>
      <td>0.2.4</td>
      <td>0.2.4</td>
      <td>sixgill</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Six-frame Genome-Inferred Libraries for LC-MS/MS</td>
      <td></td>
      <td>Up-to-date</td>
      <td></td>
      <td>Proteomics, MetaProteomics</td>
      <td>sixgill</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/sixgill</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2spectrast_irt</td>
      <td>0.1.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2spectrast_irt</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2spectrast_irt</td>
    </tr>
    <tr>
      <td></td>
      <td>spectrast2tsv</td>
      <td>0.1.0</td>
      <td>0.11.0</td>
      <td>msproteomicstools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter from spectraST files to swath input files</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>spectrast2tsv</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/spectrast2tsv</td>
    </tr>
    <tr>
      <td></td>
      <td>translate_bed_sequences</td>
      <td>0.2.0</td>
      <td>1.70</td>
      <td>biopython</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Perform 3 frame translation of BED file augmented with a sequence column</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>translate_bed_sequences</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/translate_bed_sequences</td>
    </tr>
    <tr>
      <td></td>
      <td>unipept</td>
      <td>4.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Unipept retrieves metaproteomics information</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp</td>
      <td>Proteomics</td>
      <td>unipept</td>
      <td>galaxyp</td>
      <td>https://unipept.ugent.be/apidocs</td>
    </tr>
    <tr>
      <td></td>
      <td>uniprotxml_downloader</td>
      <td>2.4.0</td>
      <td></td>
      <td>requests</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Download UniProt proteome in XML or fasta format</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Proteomics</td>
      <td>uniprotxml_downloader</td>
      <td>galaxyp</td>
      <td>https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/uniprotxml_downloader</td>
    </tr>
    <tr>
      <td></td>
      <td>validate_fasta_database</td>
      <td>0.1.5</td>
      <td>1.0</td>
      <td>validate-fasta-database</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>runs Compomics database identification tool on any FASTA database, and separates valid and invalid entries based on a series of checks.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Fasta Manipulation, Proteomics</td>
      <td>validate_fasta_database</td>
      <td>galaxyp</td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>bamparse</td>
      <td>4.1.1</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates hit count lists from bam alignments.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>bamparse</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/bamparse</td>
    </tr>
    <tr>
      <td></td>
      <td>blast_to_scaffold</td>
      <td>1.1.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate DNA scaffold from blastn or tblastx alignments of Contigs</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blast_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blast_to_scaffold</td>
    </tr>
    <tr>
      <td></td>
      <td>blastparser_and_hits</td>
      <td>2.7.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Parse blast outputs and compile hits</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>blastparser_and_hits</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/blastparser_and_hits</td>
    </tr>
    <tr>
      <td></td>
      <td>blastx_to_scaffold</td>
      <td>1.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate DNA scaffold from blastx alignment of Contigs</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Sequence Analysis, Assembly</td>
      <td>blastx_to_scaffold</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/blastx_to_scaffold</td>
    </tr>
    <tr>
      <td></td>
      <td>cap3</td>
      <td>2.0.1</td>
      <td>10.2011</td>
      <td>cap3</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>cap3 wrapper</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly</td>
      <td>cap3</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/cap3</td>
    </tr>
    <tr>
      <td></td>
      <td>deseq2_normalization</td>
      <td>1.40.2+galaxy0</td>
      <td>1.40.2</td>
      <td>bioconductor-deseq2</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Normalizes gene hitlists</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Sequence Analysis, Statistics</td>
      <td>deseq2_normalization</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/deseq2_normalization</td>
    </tr>
    <tr>
      <td></td>
      <td>fisher_test</td>
      <td>2.32.0+galaxy0</td>
      <td>2.32.0</td>
      <td>bioconductor-qvalue</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Fisher's exact test on two-column hit lists.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Statistics</td>
      <td>fishertest</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/main/tools/fisher_test</td>
    </tr>
    <tr>
      <td></td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>0.9.6</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates PCA, t-SNE and HCPC visualisation</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Transcriptomics, Visualization</td>
      <td>gsc_high_dimensions_visualisation</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/gsc_high_dimension_visualization</td>
    </tr>
    <tr>
      <td></td>
      <td>guppy</td>
      <td>0.2.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>A wrapper for the guppy basecaller tool from Oxford Nanopore Technologies</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>guppy_basecaller</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/guppy</td>
    </tr>
    <tr>
      <td></td>
      <td>high_dim_heatmap</td>
      <td>1.2.0</td>
      <td>2.17.0</td>
      <td>r-gplots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>gplot heatmap.2 function adapted for plotting large heatmaps</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/cran/gplots</td>
      <td>Visualization</td>
      <td>high_dim_heatmap</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/master/tools/high_dim_heatmap</td>
    </tr>
    <tr>
      <td></td>
      <td>mapping_quality_stats</td>
      <td>0.19.1+galaxy0</td>
      <td>1.3.2</td>
      <td>r-optparse</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Collects and shows the distribution of MAPQ values in a BAM alignment file</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Statistics</td>
      <td>mapping_quality_stats</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/mapping_quality_stats</td>
    </tr>
    <tr>
      <td></td>
      <td>mircounts</td>
      <td>1.5.1</td>
      <td></td>
      <td>tar</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates miRNA count lists from read alignments to mirBase.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics</td>
      <td>mircounts</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts</td>
    </tr>
    <tr>
      <td></td>
      <td>oases</td>
      <td>1.3.0</td>
      <td>0.2.09</td>
      <td>oases</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Short read assembler</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Assembly, RNA</td>
      <td>oases</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/oases</td>
    </tr>
    <tr>
      <td></td>
      <td>probecoverage</td>
      <td>0.7.0</td>
      <td>1.18</td>
      <td>samtools</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>computes and plots read coverage of genomic regions by sequencing datasets</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Sequence Analysis, Genomic Interval Operations, Graphics, Statistics</td>
      <td>probecoverage</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/probecoverage</td>
    </tr>
    <tr>
      <td></td>
      <td>rsem</td>
      <td></td>
      <td>1.3.3</td>
      <td>rsem</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>transcript quantification from RNA-Seq data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/deweylab/RSEM</td>
      <td>Transcriptomics, RNA</td>
      <td>rsem</td>
      <td>artbio</td>
      <td>https://github.com/artbio/tools-artbio/tree/master/tools/rsem</td>
    </tr>
    <tr>
      <td></td>
      <td>sashimi_plot</td>
      <td>0.1.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates a sashimi plot from bam files.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Transcriptomics, Graphics, Visualization</td>
      <td>sashimi_plot</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sashimi_plot</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_clusters</td>
      <td>1.3.0</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>clusters small rna reads in alignment BAM files</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_clusters</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_clusters</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_maps</td>
      <td>3.1.1</td>
      <td></td>
      <td>numpy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates small read maps from alignment BAM files</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, SAM, Graphics, Next Gen Mappers</td>
      <td>small_rna_maps</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_maps</td>
    </tr>
    <tr>
      <td></td>
      <td>small_rna_signatures</td>
      <td>3.4.2</td>
      <td>0.22.0</td>
      <td>pysam</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Computes the tendency of small RNAs to overlap with each other.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>small_rna_signatures</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/small_rna_signatures</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie</td>
      <td>2.3.0</td>
      <td>1.3.1</td>
      <td>bowtie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>bowtie wrapper tool to align small RNA sequencing reads</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Next Gen Mappers</td>
      <td>sr_bowtie</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie</td>
    </tr>
    <tr>
      <td></td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>2.8</td>
      <td>1.3.1</td>
      <td>bowtie</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Maps iteratively small RNA sequencing datasets to reference sequences.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA</td>
      <td>sr_bowtie_dataset_annotation</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/sr_bowtie_dataset_annotation</td>
    </tr>
    <tr>
      <td></td>
      <td>tarfast5</td>
      <td>0.6.1</td>
      <td></td>
      <td>pigz</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>produces a tar.gz archive of fast5 sequence files</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>Nanopore</td>
      <td>tarfast5</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/tarfast5</td>
    </tr>
    <tr>
      <td></td>
      <td>xpore</td>
      <td>2.1+galaxy0</td>
      <td>2.1</td>
      <td>xpore</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Identification and quantification of differential RNA modifications from direct RNA sequencing</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/GoekeLab/xpore</td>
      <td>Nanopore</td>
      <td>xpore</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/xpore</td>
    </tr>
    <tr>
      <td></td>
      <td>yac_clipper</td>
      <td>2.5.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Clips 3' adapters for small RNA sequencing reads.</td>
      <td></td>
      <td>To update</td>
      <td>http://artbio.fr</td>
      <td>RNA, Fastq Manipulation</td>
      <td>yac_clipper</td>
      <td>artbio</td>
      <td>https://github.com/ARTbio/tools-artbio/tree/master/tools/yac_clipper</td>
    </tr>
    <tr>
      <td></td>
      <td>EMLassemblyline</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>r-emlassemblyline</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools using EML Assembly Line R package to generate EML metadata from template metadata files and vice versa</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/EDIorg/EMLassemblyline</td>
      <td>Ecology</td>
      <td>emlassemblyline</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/EMLassemblyline</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoregionalization_workflow</td>
      <td>0.1.0+galaxy0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute ecoregionalization with BRT model predictions and clustering.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/PaulineSGN/Workflow_Galaxy</td>
      <td>Ecology</td>
      <td>ecoregionalization</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/Ecoregionalization_workflow</td>
    </tr>
    <tr>
      <td></td>
      <td>PAMPA</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to compute and analyse biodiversity metrics</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>pampa</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/PAMPA-Galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>champ_blocs</td>
      <td>0.0.0</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute indicators for turnover boulders fields</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/champ_blocs</td>
    </tr>
    <tr>
      <td></td>
      <td>consensus_from_alignments</td>
      <td>1.0.0</td>
      <td></td>
      <td>r-bioseq</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tool to compute a consensus sequence from several aligned fasta sequences</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>consalign</td>
      <td>ecology</td>
      <td>https://github.com/ColineRoyaux/Galaxy_tool_projects/tree/main/consensus_from_alignments</td>
    </tr>
    <tr>
      <td></td>
      <td>data_exploration</td>
      <td>0.0.0</td>
      <td></td>
      <td>r-tangles</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Explore data through multiple statistical tools</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Data_explo_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>xarray</td>
      <td>2022.3.0</td>
      <td></td>
      <td>xarray</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>xarray (formerly xray) is an open source project and Python package that makes working withlabelled multi-dimensional arrays simple, efficient, and fun!xarray integrates with Dask to support parallel computations and streaming computation on datasetsthat don’t fit into memory.</td>
      <td></td>
      <td>To update</td>
      <td>http://xarray.pydata.org</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/data_manipulation/xarray/</td>
    </tr>
    <tr>
      <td></td>
      <td>gdal</td>
      <td>3.0.0</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.gdal.org</td>
      <td>Ecology</td>
      <td>gdal</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/gdal</td>
    </tr>
    <tr>
      <td></td>
      <td>obisindicators</td>
      <td>0.0.2</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiveristy indicators for marine data from obis</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/obisindicators</td>
    </tr>
    <tr>
      <td></td>
      <td>regionalgam</td>
      <td>1.5</td>
      <td></td>
      <td>r-mgcv</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RetoSchmucki/regionalGAM</td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/regionalgam</td>
    </tr>
    <tr>
      <td></td>
      <td>spocc</td>
      <td>1.2.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Get species occurences data</td>
      <td></td>
      <td>To update</td>
      <td>https://cran.r-project.org/web/packages/spocc/index.html</td>
      <td>Ecology</td>
      <td>spocc_occ</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/spocc</td>
    </tr>
    <tr>
      <td></td>
      <td>srs_tools</td>
      <td>0.0.1</td>
      <td></td>
      <td>r-base</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Compute biodiversity indicators for remote sensing data from Sentinel 2</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td></td>
      <td>ecology</td>
      <td>https://github.com/Marie59/Sentinel_2A/srs_tools</td>
    </tr>
    <tr>
      <td></td>
      <td>stoc</td>
      <td>0.0.2</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools to analyse STOC data.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Ecology</td>
      <td>stoceps</td>
      <td>ecology</td>
      <td>https://github.com/Alanamosse/Galaxy-E/tree/stoctool/tools/stoc</td>
    </tr>
    <tr>
      <td></td>
      <td>vigiechiro</td>
      <td>0.1.1</td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools created by the vigiechiro team to analyses and identify chiro sounds files.</td>
      <td></td>
      <td>To update</td>
      <td>https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro</td>
      <td>Ecology</td>
      <td>vigiechiro</td>
      <td>ecology</td>
      <td>https://github.com/galaxyecology/tools-ecology/tree/master/tools/vigiechiro</td>
    </tr>
    <tr>
      <td></td>
      <td>climate-stripes</td>
      <td>1.0.2</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Create climate stripes from a tabular input file</td>
      <td></td>
      <td>To update</td>
      <td>https://www.climate-lab-book.ac.uk/2018/warming-stripes/</td>
      <td>Climate Analysis, Visualization</td>
      <td>climate_stripes</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/climate-stripes</td>
    </tr>
    <tr>
      <td></td>
      <td>mean-per-zone</td>
      <td>0.2.0</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Creates a png image showing statistic over areas as defined in the vector file</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/blob/master/tools/mean-per-zone/</td>
      <td>Visualization, GIS, Climate Analysis</td>
      <td>mean_per_zone</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/mean-per-zone</td>
    </tr>
    <tr>
      <td></td>
      <td>psy-maps</td>
      <td>1.2.1</td>
      <td></td>
      <td>python</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Visualization of regular geographical data on a map with psyplot</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/Chilipp/psy-maps</td>
      <td>Visualization, Climate Analysis</td>
      <td>psy_maps</td>
      <td>climate</td>
      <td>https://github.com/NordicESMhub/galaxy-tools/tree/master/tools/psy-maps</td>
    </tr>
    <tr>
      <td></td>
      <td>droplet-barcode-plot</td>
      <td>1.6.1+galaxy2</td>
      <td>0.0.1</td>
      <td>scxa-plots</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Make a cell barcode plot for droplet single-cell RNA-seq QC</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>droplet_barcode_plot</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/droplet-rank-plot/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_provider</td>
      <td>0.4.4</td>
      <td>0.4.6</td>
      <td>atlas-fastq-provider</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Retrieval and download of FASTQ files from ENA and other repositories such as HCA.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/atlas-fastq-provider</td>
      <td>Data Source, RNA, Transcriptomics</td>
      <td>atlas_fastq_provider</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/fastq_provider</td>
    </tr>
    <tr>
      <td></td>
      <td>gtf-2-gene-list</td>
      <td>1.52.0+galaxy0</td>
      <td>1.1.0</td>
      <td>atlas-gene-annotation-manipulation</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Utility to extract annotations from Ensembl GTF files.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>gtf2gene_list</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/gtf-2-gene-list/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>fastq_utils</td>
      <td>0.25.1+galaxy0</td>
      <td>0.25.2</td>
      <td>fastq_utils</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Set of tools for handling fastq files</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/nunofonseca/fastq_utils</td>
      <td>Transcriptomics, RNA</td>
      <td>fastq_utils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_utils</td>
    </tr>
    <tr>
      <td></td>
      <td>salmon-kallisto-mtx-to-10x</td>
      <td>0.0.1+galaxy6</td>
      <td></td>
      <td>scipy</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Transforms .mtx matrix and associated labels into a format compatible with tools expecting old-style 10X data</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary</td>
      <td>Sequence Analysis</td>
      <td>salmon_kallisto_mtx_to_10x</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/salmon-kallisto-mtx-to-10x/.shed.yml</td>
    </tr>
    <tr>
      <td></td>
      <td>cell-types-analysis</td>
      <td>1.1.1</td>
      <td>0.1.11</td>
      <td>cell-types-analysis</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for analysis of predictions from scRNAseq cell type classification tools, see https://github.com/ebi-gene-expression-group/cell-types-analysis</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics</td>
      <td>suite_cell_types_analysis</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>data-hca</td>
      <td>v0.0.4+galaxy0</td>
      <td>0.0.4</td>
      <td>hca-matrix-downloader</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for interacting with the Human Cell Atlas resource https://prod.data.humancellatlas.org/explore/projects</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_human_cell_atlas_tools</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>data-scxa</td>
      <td>v0.0.2+galaxy2</td>
      <td></td>
      <td>wget</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Tools for interacting with the EMBL-EBI Expression Atlas resource https://www.ebi.ac.uk/gxa/home https://www.ebi.ac.uk/gxa/sc/home</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, Sequence Analysis</td>
      <td>suite_ebi_expression_atlas</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>dropletutils</td>
      <td>1.0.4</td>
      <td>0.0.5</td>
      <td>dropletutils-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed DropletUtils functionality tools, based on https://github.com/ebi-gene-expression-group/dropletutils-scripts and DropletUtils 1.0.3</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_dropletutils</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>garnett</td>
      <td>0.2.8</td>
      <td>0.0.5</td>
      <td>garnett-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Garnett functionality tools, see https://github.com/ebi-gene-expression-group/garnett-cli and r-garnett 0.2.8</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_garnett</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>monocle3</td>
      <td>0.1.4</td>
      <td>0.0.9</td>
      <td>monocle3-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed monocle3 functionality tools, based on https://github.com/ebi-gene-expression-group/monocle-scripts and monocle3 0.1.2.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_monocle3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>sc3</td>
      <td>1.8.0</td>
      <td>0.0.6</td>
      <td>sc3-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed SC3 functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-sc3-scripts and SC3 1.8.0.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_sc3</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scanpy</td>
      <td>1.8.1</td>
      <td>1.1.6</td>
      <td>scanpy-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>scanpy-scripts, command-line wrapper scripts around Scanpy.</td>
      <td></td>
      <td>To update</td>
      <td>https://scanpy.readthedocs.io</td>
      <td>Transcriptomics, Sequence Analysis, RNA</td>
      <td>scanpy_scripts</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/tertiary-analysis/scanpy</td>
    </tr>
    <tr>
      <td></td>
      <td>scater</td>
      <td>1.10.0</td>
      <td>0.0.5</td>
      <td>scater-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Scater functionality tools, based on https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts and Scater 1.8.4.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scater</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scmap</td>
      <td>1.6.3</td>
      <td>0.0.11</td>
      <td>scmap-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed scmap functionality tools, based on https://github.com/ebi-gene-expression-group/scmap-cli and scmap 1.6.0.</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scmap</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>scpred</td>
      <td>1.0.2</td>
      <td>0.1.0</td>
      <td>scpred-cli</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed scPred functionality tools, see https://github.com/ebi-gene-expression-group/scpred-cli and r-scPred 1.0</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_scpred</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>seurat</td>
      <td>0.3.0</td>
      <td>4.0.0</td>
      <td>seurat-scripts</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>De-composed Seurat functionality tools, based on https://github.com/ebi-gene-expression-group/r-seurat-scripts and Seurat 2.3.1</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Transcriptomics, RNA, Statistics, Sequence Analysis</td>
      <td>suite_seurat</td>
      <td>ebi-gxa</td>
      <td>https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/</td>
    </tr>
    <tr>
      <td></td>
      <td>mzml_validator</td>
      <td>0.1.0+galaxy2</td>
      <td></td>
      <td>lxml</td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative.</td>
      <td></td>
      <td>To update</td>
      <td>https://github.com/RECETOX/galaxytools</td>
      <td>Metabolomics, Proteomics</td>
      <td></td>
      <td>recetox</td>
      <td>https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator</td>
    </tr>
    <tr>
      <td></td>
      <td>consolidate_vcfs</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Combines freebayes and mpileup files for use by vcf2snvalignment</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>consolidate_vcfs</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_density</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Filter out position based on distance between SNVs</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_density</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_stats</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl filter_stats</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_stats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>filter_vcf</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl filter_vcf</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>filter_vcf</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>find_repeats</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Find repetitive regions on a reference genome using MUMMer</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>find_repeats</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>snv_matrix</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generate matrix of SNV distances</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>snv_matrix</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>vcf2snvalignment</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Generates multiple alignment of variant calls</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>vcf2snvalignment</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>verify_map</td>
      <td>1.8.2</td>
      <td>1.8.2</td>
      <td>snvphyl-tools</td>
      <td>Up-to-date</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>Checks the mapping quality of all BAM(s)</td>
      <td></td>
      <td>Up-to-date</td>
      <td>https://snvphyl.readthedocs.io/en/latest/</td>
      <td>Sequence Analysis</td>
      <td>verify_map</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
    <tr>
      <td></td>
      <td>suite_snvphyl</td>
      <td></td>
      <td></td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>SNVPhyl suite defining all dependencies for SNVPhyl</td>
      <td></td>
      <td>To update</td>
      <td></td>
      <td>Sequence Analysis</td>
      <td>suite_snvphyl_1_2_3</td>
      <td>nml</td>
      <td>https://github.com/phac-nml/snvphyl-galaxy</td>
    </tr>
  </tbody>
</table>
</div>